---
output: pdf_document
---

\chapter{Introduction}
\section{Gene expression}
\subsection{The central dogma of gene expression}
  \begin{wrapfigure}{r}{.6\textwidth}
  \includegraphics{./figures/geneExprPath_2.pdf}
  \caption{The gene expression pathway - DNA is transcribed into RNA containing a 5' cap (black oval) introns (teal boxes),exons (purple boxes) and a poly(A) tail. RNAs are processed into mRNAs that consisting of a 5' cap, exons and a poly(A) tail. mRNAs can be transported out of the cellular nucleus into the cytoplasm where they can be degraded, stored or translated into proteins depending on cellular demands. Synthesised proteins can be degraded by proteosomes. \label{fig:geneExprPath}}
\end{wrapfigure}
 The whole genetic code of an organisms is stored as deoxyribonucleic acid (DNA) molecules in a double stranded formation as highly condensed chromosomes. Transcription is the process whereby temporary copies of the DNA are generated, called transcripts or ribonucleic acid (RNA), and occurs in the cell’s core (nucleus). RNAs can undergo processing by which multiple different variants coming from the same region are produced. A subset of processed RNAs is the so called messenger RNA (mRNA). These mRNAs are transported from the nucleus into the cytoplasm where they can be stored, degraded or associate with ribosomes. Ribosomes translate mRNAs in proteins that fulfill major functions within the cell. Proteins themselves can also undergo degradation. (**see figure \ref{fig:geneExprPath}**). This flow of genetic information into expressed proteins is commonly referred to as the central dogma in molecular biology [@Crick1970].
\clearpage
\subsection{Contribution to gene expression}
 Proteins are the last product of the gene expression pathway and carry out the vast majority of all cellular functions. While it is apparent that modulation of protein levels will offer information on the changes in gene expression, it cannot completely answer the question as to why the levels change. In a disease context, protein levels alone might offer sufficient insight to explain phenotypic differences. However, information on how these differences arise mechanistically is obscured. Yet, these differences in underlying biological mechanisms can be used as targets in therapeutic strategies.
  
 Experimental methods to measure gene expression at different steps are often referred to as "omics" (e.g. proteomics for protein expression, transcriptomics for mRNA expression). These methods provide snapshots of the step under scrutiny in a specific context (steady state or perturbation) for a large portion of genes or proteins. Initially, transcriptomics studies approached gene expression with the assumption that mRNA expression is the main determinant for protein levels and therefore could be used as a proxy for them. However, this view was challenged by several landmark studies that observed a poor mRNA to protein correlation and indicated a larger role of post transcriptional regulation in gene expression than previously assumed [@Lu2005; @Vogel2012; @deSousaAbreu2009; @Schwanhausser2011; @Silva2016].
  
 The debate regarding which step of the gene expression pathway contributes most is ongoing, nevertheless an understanding has been reached that the cellular context is a major determinant. At steady state, mRNA levels seem to explain protein abundance best, however in perturbed systems the contribution of transcript abundance alone is not sufficient to explain protein abundance [@Liu2016].  For example in a study that challenged immune cells, protein levels were dependent on cellular transcript levels [@Jovanovic2015]. In contrast a study investigating cells under stress observed extensive modulation at the protein levels, whereas mRNA transcript abundance was only mildly affected [@Cheng2016].
  
 While the contribution of different steps of the gene expression is dependent on many different factors, e.g. cellular state or treatments, mRNA translation (synthesis of proteins) is an essential process of this pathway. Furthermore, dysregulation of mRNA translation has been observed in multiple diseases, ranging from neurological disorders to cancer which warrants for a comprehensive understanding of this process [@Kapur2018; @Ruggero2013; @Lee2021; @Graff2009]. This thesis will focus on the role of mRNA translation in the context of cancer.
\newline
\section{mRNA translation}
\subsection{Overview of an mRNA}
 After transcription, primary RNA transcripts are processed into mRNAs. mRNAs contain a protein coding region which is flanked by untranslated regions (5' and 3' UTRs) that exert translational control over the mRNA (see \ref{regmRNA}). The 5' has a cap that is important for translation initiation [@Grifo1983], while the 3' end has a poly-A tail protecting the mRNA against degradation [@Wilusz2001]. Multiple different mRNAs (isoforms or transcript variants) from the same genomic region exist. These variants can arise due to a process called alternative splicing which alters the exon composition (e.g. coding region) of an mRNA. These variants can co-exist at the same time and have distinct properties and can perform distinct functions [@JolyAnne-Laure2018]. Furthermore, splicing resulting in alternative UTRs for the same mRNA can introduce to differences in translation [@Jewer2020;@Floor2016].
\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/UTRFeatures.pdf}
  \caption{ An mRNA consists of a coding sequence (green), 5' (light orange) and 3' (purple) untranslated regions flanking the coding sequence, a 5' cap and a poly-A tail. Located within the 5' and 3' untranslated region are cis elements (light red boxes) that can exert translational control by interfering with ribosomal movement along the mRNA or interact with trans factors (blue and red) or recruit the 43s ribsome (dark green) to the mRNA (see section \ref{regmRNA}). The codon composition of the coding sequence influences translation elongation rates (see section \ref{tRNA}).   
 \label{fig:UTRFeat}}
\end{wrapfigure}
\clearpage
\subsection{Translation of an mRNA}
  For the vast majority of protein coding mRNAs, eukaryotic mRNA translation occurs in the cytoplasm, however a small subset of mRNAs is translated in the mitochondria [@DSouza2018]. mRNA translation is a process that includes initiation, elongation, termination and ribosome recycling and is an essential process **see Figure \ref{fig:doodlemRNASteps}**. During the initiation phase a ribosome will associate with the mRNA and starts scanning along the mRNA for a start codon to begin synthesis of the polypeptide chain by incorporating amino acids [@Hinnebusch2014]. Amino acids are transferred to the ribosome by specialised RNAs called transfer RNAs (tRNA) that can recognise the genetic code in the mRNA. The availability of tRNAs as well as their modification can influence the rate of elongation. The order by which amino acids are incorporated is dictated by the order of the codons of the open reading frame (ORF). There are more possible codon combinations in comparison to amino acids. This allows that amino acids are encoded by multiple codons, for example lysine is encoded by AAA and AAG. Once the ribosome encounters a stop codon, translation will terminate and the polypeptide chain will be released. The ribosome then disassociates from the mRNA and can be recycled to engage translation of the same or another mRNA. The following sections describe these processes in more detail. 

\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/doodleTranslation.pdf}
  \caption{mRNA translation initiation, elongation, termination and ribosome recycling steps - The ribosome binds to the mRNA and initiates scanning for a start codon. The elongation phase incorporates amino acids into a polypeptide chain (i.e. the protein product). Once the end of the coding sequence is detected, by recognition of stop codons, the ribosome terminates translation and releases the polypeptide chain. The ribosome can then be recycled to participate in the translation of another mRNA or reinitiate. \label{fig:doodlemRNASteps}}
\end{wrapfigure}
\clearpage

\subsection{Initiation} \label{initiation}

 For the initiation to commence, in eukaryotes, two complexes are required; the pre-initiation complex (PIC) and the eukaryotic initiation factor 4F (eIF4F) complex. Both these complexes are governed by signalling pathways that regulate their availability dependent on cellular cues (**see section \ref{regmRNA}**). The PIC consists of the methionyl-initiatior transfer RNA (met-tRNAi) in a ternary complex (TC) with guanosine triphosphate (GTP) bound eIF2 [@Asano2000]. eIF4F is the translation initation complex containing three eIFs: eIF4E, the 5' cap binding protein, eIF4G a scaffold protein and eIF4A and RNA helicase [@Grifo1983; @Hinnebusch2006]. eIF4F recruites the PIC to the 5' cap of the mRNA after which scanning for a start codon (AUG) occurs. After AUG recognition  eIF2-GTP  is  hydrolyzed  forming  a  stable  48S  PIC.  After release  of  eIF2-GTP  the  60S ribosomal  subunit  joins  to  form  the  80S  ribosome  and  protein  synthesis can  start (**Figure \ref{initiation}**).

\subsection{Elongation}

 The 80S ribosome contains three sites important for decoding an mRNA: the acceptor (A), peptidyl (P) and exit (E) sites. During elongation in eukaryotes, aminoacytelated tRNAs are delivered to the A-site in a ternary complex with eukaryotic elongation factor 1A (eEF1A) and Guanosine triphosphate (GTP). When the tRNA recognises its cognate codon and pairs, a bond between the amino acid and the polypeptide chain is formed. The formation of the bonds causes the ribosomal units to rotate in relation to each other [@Munro2007; @Moazed1989]. The rotation causes a shift of the tRNA acceptor end from the A and P to the P and E sites, whereas the codon end remains in the A and P site. This is the "hyrbid" state of the tRNAs in the ribosome[@Dorner2006]. eEF2 then promotes the translocation by which the codon end of the tRNA follow into the P and E sites. The deacytelated tRNA is then released from the ribosome. This process is repeated until a stop codon (UAA, UGA or UAG) enters the A site of the ribosome [@Hellen2018].

\begin{figure}[ht]
\includegraphics{./figures/initiation.jpg} 
  \caption{Pathway of eukaryotic translation initiation via ribosomal scanning.
  \label{fig:initiation}}
\end{figure}

\clearpage

\subsection{Termination and recycling} 
\begin{wrapfigure}{r}{.4\textwidth}
  \includegraphics{./figures/polysome.pdf}
  \caption{Electromicrongraph of ribosomes extracted from different positions along a sucrose gradient used for polysome fractionation (A-C). For details on polysome fractionation see section \ref{exptMethod}. Reprinted with permission. DR. T. STAEHELIN et. al. Nature.1963 Aug 31;199:865-70.doi: 10.1038/199865a0. Copyright © 1963, Nature Publishing Group.
  Schematic of the TAPES method - A ribosome (A) can initiate with rate \(\alpha\) on an mRNA with a coding sequence with codons i= 1 ... L. The elongation rate at a specific codon is defined by \(\omega_i\) and \(\beta\) determines the termination rate once a stop codon is encountered. TASEP has been constantly modified, e.g. to allow for correction of initiation or elongation when the following codon is already occupied. Reprinted with permission. Juraj Szavits-Nossan and Martin R. Evans. 10.1103/PhysRevE.101.062404. ©2020 American Physical Society. 
 \label{fig:polysomes}}
\end{wrapfigure}
 mRNA translation termination is  facilitated  by  two  eukaryotic release  factors (eRF), eRF2  and  eRF3-GTP [@Stansfield1995;@Alkalaeva2006]. A TC  containing  eRF2  and  eRF3-GTP binds to the A-site of the ribosome upon recognition of a stop codon. This causes an ATP hydrolysis event resulting in a conformational change and release of the polypeptide chain. eRF1 and the ATP binding cassette protein (ABCE1) together promote  the  splitting  of  the  60S  and  40S  subunits after which they can be recycled [@Pisarev2010;@Dever2012;@Hellen2018]. 

\subsection{Translation efficiency}
 Each ribosome synthesises a single protein during translation of an mRNA assuming it is not prematurely terminated. It has been known since the '60s that translation of an mRNA occurs via multiple bound ribosomes (polysomes) simultaneaously (**see figure \ref{fig:polysomes}A-C**) [@Warner1962;@Staehelin1963]. Therefore the translation efficiency of an mRNA depends on the number of ribosomes it is associated with. While all steps of translation can affect the translation efficiency of an mRNA, it is most commonly regulated at the initiation step [@Richter2015;@Dever2012;@Jackson2010]. 

 Experimental methods, e.g. polysome profiling, are used to study translation efficiencies. Furthermore, there are efforts to model translation kinetics such as totally asymmetric simple exclusion process (TASEP) to obtain a better understanding of ribosome movements along the mRNA [@MacDonald1968;@Maniloff1969] (**See figure \ref{fig:polysomes}D**). A model that assessed ribosome traffic under steady state and high initiation rates indicates traffic is not only dependent on the densities of ribosomes but also codon specific elongation rates and their distribution along the mRNA [@Szavits-Nossan2020a]. The next section will go further into detail how initiation rates and elongation can be regulated.
\newline
\section{Regulation of mRNA translation} \label{regmRNA}

 mRNA translation is the most energy consuming process in the cell. In rat it was estimated that translation accounts for  ~20% of the cellular energy consumption [@Buttgereit1995]. The high energy consumption of mRNA translation  and its central role in the gene expression pathway requires it to be tightly regulated, especially in cancer where dysregulated metabolism is common [@Hanahan2011].

 Regulation of translation can be exerted at a global level (i.e. regulation of a large set of mRNA simultaneously) by e.g. perturbations of major signalling pathways impinging on mRNA translation. Furthermore, a strong feature of translational control is that it can affect distinct mRNA  populations selectively. This form of translational regulation acts on characteristics of mRNAs, e.g. through cis elements in the UTRs, RNA binding proteins (RBP) or regulation of initiation factors (**see figure \ref{fig:UTRFeat}**) [@Leppek2018]. Below we will discuss multiple mechanisms that regulate mRNA translation globally or selectively.

\subsection{mTOR} \label{mTOR}
 mTOR is  a  conserved  Ser/Thr  kinase  and  is  found  in  two  structurally  and  functionally  distinct  complexes,  mTORC1  and mTORC2 [@Saxton2017;@Pearce2007]. In a growth promoting environment mTOR switches cell metabolism to increase protein synthesis, lipids and nucleotides, while suppressing catabolic pathways. mTORC2 promotes survival via signalling through protein kinase A (AKT) and is involved in metabolism [@Sarbassov2005].

 mTOR activity is modulated via growth factor signalling (i.e. insulin and insulin-like growth factor; IGF1). Growth factor signalling is mediated through the phosphoinositide 3-kinase (PI3K) / AKT and Ras/mitogen-activated protein kinase (MAPK) pathways. Both these pathways are involved in oncogenic signalling and under investigation as therapeutic targets in cancer [@Hilger2002;@Yang2019]. In several cancers (e.g. breast, lung, prostate and colon) the gene encoding the p110\(\alpha\) subunit of PI3K is frequently mutated or amplified [@Levine2005;@Samuels2004;@Lee2005]. The E545K mutation  leads to a reduced inhibition of p85 at the catalytic p110\(\alpha\) subunit of PI3K [@Jiang2018]. In **study 3** we investigate oncogenic signalling via the PI3K pathway activated by insulin and the role of mTOR in mediating the resulting effects on gene expression in the MCF7 breast cancer cell line that harbours the E545K mutation [@Schneck2013]. Furthermore, hyperactivity of PI3K/AKT and Ras/MAPK signalling pathways have been reported in multiple cancers and been linked anti- cancer therapy resistance [@Populo2012;@Tan2013;@Salaroglio2019]. mTOR, as a downstream actor of these pathways integrating their signals, has therefore become a focus of anti cancer therapy by either targeting mTORC1 or using dual inhibitors for PI3K and mTOR [@Bhat2015].

\begin{wrapfigure}{r}{0.6\textwidth}
  \includegraphics{./figures/mTORsignal.jpg}
  \caption{Schematic representation of mTOR signaling to the translational machinery. Philippe P. Roux, and Ivan Topisirovic Mol. Cell. Biol. 2018; doi:10.1128/MCB.00070-18. Reprinted with permission. Copyright © 2018, American Society for Microbiology
 \label{fig:mtorsignal}}
\end{wrapfigure}
 mTOR also fulfills a central role in metabolic signalling where it integrates amino acid availability, glucose and cellular oxygen levels to form an appropriate response. For example, increased amino acid availability induces relocalisation of mTOR into proximity of Rag GTPases leading to its activation [@Sancak2008]. Furthermore, Glucose deprivation and oxygen availability regulate mTOR activity via adenosine-mono-phosphate kinase (AMPK) signalling. AMPK is activated when a shift in the cellular AMP to adenosine-tri-phophate (ATP) ratio is sensed [@Sanders2007;@Kimball2006]. While hypoxia (i.e. deprivation of oxygen) inhibits protein synthesis in normal cells, in breast cancer, protein synthesis was not inhibited during hypoxia which is attributed to uncontrolled mTOR signalling [@Connolly2006]. 

 Given the central of mTOR governing proliferation, growth and metabolism, which are often deregulated in cancer, it is vital to comprehensively understand mTORC1 signalling herein [@Hanahan2011].

\subsubsection{Global regulation of translation via mTOR}
 mTOR regulates mRNA translation mainly by regulating availability of eIF4E through phosphorylation of 4E binding proteins (4E-BPs)[@Gingras1999]. eIF4E is required for mRNA 5' cap binding and formation of the eIF4F complex. Therefore, inhibition of mTOR leads to a down regulation of cap dependent mRNA translation.
Other downstream targets of mTOR such as S6k has been shown to regulate components of the translation machinery, e.g. by activation of programmed cell death protein 4(PDCD4). PDCD4 competes for binding to eIF4G with eIF4A [@Dorrello2006;@Goke2002].  

\subsubsection{Selective or "mTOR sensitive" regulation of translation} 
 Selective or "mTOR sensitive" regulation of translation acts on transcripts with a terminal oligo pyrimidine motif (TOP mRNAs). This TOP motif consists of a C followed by a stretch of 4-15 pyrimidines directly after the 5' cap. TOP mRNAs show near complete dissociation from ribosomes under conditions when mTOR is inhibited and are  enriched for genes encoding for parts of the translation machinery [@Meyuhas2000; @Thoreen2012; @Yamashita2008]. Recent works indicate the importance of La ribonucleoprotein domain family member 1 (LARP1) in regulation of TOP mRNAs with contradictory findings [@Hopkins2016;@Jia2021;@Fonseca2015;@Maraia2017]. A panel of researches were asked to evaluate these findings which led to the establishment of a model for translational regulation via LARP1 [@Berman2021]. Herein, LARP1 is thought to bind to the 5' mRNA cap of TOP mRNAs via its DM15 domain and represses translation by obstructing eIF4E binding in instances where mTOR activity is abolished. In environments where mTOR is active phosphorylation of the DM15 occurs by mTOR causing it to release the 5' cap, wile the la domain of LARP1 remains bound to the mRNA thereby stabilizing it to facilitate translation [@Berman2021]. Other instances of selective translation are for mRNAs that lack the TOP motif, but show sensitivity to mTOR activity due to its role in eIF4F complex formation. 

\subsection{selective regulation through members of the eIF4F complex}
 As mentioned before, the eIF4F complex consists of eIF4E, eIF4A and eIF4G and is required for translation of all cap dependent mRNAs. However, characteristics of 5' UTRs introduce variation to how well mRNAs can be translated. Graff et al. proposed a model of strong and weak mRNAs. Herein, strong mRNAs are widely expressed and represent the majority of cellular mRNAs and are characterised by short unstructured 5' UTRs, e.g. \(\beta\)-actin. On the other hand, weak mRNAs show long and structured 5' UTRs and encode for potent growth and survival factors, e.g. c-Myc, ornithine decarboxylase (ODC1) and vascular endothelial growth factor (VEGF). Translation of "strong" mRNAs would remain effective in conditions where eIF4F complex availability would be limited. However, "weak" mRNAs show sensitivity eIF4F availability which is dependent on eIF4E expression [@Graff2008]. Elevated eIF4E expression is common in cancer and drives malignancy [@DeBenedetti2004]. 

 Studies investigated the effects of eIF4A inhibition on mRNA translation and the underlying characteristics of eIF4A dependent mRNAs. Therein, mRNAs that were senstitive to silvesterol treatment showed characteristics of "weak" mRNAs, i.e. long and structured 5' UTRs [@Wolfe2014;@Rubio2014;@Waldron2018]. In addition one third of eIF4A dependent mRNAs were encoded by genes with multiple 5' UTR variants, while for eIF4A independent mRNAs this was < 1% [@Rubio2014]. Translation of mRNAs with different 5' UTR variants has recently been implicated to drive cancer cell plasticity towards more "stem cell like" phenotypes during hypoxia [@Jewer2020].

 Recently, a more nuanced picture for translation of mRNAs sensitive to inhibition of components of the eIF4F complex has been shown [@Gandin2016a]. Therein, a comparison of treatments with an mTOR inhibitor (i.e. limiting eIF4F complex assembly) and silvesterol were evaluated leading to identification of mTOR-eIF4E and mTOR-eIF4A senstive mRNAs. Indeed, as shown before mTOR-eIF4A senstive mRNAs were characterised by long and structured 5' UTRs and endoced for pro surival proteins, however their translation was also dependent on eIF4E availability. mTOR-eIF4E sensitive mRNAs were characterised by short 5' UTRs and did not show dependence on eIF4A. Furthermore, these mRNAs encode for proteins related to metabolism.
 
\subsection{The integrated stress response} 
 The integrated stress response is a pathway which can be activated through kinases responding to various stress signals. These kinases include Protein kinase R-like endoplasmic reticulum kinase (PERK) which is activated by misfolded peptides in the endoplasmatic reticulum (ER), Heme regulated eIF2alpha kinase (HRI) which is activated during oxidative stress, protein kinase R (PKR) which is activated in response to certain viral infections by binding to double-stranded RNA (dsRNA) and general control nonderepressible 2 (GCN2) which is activated when cells are deprived of amino acids[@Kapur2017;@Guan2017;@Taniuchi2016;@Andreev2015;@Lemaire2005]. During the integrated stress response the alpha subunit of eIF2 is phosphorylated. Upon eIF2\(\alpha\) phosphorylation, eIF2\(\alpha\) directly engages the guanine nucleotide exchange factor eIF2B and prevents conversion of inactive eIF2-GDP to  active eIF2-GTP needed for met-tRNAi incorporation in the TC, therefore inhibiting translation by reducing PIC availability [@Sonenberg2009] (**see also figure \ref{fig:initiation}**).

\subsubsection{Global and selective regulation of translation via the ISR}

 Global and selective regulation of translation via the ISR is, similar to mTOR signalling, achieved at a global and selective level. Phosphorylation of eIF2 alpha limits ternary complex availability, therefore ribosome recruitment to the 5' cap is limited which results in a reduction of translation initiation. While global translation is reduced upon ISR, translation of a selective subset of mRNA with upstream open reading frames (uORFs) is increased. A uORF is a reading frame that originates in the 5' UTR of an mRNA upstream of which the AUG precedes that of the coding sequence. uORFs can be out of frame with the main ORF (mORF) and, when translated, lower the expression of the mORF [@Kozak1984]. ATF4, a transcription factor for stress response genes, contains multiple uORFs of which one partially overlaps with the mORF. Under normal conditions ATF4 mRNA translation is initiated at uORF1 and reinitiation at uORF2 occurs. The overlap of uORF2 with the mORF causes ribosomes to synthesise protein from uORF2 thereby inhibiting the translation of the coding sequence. Limitation of TC availability during ISR causes longer ribosome scanning times leading to that ribosomes scan past uORF2 and initiate at the mORF [@Pakos-Zebrucka2016].

 Ribosome profiling studies indicate that 50% of mammalian mRNAs harbour uORFs. mRNAs containing uORFs include oncogenes and transcripts important in differentiation and cell cycle [@Calvo2009;@Ingolia2011;@Morris1995]. The surrounding sequence of the uORF is important for initiation of translation. An AUG in the classical Kozak context (i.e. [A,G]..**ATG**G) is most efficient for translation initiation due to better recognition by the ribosome [@Kozak1986;@Calvo2009]. However, uORFs that do not contain AUG have been described as well. For example in Fli-1 two conserved uORFs have been described where one contains an AUG and the other GUG [@Sarrazin2000]. Translation of either uORF results in different proteins isoforms endoded in the same mRNA. Furthermore, other elements (e.g. structures) in proximity of the uORF can also influence its translation [@Ruan1994]. 

\subsection{Regulation of mRNA translation by tRNAs} \label{tRNA}
\begin{wrapfigure}{o}{0.6\textwidth}
  \includegraphics{./figures/tRNA.pdf}
  \caption{ (A) Schematic representation of the tRNA cloverleaf structure with indicated anti-codon and amino acid acceptor sites.  (B) Schematic representation of the proliferation and differentiation mRNAs dependent on distinct tRNA subsets. (C) Schematic representation of the U34 wobble position and the catalytic enzymes involved in this modification that is implied in tumor progression.
 \label{fig:tRNA}}
\end{wrapfigure}
 As touched upon earlier, tRNAs are an essential part of the translation machinery that carry the amino acids to the ribosome. tRNAs consist of a 76-90 long nucleotide sequence set into a "cloverleaf" structure forming several loops [@Sharp1985] (**see figure \ref{tRNA}**). The acceptor stem binds the amino acid carried by the tRNA, while the anti-codon loop binds to the mRNA within the ribosome via classical watson-crick pairing [@Watson1953]. Multiple codons can encode for the same amino acid (synonymous codons), however the availability of the tRNAs for different codons may vary which can influence elongation rates. 

 This supply ( i.e. tRNA availability) and demand (i.e. codon in expressed mRNA transcripts) relationship has been found to vary across different cellular states, e.g. proliferation and differentiation. Gingold et. al. observed two differen tRNA subsets; one induced under proliferation that is otherwise repressed and a subset with similar regulation under differentiation of which the supply matches the codon demand of the transcriptome [@Gingold2014]. This model has been disputed and it was proposed that the observed differences would be attributed to GC content in the mRNA [@Rudolph2016]. Nevertheless, abbarent tRNA expression and codon usage have been reported in cancer [@Zhang2018]. Furthermore, a comprehensive study including small RNAseq (i.e. for identification of tRNAs) and protein samples across 17 tissues obtained from the The Cancer Genome Atlas (TCGA) reported a tRNA signature stratified by Ki67 (a proliferation marker) staining with implications for patient survival [@Hernandez-Alias2020]. Therefore, while a consensus on proliferation specific tRNA subsets might not have been reached, emerging evidence implicates a role thereof in cancer [@Hernandez-Alias2020;@Gingold2014;@Zhang2018]. More specifically, increased expression of \(tRNA_{CCG}^{Arg}\) and \(tRNA_{UUC}^{Glu}\) has been observed in breast cancer cell lines and proposed to drive metastasis [@Goodarzi2016]. 

 Others report of a role of tRNAs in cancer attributed to tRNA modifications, specifically at the U34 anti- codon (or wobble) position which is highly conserved [@Rapino2017;@ElYacoubi2012]. Wobbling was proposed by Francis Crick and refers to the ability of non-watson-crick base pairing of tRNA anti codons [@Crick1966]. This enables a smaller set of tRNAs (41-55 in eukaryotes) to encode for the 64 possible codon combinations [@Goodenbour2006]. In mammals, the U34 modification catalytic cascade involves the acetyltransferase Elongator (ELP3), the methyltransferase TRM9-like domain of Alkylation repair homolog 8 (ALKBH8), and the urmylation (URM) pathway, that includes the cytosolic thiouridylase homolog 1 and 2 (CTU1/CTU2) [@Kalhor2003;@Karlsborn2014]. These enzymes ultimately modify the U34 modification into 5-methoxycarbonyl-methyl-2-thiouridine (\(mcm^5s^2U\)) which ensures cognate codon recognition. This modification is thought to occur in a small subset of tRNAs, namely  \(tRNA^{UUU}\), \(tRNA^{UUC}\), \(tRNA^{UUG}\), \(tRNA^{UCC}\), and \(tRNA^{UCU}\). 

 Loss of the ability to modify U34 has been shown to reduce translation elongation rates with varying effects on protein expression [@Nedialkova2015;@Deng2015;@Zinshteyn2013]. While in some cases U34 dependent signalling led to ribosome stalling resulting in protein aggregates and increased stress [@Nedialkova2015;@Zinshteyn2013], others reported a subtle dowregulation of proteins encoded by mRNAs requiring U34-modified tRNAs [@Deng2015]. U34 modification dependent tRNAs have been shown to play a role in cancer. For example, ELP3 is important in tumor initiation in the intestine and promotes breast cancer invasion as well as progression to metastasis [@Ladang2015;@Delaunay2016]. Furthermore, loss of U34 modification in a prostate cancer model lead to an adaptive transcriptional response for mRNAs whose translation was dependent on the U34 modification [@Lorent2019].

\subsection{RNA binding proteins and trans-acting factors}
 The UTRs of an mRNA contain sequence elements to which small RNA and RNA binding proteins (RBPs) bind and exert translational regulation. For instance, microRNAs (miRNA), a small class of non coding RNA. The precise role and mechanisms in regulation of translation of miRNAs is still under active investigation [@Oliveto2017]. However, they can directly bind to other RNAs and silence them accomplished through translational repression or destabilisation [Jonas2015]. Regulation of gene expression by miRNAs has been observed in cancer where miRNAs have been implicated to promote tumorgenesis or act as tumor suppressors [@Sampson2007;@Muniyappa2009;@Tian2010;@Nagpal2015].

 RBPs are a class of proteins involved in many regulatory steps of gene expression and account for ~7.5% of the protein coding genes. The poly-A-binding-protein (PABP) is a RBP involved in mRNA translation initiation. PABP is thought to form a closed loop complex of the 3' end to the 5' by interacting with eIF4G. This closed loop is proposed to promote translation and prevent mRNA decay [@Afonina2014;@Amrani2008] (**see also figure \ref{fig:initiation}**). Another important RBP implicated in regulation of translation is Human antigen R (HuR). HuR preferentially binds to AU-rich sequences in the 3' UTR and acts as a stabilizing agent and is involved in RNA-processing [@Levine1993;@Baou2011;@Fan1998;@Peng1998]. Furthermore, a study in adenocarcinoma implicates that HuR inhibits translation of proapoptotic factors [@Winkler2014]. Other studies in breast, colon and lung cancer observed correlation between HuR and malignancy. Among its targets are hypoxia inducible factor 1 (HIF-1), VEGF and c-Myc which are known drivers for tumor progression [@Denkert2004;@LopezdeSilanes2003;@LopezdeSilanes2005].
\newline
\section{Expertimental methods to measure mRNA translation} \label{exptMethod}
\begin{wrapfigure}{r}{0.6\textwidth}
    \includegraphics{./figures/polyRibo.pdf}
  \caption{Polysome profiling and ribosome profiling workflows. In polysome profiling a fraction from whole cytoplasmic RNA is loaded onto a sucrose gradient on which they get separated by sedimentation using ultra centrifugation. Fractions corresponding to efficiently translated mRNAs are collected and can be quantified with for example RNA sequencing (left). During ribosome profiling a fraction from the whole cytoplasmic RNA is exposed to a digestion agent which disturbs the RNA. The ribosomes will protect fragments thereby creating ribosome protected fragments. These fragments are then isolated and can be sequenced.  \label{fig:polyRibo}}
\end{wrapfigure}
 Methods that measure mRNA translation try to capture the number of ribosomes an mRNA is associated with on the principle that this is directly correlated with their translation efficiency. This is under the assumption that elongation rates across pools of mRNAs are similar which is supported by ribosome kinetic studies [@Ingolia2011]. Two methods are predominantly used  for measuring mRNA translation, or changes in translation efficiencies across conditions, namely polysome profiling and ribosome profiling (**see figure \ref{fig:polyRibo}**). The methods will be explained below in further detail.


\subsection{Polysome profiling} 
 Polysome profiling is a technique to measure changes in translational efficiencies of mRNAs between two or more conditions. Polysome profiling allows for separation of polysomes from monosomes, ribosomal subunits and messenger ribonucleoprotein particles (mRNPs). During the assay, ribosomes are immobilized on the mRNAs using translation elongation inhibitors (e.g. cycloheximide (CHX)). A portion of cytoplasmic RNA extracts then sediment on a linear sucrose gradient (5-50%) using ultra centrifugation. The resulting gradient is fractionated and mRNAs with different number of bound ribosomes can be extracted and analyzed for changes in translational efficiency [@Gandin2014]. Typically fractions belonging to mRNAs with more than 3 bound ribosomes are pooled. A 3-ribosome cut off has been chosen as it is thought to capture most biologically relevant changes in translation efficiency [@Gandin2016a].
\begin{wrapfigure}{r}{0.6\textwidth}
    \includegraphics[width=0.9\linewidth]{./figures/polysome_shifts.pdf}
  \caption{Polysome profliles -  (top left) Schematic representation of a polysome profile using linear sucrose gradient fractionation. Indicated in the polysome profiles are the 40S, 60S ribosomal subunits as well as the 80S monosome. H.P. indicates heavy polysome fractions.Between conditions (i.e. black an pink lines) distribution changes for mRNA abundance (grey and green; top right), translation (grey and red; bottom left) and translation within high polysome fractions (grey and red; bottom right) are illustrated. \label{fig:polysome}}
\end{wrapfigure}
An illustration of a polysome profile with peaks for the 40S, 60S subunits and 80S ribosome can be seen in (**Fig \ref{fig:polysome} top left**). Subsequent peaks along the frations indicate the mRNAs with 2 or more bound ribosomes. mRNAs are typically normally distributed along the fractions, i.e. in a pool of the same mRNA many will be associated with different numbers of ribosomes [@Gandin2016a]. Changes in mRNA abundance will lead to an overall increase in the amount of isolated polysome-associated mRNA without a shift of the distribution along the fractions (**Fig \ref{fig:polysome} top right**). This means that the translation efficiency per mRNA remains unchanged. Changes in translational efficiency  can be observed by shifts of polysome association for mRNAs from the light (inefficiently translated) towards the heavy (efficiently translated) polysome fractions or vice versa (**Fig \ref{fig:polysome} bottom left**). Shift within the heavy polysome fractions (i.e. with a mean distribution around 4 bound ribosomes to 7 bound ribosomes) can also occur (**Fig \ref{fig:polysome} bottom right**). Quantification of mRNA levels within each fraction can be assessed using Northern blotting or reverse transcription quantitative polymerase chain reaction (RT-qPCR). For transcriptome wide studies quantification is done using either DNA-microarrays or RNA sequencing. 

 Pooling of mRNAs as well as collection of multiple fractions makes polysome profiling inconvenient when dealing with large samples sizes or experiments with low amounts of input RNA. Therefore, an optimized sucrose gradient was developed where mRNAs bound to >3 ribosomes are collected on a sucrose cushion and thereby can be isolated from one single fraction [@Liang2018]. This optimized gradient allows for application of polysome profiling in small tissue samples where RNA quantity is limiting and reduces labor intensity of the assay. 

 Polysome-associated mRNA levels are subject to changes in translation efficiency as well as factors contributing to cytosolic mRNA levels that impact to pool of mRNAs that can associate with ribosomes e.g. transcription and mRNA stability.  Therefore, to identify true changes in translation efficiency it is important to collect cytoplasmic mRNA and polysome-associated mRNA from the same sample in parallel to correct for such mechanisms during downstream analysis. 

\subsection{Ribosome profiling} \label{riboseq}
 Ribosome profiling is a technique that enables sequencing of ribosome protected mRNA fragments (RPFs) [@Steitz1969]. In the assay ribosomes are immobilized on the mRNAs using, similar to polysome profiling, translation elongations inhibitors (e.g. CHX) [@Ingolia2009;@Ingolia2016]. One limitation with the use of translation elongation inhibitors is the distortion of ribosome distributions especially at translation initiation sites [@Gerashchenko2014;@Hussmann2015]. For example, treatment with cyclohexamide can affect identification of uORFs in ribosome profiling experiments [@Spealman2018]. These introduced artefacts need to be accounted for in the downstream analysis when assessing ribosome position along the mRNA. 

 RPFs are obtained by RNAse treatment that degrades the links of RNA between ribosomes leaving single ribosomes with a ~28 nucleotide long RNA fragment within each ribosome. However, as ribosomal RNAs (rRNA) is also degraded during this process they represent a big fraction herein. The RPFs are then isolated using ultra centrifugation through a sucrose cushion. During this step other RNA fragments such as rRNAs,  non-coding RNAs (ncRNAs) or large ribonucleoprotein complexes can co-migrate and contaminate the sample. Typically RPFs with a size ranging from 25-30 nucleotides are selected for quantification. However, among these sizes could be RNA protected by RBPs that have no ribosome associated to it. Furthermore, ribosomes undergoing conformational changes have been shown to protect fragments corresponding to a length of 21nt only when translation elongation is blocked by e.g. CHX [@Lareau2014]. Therefore, size selection can distort the estimation of translation efficiency from ribosome profiling data [@Andreev2017]. From the resulting RPFs libraries can be constructed and quantified using RNA sequencing. During library construction additional biases due to enzyme sequence preferences can be introduced that can lead wrong estimations of codon positions within the ribosome [@Artieri2014a].

 Initially, fragmented total mRNA using alkaline hydrolysis of the same size were retrieved in paralell to RPFs. This was achieved by extraction of total mRNA from cell lysate followed by purification via recovery of polyadenylated messages or removal of ribosomal RNA [@Ingolia2009;@Brar2015]. The random fragmentation of total mRNA has been shown to underlie experimental bias. Therefore, now unfragmented total mRNA is sequenced in parallel.

\subsection{Comparing ribosome and polysome profiling}
 Albeit both methods generate count data after quantification with RNA sequencing, there are some key aspects that differ between the techniques. Polysome profiling separates efficiently translated mRNAs from non- efficiently translated mRNAs along a sucrose gradient thereby creating an mRNA based perspective for analysing changes in translational efficiencies. In contrast, ribosome profiling determines translational efficiencies by counting the number of RPFs of both efficiently and non-efficiently translated mRNAs. This can have implications for identifaction of transcript variants. For example  If a ribosome would not protect a fragment spanning a variant the information would be lost during ribosome profiling [@Floor2016]. Whereas in polysome profiling the quantification is based on the whole mRNA. This gives polysome profiling the advantage in cases of transcript variants with different 5' UTR lengths.

 Changes in translational efficiencies, e.g. shifts in ribosome association, can be dramatic (i.e. near complete dissociation of ribosomes from an mRNA) or subtle (shifts from e.g. 2 to 4 ribosomes) and could be due to different properties of the mRNAs [@Gandin2016a]. When these two changes in translation efficiencies are present in parallel, ribosome profiling is biased towards identification of dramatic shifts and masks the subtle ones, leading to false biological conclusions [@Gandin2016a]. This has been attrituted to the indirect estimation of translation efficiencies from counting RPFs for ribosome profiling, which is highly dependent on the abundance of mRNAs. In polysome profiling  this effect is much less pronounced as changes in translation efficiency are directly estimated from the mRNAs associated with heavy polysomes [@Masvidal2017]. Higher sensitivity in detecting changes in translational efficiencies on a global scale makes polysome profiling more suitable for transcriptome-wide studies [@Gandin2016a;@Masvidal2017].

 An advantage of ribosome profiling is that it provides exact nucleotide positions occupied by ribosomes. This offers information at a single nucleotide level where the ribosome sits. Polysome profiling cannot reveal ribosome locations along the mRNA. The single nucleotide resolution of ribosome profiling is necessary in contexts studying local translation events such as ribosomal frame shifts [@Rato2011] or uORF translation [@Andreev2015]. 

 Both methods have their strengths and weaknesses and therefore each method should be considered depending on the underlying biological question.
\newline
\section{Modes for regulation of gene expression through mRNA translation} \label{modes}

From transcriptome-wide assessments of translation using ribosome or polysome profiling expression levels are obtained for both cytoplasmic and polysome-associated mRNAs (or RPFs). For simplicity, from now on, these RNA types will be referred to as total mRNA (i.e. cytoplasmic mRNA) and translated mRNA (i.e. polysome-associated mRNA or RPFs). The estimation of expression levels for both translated mRNA and total mRNA allows for interrogation at two steps of the gene expression pathway and their interaction. The interplay of total mRNA with translated mRNA can give valuable insights about the underlying mechanisms that govern gene expression in the studied system.
\begin{wrapfigure}{l}{0.6\textwidth}
  \includegraphics{./figures/geneModes_MRNA.pdf}
  \caption{Regulatory modes of gene expression - Schematic representation of regulatory modes of translation efficiency in a fold-change scatter plot. Indicated in red are changes in translation (i.e. changes in translated mRNA but not total mRNA), in green changes in mRNA abundance (i.e. congruent changes between total mRNA and translated mRNA) and in blue translational buffering (i.e. changes in total mRNA levels but not translated mRNA levels). TE changes as the TE-score would estimate them are indicated.\label{fig:modes}}
\end{wrapfigure}
 When comparing perturbed systems to their corresponding control state we typically observe three "modes" in which translated mRNA and total mRNA distinctly interact that impact gene expression (**See figure \ref{modes}**). We refer to these modes as "translation", "mRNA abundance" and "translational buffering". 

\subsection{Translation} 
 A change in translation occurs when translated mRNA levels either increase or decrease, however corresponding total mRNA levels remain constant or change to a lesser extent resulting in a change of their translation efficiency. A prominent example of this mode can be observed for TOP mRNAs. These mRNAs, under conditions when mTOR is inhibited, show a near complete dissassociation from ribosomes [@Gandin2016a]. mRNAs under the translation mode are expected to reshape the proteome (**See figure \ref{modes}**). 

\subsection{mRNA Abundance} 
 Another mode that impacts the proteome is a change in mRNA abundance, here the translated mRNA level changes to a similar magnitude as the total mRNA level. For these mRNAs the translation efficiency is unaltered, as the change in total mRNA levels explains the change in translated mRNA levels. Nevertheless, as there are more (or less) mRNAs being translated an effect the proteome is expected. The underlying biological implication for this mode mRNAs is often related to transcription or mRNA stability (**See figure \ref{modes}**).

\subsection{Translational buffering} \label{modeBuffering}

The third mode for regulation of gene expression is termed translational buffering. Under this mode a change in total mRNA levels is observed, whereas polysome-associated mRNA levels remain constant between conditions. Translational buffering also reflects a change in translation efficiency as the proportion of mRNAs to associated with ribosomes is altered. In contrast to changes in translation and mRNA abundance, translational buffering has been shown to maintain protein homeostasis rather than reshape the proteome [@Lorent2019;@Lalanne2018;@McManus2014].

Currently the literature supports multiple forms of translational buffering. Translational buffering can compensate for differences in mRNA levels due to e.g. differences in gene dosages, so that protein levels remain similar [@Dassi2015;@Lalanne2018]. Furthermore, rather than compensating it can "offset" translation to temporarily alter the mRNA:protein ratio in response to a stimulus [@Lorent2019]. 

Translational buffering in its compensation form has been observed at steady state levels. A study comparing co evolution of transcription and translation across seven different organs and mammals showed overall greater divergence of transcription as compared to translation [@Wang2020]. Similar compensation at the level of translation has been observed between individuals, tissues and prokaryotes [@Cenik2015;@Perl2017;@Lalanne2018;@Dassi2015;@Artieri2014].

Compensation via translational buffering can also enforce equilibration of pathway or protein complex stoichometry [@Lalanne2018;@Li2014]. An example of this was observed in  evolutionary distant bacetria, i.e. B. subtilis and E. coli. In B. subtilis translation related factors rpsP and rplS are located in different operons, whereas in E. coli they lie within an operon together with rimM and trmD. While B. subtilis can fine tune transcription at the different operons, in E. coli these will be transcribed together. However, rimM and trmD are only needed in low protein abundance, whereas rpsP and rplS are required in high abundance. E. coli compensates the transcriptional input at the translational level and thus equilibriates for requirements in pathway stoichometry [@Lalanne2018].
 
As mentioned above a different form of translational buffering can be observed  in perturbed systems that offset changes in transcription at the level of translation temporarily. For example, translational offsetting was observed in prostate cancer cells where a transcriptional program was induced under estrogen receptor \(\alpha\) (ER\(\alpha\)) depletion that was showed no increase in polysome-associated mRNA. mRNAs whose transcription was translationally offset required the tRNA u34 modification. ER\(\alpha\) has been shown to regulate the expression of the catalytic enzymes required for the u34 modification [@Lorent2019]. Thus, depletion of ER\(\alpha\) led to that tRNAs could not be properly modified at the u34 position and therefore the translation efficiency of mRNAs requiring the modification was reduced despite increased total mRNA levels. 
\newline
\section{Algorithms for analysis of changes in translation efficiencies}\label{algorithm}

 Given multiple roles of mRNA translation to regulate the proteome via translation, mRNA abundance and buffering it is critical to distinguish them as their underlying mechanisms can have different biological implications. In this section we will discuss methods that analyse polysome-profiling and ribosome profiling data to estimate changes in translation efficiencies across 2 or more conditions and how these methods identify different modes of gene expression.

 Initially analysis of transcriptome-wide translation studies used an approach called the translation efficiency score (TE-score) that uses the following equation: 
\[\varDelta TE = \frac{\frac{P_{c2}}{T_{c2}}} {\frac{P_{c1}}{T_{c1}}}\\\]

This score is calculated the ratio of the ratios between polysome-associated mRNA levels (P) divided by total mRNA levels (T) within each condition (i.e. C1 and C2). The TE- score approach has been shown to be prone to spurious correlations [@Larsson2010]. Spurious correlations arise due to that the ratio of polysome-associated mRNA and total mRNA can systematically correlate with total mRNA levels which is not corrected for in this equation and leads to an elevated type-1 error.

 **Figure \ref{fig:TE}** gives an overview of the relationship between a change in  TE-score and each gene expression mode (**see also figure \ref{fig:modes}**). Changes in mRNA abundance will lead to a \(\varDelta\)TE close to 0 in log space (i.e. no change in translation efficiency) as total mRNA and translated mRNA change with a similar magnitude. However, in the case of both translation and translational buffering, the nominator and denominator in the TE-score equation change leading to a \(\varDelta\)TE (TE < 0 or TE > 0) and thereby identification of both changes in translation and translational buffering simultaneously. Therefore, the TE-score method fails to differentiate between changes in translation and translational buffering which can have drastic consequences for the biological interpretation of the results [@Oertlin2019] (**see also section \ref{modes}**).
\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_TE.pdf}
  \caption{TE scores for regulatory modes of gene expression -  Schematic representation of regulatory modes of translation efficiency in a fold-change scatter plot. Indicated in red are changes in translation efficiency altering protein levels, in green changes in mRNA abundance and in blue changes in translation efficiency leading to translational buffering/offsetting. The shifts for the translation efficiency (TE) score are indicated. \label{fig:TE}}
\end{wrapfigure}
 The TE-score approach was questioned when proposing the Analysis of Translation Activity (anota) algorithm which was developed for DNA-microarray data [@Larsson2010]. anota combines analysis of partial variance (APV) [@Schleifer1993] with a random variance model (RVM) [@Wright2003]. RVM estimates gene variance using shared information across all genes to increase power for detection of differential expression [@Wright2003]. anota uses a two-step process that firstly assesses the model assumptions for (i) absence of highly influential data points, (ii) samples classes sharing a common slope, (iii) homoscedasticity of residuals and (iv) normal distribution of per gene residuals. In the second step anota performs analysis of changes in translational activity using the following model:

\[y_{gi} = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 
  
here \(y_{gi}\) is the polysome associated mRNA expression, \(\beta_g^{RNA}\) describes the relationship to total RNA for the \(gth\) gene of the \(ith\) sample column of model matrix \(X\); \(\beta_g^{cond}\) represent the difference in intercept between treatment classes and \(\varepsilon_{gi}\) denotes the residual error. 

\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_anota_Larsson.pdf}
  \caption{anota gene models - Schematic representation of the anota analysis models. Translation mRNA expression is set out against total mRNA expression for each biological replicate and treatment condition. Top left shows the model of a gene that is deferentially translated (i.e. change in translated but not total mRNA). The difference in the slope intercepts are used to estimate changes in translation efficiencies between conditions i.e. dTE. Other gene models are shown; change in translation efficiency with varying total mRNA levels (top right); change in mRNA abundance (bottom left) and translational buffering (bottom right).
  \label{fig:anota}}
\end{wrapfigure}

 Within anota a common slope for the treatment classes that describes the translated mRNA to total mRNA relationship is calculated. The difference between the slope intercepts is then interpreted as the change in translation efficiency. A simplified view of this model can be seen in (**Figure \ref{fig:anota} top left**). Here expression for translated mRNA and total mRNA are modeled over two sample classes with each 4 replicates. Furthermore, changes in translation efficiencies can also be observed when translated mRNAs shift to a larger extent than the total mRNA levels (**Figure \ref{fig:anota} top right**). Identification of genes in this category can be a challenge, especially in highly variable data sets, as they resemble mRNA abundance genes (**Figure \ref{fig:anota} bottom left**). Nevertheless, using the linear regression analysis anota accurately corrects changes in translated mRNA as can be seen in (**Figure \ref{fig:anota} bottom right**) where a change in total mRNA   but not translated mRNA levels is observed. For this gene the difference in slope intercepts between sample classes is small and will not be identified as differentially translated as would be the case in the TE-score approach. anota was developed at a time where translational buffering was not considered in data sets. Naturally, the methods lacks a setting to analyse translational buffering. This was addressed in anota's successor, anota2seq, and will be discussed in **Study 1**.

 Advances in experimental methods warrant for appropriate statistical methods to analyse data resulting from them. DNA-microarray was the dominant platform to assess transcriptome-wide changes before the advent of RNA sequencing. DNA-microarrays measure intensity after hybridisation events is an indicator of expression, whereas in RNA sequencing reads of constructed libraries are counted. Intensity data from DNA-microarray can be normalised and transformed (i.e. log transformation) to fulfill the requirements for application of linear models, whereas RNA sequencing harbours additional characteristics that need to be accounted for. Therefore, algorithms developed for analysis of DNA- microarray are not directly applicable to RNA sequencing data as is the case for the anota algorithm.

 RNA sequencing data shows variance that is greater than the mean which is commonly referred to as overdispersion. Count data from RNA sequencing have been initially approached using Poisson distributions which assumes that the variance is equal to the mean [@Lu2005]. Now established RNA sequencing analysis frameworks such as edgeR and DESeq2  use negative binomial distributions in combination with generalised linear models (GLMs) [@Robinson2010; @Love2014]. The negative binomial distribution uses a dispersion parameter to account for differences in the mean-variance relationship across the expression range [@McCarthy2012]. While analysis principles of DESeq2 and edgeR are similar they differ in their normalisation method, dispersion estimation and information sharing across genes. In a simple differential expression analysis between two conditions with one RNA type the GLM model would be as in the following equation: 

\[log(y_{gi}) = \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 

here \(\beta_g^{cond}\ X_i^{cond}\) represent the condition (i.e. control and treatment) log2 fold change for the \(gth\) gene \(ith\) sample column of the model matrix X and \(\varepsilon_{gi}\) denotes the residual error. When analysing changes in translation effiencies additional parameter for RNA type (i.e. total mRNA or translated mRNA) and the interaction between the RNA type and condition are added so that:

\[log(y_{gi}) = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \beta_g^{RNA:cond}\ X_i^{interaction} + \varepsilon_gi\] 

 In this model the interaction term is interpreted as the change in translation effiencies [@Chothani2019]. Other methods (i.e. Ribodiff [@Zhong2017], Riborex [@Li2017] and deltaTE [@Chothani2019]) borrow this analysis principle of an GLM with an interaction term by applying this exact model. A notable difference is that Ribodiff allows dispersion estimation for translated mRNA and total mRNA separately as variance differences between the RNA types can be expected due to varying experimental protocols [@Zhong2017; @Liang2018]. While the flexibility of GLMs allows for complex study designs involving 2 or more treatment conditions, Riborex and Ribodiff limit the study design to only two conditions. DeltaTE gives their users full flexibility of the DESeq2 GLM model. Xtail is a method developed for ribosome profiling that makes use of DESeq2 for RNAseq count normalisation [@Xiao2016]. Their assessment of differences in translation efficiencies relies on probability matrices for the ratio of translated mRNA over total mRNA within condition and a between condition ratio of these ratios. Babel was the first algorithm designed solely for analysis of differential translation and uses an error-in-variables regression analysis [@Olshen2013]. The error-in- variables regression allows accounting for variable total mRNA levels when assessing changes in translation.  Although these methods have distinct approaches to identify changes in translation efficiencies, their principle of analysis is similar to comparing a ratio of ratios (see TE-score equation above). Therefore these methods suffer from similar issues as the TE-score which will be discussed in **Study 1**.


