---
output: pdf_document
---

\chapter{Introduction}
\section{Gene expression}
\subsection{The central dogma of gene expression}
  \begin{wrapfigure}{r}{.6\textwidth}
  \includegraphics{./figures/geneExprPath_2.pdf}
  \caption{The gene expression pathway - DNA is transcribed into RNA. RNAs are processed into mRNAs that consist of a 5' cap, exons and a poly(A) tail. mRNAs can be transported out of the cellular nucleus into the cytoplasm where they can be degraded, stored or translated into proteins depending on cellular demands. Synthesised proteins can be degraded by proteosomes. Coloured boxes in the DNA refer to introns (teal) and exons (purple). Introns are non-coding parts of the genome, whereas exons encoding regions. mRNAs are depicted as purple lines, i.e. a series of exons, with an AAAAA extension and an oval shape at its start. Proteins are depicted as a series of pink balls.  \label{fig:geneExprPath}}
\end{wrapfigure}
In eukaryotic cells, genetic code is stored as deoxyribonucleic acid (DNA) molecules in the nucleus. Transcription is the process whereby temporary copies of the DNA are generated and occurs in the nucleus. These copies are called ribonucleic acid (RNA). RNAs undergo processing by which multiple different variants coming from the same gene are produced. A subset of protein-encoding processed RNAs is the so called messenger RNA (mRNA). mRNAs are transported from the nucleus into the cytoplasm where they can be stored, degraded or translated into proteins. Proteins themselves can also be degraded (*see figure \ref{fig:geneExprPath}*). This flow of genetic information into expressed proteins is commonly referred to as the central dogma in molecular biology [@Crick1970].
\clearpage
\subsection{Contribution to gene expression}
 Proteins are the last product of the gene expression pathway and carry out the vast majority of all cellular functions. While it is apparent that modulation of protein levels will offer information on the changes in gene expression, it cannot completely answer the question as to why the protein levels change. In a disease context, protein levels alone might only offer sufficient insight to explain phenotypic differences. Studying the mechanisms that drive differences in protein levels is thus required to improve understanding of biological processes and their dysregulation in disease.

Recently developed system biology methods allow investigation of gene expression at multiple levels on a genome-wide scale. Initially, transcriptomics studies were applied to study gene expression with the assumption that mRNA expression is the main determinant for protein levels and therefore changes in transcipt abundance may be used as a proxy for alterations in the proteome. However, this view was challenged by several landmark studies that observed a poor mRNA to protein correlation and indicated a larger role of post-transcriptional regulation in gene expression than previously assumed [@Lu2005; @Vogel2012; @deSousaAbreu2009; @Schwanhausser2011; @Silva2016].
  
 The debate regarding which step of the gene expression pathway contributes most to the composition of the proteome is ongoing, nevertheless an understanding has been reached that the context under which studies are carried out is a major determinant. At steady state, mRNA levels seem to explain protein abundance best, however in perturbed systems (e.g. under stress or growth factor signalling) the contribution of transcript abundance appears to have less impact relative to post-transcriptional steps in regulation of gene expression [@Liu2016].  Moreover, it appears that correlation between mRNA and protein levels also depends on the type of the stimulus. For example, in a study that stimulated bone marrow-derived dendritic cells with LPS, protein levels were dependent on cellular transcript levels [@Jovanovic2015]. In contrast, a study investigating cells under endoplasmatic reticulum stress observed extensive modulation of protein levels, whereas mRNA abundance was only mildly affected [@Cheng2016].
  
 Thus, the contribution of different steps of the gene expression pathway is dependent on many different factors, e.g. cellular state or treatments. Nevertheless, mRNA translation is an essential process in determining composition of the proteome. Furthermore, dysregulation of mRNA translation has been observed in multiple diseases, ranging from neurological disorders to cancer which warrants for a comprehensive understanding of this process [@Kapur2018; @Ruggero2013; @Lee2021; @Graff2009; @Tahmasebi2018]. This thesis focusses on increasing understanding of the role of mRNA translation in cancer.
\newline
\section{mRNA translation}
\subsection{Schematic representation of mRNA}
mRNAs contain a protein coding region which is flanked by untranslated regions (5' and 3' UTRs). UTRs contain post-transcriptional regulatory elements that may affect localisation, stability and translation of the mRNA [@Loya2008; @Leppek2018; @Mignone2002] (*see figure \ref{fig:UTRFeat}, see also section \ref{regmRNA} for details*). 
 
\begin{figure}[H]
  \includegraphics{./figures/UTRFeatures_2.pdf}
  \caption{ Schematic overview of mRNA - mRNA consists of a coding sequence, 5' and 3' untranslated regions flanking the coding sequence, a 5' cap and a poly-(A)-tail. Located within the 5' and 3' untranslated region are post-transcriptional regulatory elements that can influence gene expression. uORF; upstream open reading frame; IRES, internal ribosome entry site; CPE, cytoplasmic polyadenylation site; AAUAAA, polyadenylation signal. See section \ref{regmRNA} for details on these elements.
 \label{fig:UTRFeat}}
\end{figure}

The 5' end of mRNAs contain a 7-methyl-guanylate (m7G) cap that is important for translation initiation, while the 3' end has a poly-A tail protecting the mRNA against degradation [@Grifo1983;@Wilusz2001]. Multiple different mRNA variants (also called isoforms) from the same gene can exist. Isoforms may arise due to alternave transcription start site usage or a process called alternative splicing. Isoforms can co-exist at the same time and have differing properties that can perform distinct functions [@JolyAnne-Laure2018]. Furthermore, alternative splicing may result in isoforms with different 5' UTRs. This leads to altered translation of mRNAs encoding for the same protein [@Jewer2020;@Floor2016].

\subsection{The steps in mRNA translation} \label{translation}
  In eukryotes, mRNA translation occurs in the cytoplasm for the vast majority of protein coding mRNAs. However a small subset of mRNAs encoded by mitochondrial DNA is translated in the mitochondria [@DSouza2018]. mRNA translation is a process that consits of several steps: initiation, elongation, termination and ribosome recycling (*see figure \ref{fig:doodlemRNASteps}*). These steps will be discussed in detail below.

\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/doodleTranslation.pdf}
  \caption{mRNA translation initiation, elongation, termination and ribosome recycling steps - The ribosome binds to the mRNA and initiates scanning for a start codon (AUG). The elongation phase incorporates amino acids into a polypeptide chain (i.e. the protein product). Once a stop codon (e.g. UGA) is detected, the ribosome terminates translation and releases the polypeptide chain. The ribosome can then be recycled to participate in the translation of another mRNA or reinitiate on the same mRNA. Green ovals represent the ribosomal subunits. Within the ribosome the E, P and A-sites are indicated with yellow boxes (see section \ref{elongation} for details). Polypeptide chains are depicted as a series of pink balls. Orange crosses represent transfer RNAs (tRNA) (see section \ref{elongation} and \ref{tRNA} for more details on tRNAs).  \label{fig:doodlemRNASteps}}
\end{wrapfigure}
\clearpage

\subsubsection{Initiation} \label{initiation}

In eukaryotes, 5' cap-dependent initiation of mRNA translation consists of multiple stages. First, a ternary complex (TC) consisting of guanosine-tri-phosphate (GTP) bound eukaryotic initiation factor (eIF) 2 and methionine-initiator transfer RNA (met-tRNAi) is formed. This is followed by formation of the 43S pre-initiation complex (PIC). The PIC consists of a 40S ribosome subunit, the TC and, translation iniation factors eIF1, eIF1A, eIF3 and, eIF5 [@Asano2000]. mRNAs then undergo "activation". Here, the 5' cap proximal structure is bound by eIF4F. eIF4F is the 5' cap binding complex consisting of: the 5' cap binding protein eIF4E, the RNA helicase eIF4A and, a scaffold protein eIF4G [@Grifo1983]. The Poly (A) binding protein (PABP) binds to the poly(A) tail of the 3' UTR and causes circularisation of the mRNA. The circularisation improves stability of the mRNA and aids in recruitment of translation initiation factors [@Ivanov2016]. Upon mRNA recruitment, the ATP-dependent eIF4A helicase activity of the eIF4F complex facilitates scanning of 43S PIC along the 5'UTR together with eIF4B and eIF4H. Recognition of the translation initiation codon (AUG) induces formation of the 48S initiation complex. Once the initiation codon (AUGi) is reached, displacement of eIF1 occurs which allows eIF5 to hydrolyse eIF2-bound GTP. The 60S ribosomal subunit then joins the 40S ribosomal subunit which causes the release of eIF2-GDP and other initiation factors (eIF1, eIF3, eIF4B, eIF4F and eIF5). Subunit joining is mediated by eIF5B. After subunit joining the 80S ribosome is formed and the elongation process starts (*see figure \ref{initiation}*) [@Asano2000; @Hinnebusch2006; @Jackson2010]. A mechanism for cap dependent translation initiation that is independent of eIF4E has been described. Here, eIF3d binds the 5' cap of mRNAs containing an RNA element that blocks eIF4E binding [@Lee2015,@Lee2016]. Next to 5'cap-dependent initiation, mRNA translation can also be initiated in a 5' cap-independent manner, e.g. via the internal ribosome entry site (IRES) (*see figure \ref{fig:UTRFeat}*). Mechanisms for 5' cap-independent mRNA translation initiation are extensively reviewed elsewhere [@Shatsky2018;@Lacerda2017]. 

\subsubsection{Elongation} \label{elongation}
The 80S ribosome contains three sites important for decoding an mRNA: the aminoacyl (A), peptidyl (P) and exit (E) sites. During elongation in eukaryotes, aminoacytelated tRNAs are delivered to the A-site in a ternary complex with eukaryotic elongation factor (eEF) 1A and GTP. If the tRNA is cognate to the codon in the A-site of the ribosome, GTP bound to eEF1A is hydrolysed. This causes eEF1A to release and accommodates the aminoacetylated tRNA in the A-site. This is followed by a peptidyl transferase reaction. This reactions forms the peptide bond between the peptidyl tRNA in the P-site and amino group of the aminoacyl-tRNA in the A-site (aa-tRNA) and is catalyzed by 60S ribosomal subunit. After the peptide bond is formed a translocation step occurs through eEF2-GTP hydrolysis. The translocation step encompasses that the deacetylated tRNA currently in the P-site moves into the E-site. Likewise the peptidyl tRNA in the A-site moves to the P-site. Which leaves the A-site open for other aa-tRNAs. The deacytelated tRNA in the E-site is then released from the ribosome [@Dever2012]. This process is repeated until a stop codon (UAA, UGA or UAG) enters the A-site of the ribosome.
\begin{figure}[ht]
\centering
\includegraphics[width=1.0\linewidth,height=0.8\textheight]{./figures/initiation.jpg} 
  \caption{Pathway of eukaryotic translation initiation via ribosomal scanning. Reprinted with permission. Cold Spring Harb Perspect Biol. 2012 Oct; 4(10): a011544.doi: 10.1101/cshperspect.a011544.© 2012 Cold Spring Harbor Laboratory Press.
  \label{fig:initiation}}
\end{figure}

\clearpage

\subsubsection{Termination and recycling} 
\begin{wrapfigure}{r}{.4\textwidth}
  \includegraphics{./figures/polysome.pdf}
  \caption{Electromicrongraph of ribosomes extracted from different positions along a sucrose gradient used for polysome fractionation (A-C). Multiple ribosomes bound to the same mRNA can be observed. For details on polysome fractionation see section \ref{exptMethod}. Reprinted with permission. DR. T. STAEHELIN et. al. Nature.1963 Aug 31;199:865-70.doi: 10.1038/199865a0. Copyright © 1963, Nature Publishing Group.
 \label{fig:polysomes}}
\end{wrapfigure}
 mRNA translation termination is  facilitated  by  two  eukaryotic release  factors (eRF), eRF1  and  eRF3-GTP [@Stansfield1995;@Alkalaeva2006]. The  eRF1:eRF3-GTP complex binds to the A-site of the ribosome upon recognition of a stop codon. This causes an hydrolysis event resulting in a conformational change and release of the polypeptide chain. eRF1 and the ATP binding cassette protein (ABCE1) together promote  the  splitting  of  the  60S  and  40S  subunits after which they can be recycled [@Pisarev2010;@Hellen2018]. 

\subsection{Translation efficiency} \label{transEff}
 Each ribosome synthesises a single protein during translation of an mRNA assuming it is not prematurely terminated. It has been known since the '60s that translation of an mRNA occurs via multiple bound ribosomes (polysomes) simultaneaously (*see figure \ref{fig:polysomes}A-C*) [@Warner1962;@Staehelin1963]. Therefore the translation efficiency of an mRNA depends on the number of ribosomes it is associated with. Experimental methods like polysome profiling and ribosome profiling are used to study translation efficiencies by measuring the number of bound ribosomes on an mRNA (*explained in detail in section \ref{exptMethod}*). While all steps of translation can affect the translation efficiency of an mRNA, it is thought to most commonly be regulated at the initiation step [@Richter2015;@Dever2012;@Jackson2010]. This is supported by findings using polysome profiling in yeast that was cultured in nutrient rich medium where initiation was rate-limiting for most mRNAs [@Arava2003]. Furthermore, a recent ribosome profiling study assessed transcriptome-wide elongation rates. This revealed a similar rate of elongation for mRNAs of different classes, e.g. mRNAs with differences in 5' UTR characterstics [@Ingolia2011]. The next section will go further into detail how translation initiation and elongation can be regulated.
\newline
\section{Regulation of mRNA translation} \label{regmRNA}

 mRNA translation is the most energy consuming process in the cell. In a study using concanavalin A stimulated rat thymocytes, it was estimated that translation accounts for  ~20% of the cellular energy consumption [@Buttgereit1995]. The high energy consumption of mRNA translation  and its central role in the gene expression pathway requires it to be tightly regulated.

 Regulation of translation can be exerted at a global level, i.e. regulation of a large set of mRNA simultaneously. Global regulation of mRNA translation can be achieved by, e.g. perturbations of major signalling pathways impinging on mRNA translation. Regulation of mRNA translation is that it can affect specific mRNA populations selectively. Selective translational regulation acts on characteristics of mRNAs, e.g. through 5' UTRs or RNA binding proteins (RBP) that bind to the 3' UTRs (**see figure \ref{fig:UTRFeat}**) [@Leppek2018]. Furthermore, regulation of initiation factors can also regulate translation selectively [@Wolfe2014,@Gandin2016a]. Below we will discuss multiple mechanisms that regulate mRNA translation globally or selectively.

\subsection{mTOR} \label{mTOR}
 mTOR is  a  conserved  Ser/Thr  kinase  and  is  found  in  two  structurally  and  functionally  distinct  complexes,  mTORC1  and mTORC2 [@Saxton2017;@Pearce2007]. In a growth promoting environment mTOR regulates cell metabolism to increase protein synthesis, lipids and nucleotides, while suppressing catabolic pathways, e.g. autophagy. mTORC2 promotes survival via signalling through protein kinase A (AKT), anabolic metabolism, and cytoskeleton regulation [@Sarbassov2005,@Zoncu2011].

 mTORC1 activity is modulated via hormone and growth factor signalling, e.g. insulin and insulin-like growth factor (IGF1). This signalling is predominantly mediated through the phosphoinositide 3-kinase (PI3K) / AKT pathway. PI3K activation generates phosphatidylinositol-3,4,5-trisphosphate (PIP3). Phosphatase and tensin homologue (PTEN) can counteract this by hydrolsis of PIP3 to phosphatidylinositol 4,5-bisphosphate (PIP2). PIP3 recruits phosphoinositide-dependent kinase 1 (PDK1) and AKT to the plasma membrane. At the plasma membrane AKT is activated through phosphorylation of PDK1. AKT in turn increases activity towards its substrate tuberous sclerosis complex (TSC). TSC consists of a scaffold protein, TSC1, and a GTPase, TSC2. TSC negativeley regulates mTORC1 activity. This negative regulation occurs through hydrolsis of Ras homologue enriched in brain (Rheb) that leads to its inactivation. Rheb binds to mTOR to promote its activity (*see Figure \ref{fig:mtorsignal}*). Furthermore, several cross-talk mechanism with other pathways (e.g. RAS/ERK) have been shown to lead to mTORC1 activation (reviewed in [@Shaw2006]). The mechanisms of mTORC2 activation are less well understood.

The PI3K pathway is involved in oncogenic signalling and under investigation as therapeutic target in cancer [@Hilger2002;@Yang2019]. In several cancers (e.g. breast, lung, prostate and colon) the gene encoding the catalytic p110\(\alpha\) subunit of PI3K (PI3KCA) is frequently mutated or amplified [@Levine2005;@Samuels2004;@Lee2005]. The E545K mutation leads to a reduced inhibitory effect of the regulatory p85 subunit on PI3KCA. In **study 3**, we investigate oncogenic signalling via the PI3K pathway activated by insulin. Here, were focus on the role of mTOR in mediating the effects of insulin on gene expression in the MCF7 breast cancer cell line that harbours the E545K mutation [@Schneck2013]. Hyperactivity of PI3K/AKT signalling has been reported in multiple cancers and has been linked to anti-cancer therapy resistance [@Populo2012;@Tan2013;@Salaroglio2019]. Furthermore, both PTEN and the TSC act as tumour suppressors and are frequently mutated in cancer [@Song2012;@Mak2004]. Therefore, mTOR has become a focus of anti-cancer therapy by either targeting mTORC1 or using dual inhibitors for PI3K and mTOR [@Bhat2015].
\clearpage
 \begin{figure}[ht]
 \centering
  \includegraphics{./figures/mTORsignal.jpg}
  \caption{Schematic representation of mTOR signaling to the translational machinery. Philippe P. Roux, and Ivan Topisirovic Mol. Cell. Biol. 2018; doi:10.1128/MCB.00070-18. Reprinted with permission. Copyright © 2018, American Society for Microbiology
 \label{fig:mtorsignal}}
\end{figure}

 mTOR also fulfills a central role in metabolic signalling. Here, mTOR integrates signals arising through amino acid availability, glucose metabolism and cellular oxygen levels. For example, increased amino acid availability induces relocalisation of mTOR into proximity of Rag GTPases leading to its activation through Rheb [@Sancak2008]. Furthermore, Glucose deprivation leads to increased adenosine-mono-phosphate kinase (AMPK) signalling via serine/threonine kinase 11 (LKB1) [@Kimball2006;@Sanders2007;@Shaw2009]. In turn, AMPK phosphorylates TSC2 leading to its activation [@Kimball2006]. LKB1 mutations have been found in cancer and are being considered as targets in anti-cancer therapy [@Zhao2014]. Hypoxia also inhibits mTOR via regulated in development and DNA damage response 1 (REDD1) which stabilises the TSC [@Brugarolas2004]. While hypoxia (i.e. deprivation of oxygen) inhibits protein synthesis in normal cells, in breast cancer, protein synthesis appeared not to be significantly inhibited during hypoxia which is attributed to uncontrolled mTOR signalling [@Connolly2006].

In cancer, cellular metabolism, proliferation as well as growth are often dysregulated [@Hanahan2011]. Given the central of mTOR governing proliferation, growth and metabolism it is vital to comprehensively understand mTOR signalling in this disease [@Roux2018].

\subsubsection{Global regulation of translation via mTOR}
Well studied downstream targets of mTOR for the regulation of mRNA translation are 4E-binding proteins (4E-BP) and  ribosomal protein S6 kinases (S6Ks). mTOR phosphorylates 4E-BP leading to the release of eIF4E that then can engange in eIF4F complex formation [@Gingras1999]. Therefore, inhibition of mTOR leads to a down regulation of 5' cap-dependent mRNA translation. 

S6Ks have been shown to regulate phosphorylation of multiple components of the translation machinery, e.g. ribosomal protein S6 (rpS6), programmed cell death protein 4 (PDCD4), eEF2 kinase and eIF4B. S6K phosphorylates rpS6 which has been implicated in the regulation of cellular growth and protein synthesis [@Ruvinsky2005]. Furthermore, S6K/rpS6 signalling was suggested to be involed in ribosome biogenesis. Another S6K target is eEF2 kinase which phosphorylates and inhibits eEF2, thus attenuating elongation rates. [@Wang2001]. MTORC1 phosphorylates and inhibits eEF2K either directly or via S6Ks. Furthermore, phosphorylation of PDCD4 by S6K triggers its degradation. PDCD4 blocks eIF4G-eIF4A interactions repressing eIF4A activity and cap-dependent mRNA translation [@Dorrello2006;@Goke2002]. Lastly, phosphorylation of eIF4B by S6K is suggested to stimulate the unwinding activity of eIF4A [@Rogers2001].

Collectively, through acting on its downstream targets, mTOR regulates translation globally via a number of mechanisms.

\subsubsection{Selective or "mTOR sensitive" regulation of translation} 

\paragraph{Terminal oligo pyrmidine mRNAs}
 Selective or "mTOR sensitive" regulation of translation can act on a terminal oligo pyrimidine (TOP) motif in the 5' UTR of mRNAs. This TOP motif consists of a C followed by a stretch of 4-15 pyrimidines directly after the 5' cap. TOP mRNAs show near complete dissociation from ribosomes under conditions when mTOR is inhibited and are  enriched for genes encoding for components of the translation machinery [@Meyuhas2000; @Thoreen2012; @Yamashita2008]. Recent works indicate the importance of La ribonucleoprotein domain family member 1 (LARP1) in regulation of TOP mRNAs with contradictory findings [@Hopkins2016;@Jia2021;@Fonseca2015;@Maraia2017]. A panel of researches was asked to evaluate these findings, which led to the establishment of a model for translational regulation via LARP1 [@Berman2021]. According to this model, LARP1 binds to the 5' mRNA cap of TOP mRNAs via its DM15 domain. Binding of DM15 represses translation of TOP mRNAs by obstructing eIF4E binding and occurs in instances where mTOR activity is reduced. Under conditions where mTOR is active, DM15 is phosphorylated by mTOR. This causes DM15 to release the 5' cap. However, the la domain of LARP1 remains bound to the 3' UTR. The still bound la domain stabilises the mRNA to facilitate translation [@Berman2021]. Other instances of selective translation by mTOR are explained in the next section. 

\paragraph{Selective regulation through members of the eIF4F complex} \label{sel4F}
 As mentioned above, the eIF4F complex consists of eIF4E, eIF4A and eIF4G and is required for cap-dependent mRNA translation [@Hinnebusch2014]. The availability of the eIF4F complex is limited under basal conditions due to that eIF4E is bound by 4E-BPs. Therefore, under basal conditions mRNAs must compete for access to components of the translation machinery. Such competition is affected by characteristics of 5' UTRs that introduce variation to how well mRNAs can be translated. Benedetti et. al. derived and expanded on a model for mRNA competition origally proposed by Lodish [@DeBenedetti2004 ;@Lodish1974]. Herein, "strong" mRNAs are widely expressed and, represent the majority of cellular mRNAs and are characterised by optimally long and unstructured 5' UTRs, e.g. \(\beta\)-actin. An optimal 5' UTR length of 70-150 nucleotides was proposed by Kozak [@Kozak1987]. On the other hand, "weak" mRNAs have long and structured 5' UTRs. "Weak" mRNAs encode for potent growth and survival factors, e.g. c-Myc, ornithine decarboxylase (ODC1) and vascular endothelial growth factor (VEGF). Translation of "strong" mRNAs would remain effective in conditions where eIF4F complex availability would be limited. However, "weak" mRNAs show sensitivity to eIF4F availability which is dependent on eIF4E expression and/or availability [@Graff2008]. Elevated eIF4E expression and/or availability due to increased mTORC1 activity and consequent inhibition of 4E-BPs are common in cancer and drives malignancy due to selective induction of translation of tumor promoting mRNAs [@DeBenedetti2004]. 

 In addition to eIF4E, other eIF4F subunits have been shown to also selectively impact mRNA translation. Therein, mRNAs dependent on eIF4A showed characteristics of "weak" mRNAs, i.e. these mRNAs had long and structured 5' UTRs [@Wolfe2014;@Rubio2014;@Waldron2018]. Stuctural elemenents in the 5' UTRs include classical hairpins formed through Watson-Crick basepairing (*see Figure \ref{fig:UTRFeat}*) [@Leppek2018]. Other structures formed via hoogsteen basepairing have also been proposed to regulate eIF4A dependent mRNA translation [@Wolfe2014]. These structures are called G-quadruplexes. G-quadruplexes are stable structures formed by stacking two or more G-tetrads [@Kwok2017]. However, whether G-quadruplexes form in RNA is still debated [@Guo2016;@Biffi2014;@Laguerre2015;@Weldon2016]. Formation of G-quadruplex structures is predicted based on occurrences of \(GGC_4\) motif repeats in the 5' UTR of mRNAs [@Wolfe2014;@Singh2021]. Indeed, multiple studies report enrichments of \(GGC_4\) motifs in mRNAs whose translation is dependent on eIF4A [@Modelska2015;@Rubio2014;@Waldron2018;@Singh2021]. However, Waldron et al. show that \(GCC_4\) motifs fail to form G-quadruplexes in their reporter mRNA system. The authors concluded that eIF4A dependence of mRNAs with \(GGC_4\) motifs enriched in their 5' UTR was likely mediated by classical hairpin-like structures [@Waldron2018]. In addition, in Rubio et al. one third of eIF4A-dependent mRNAs exhibited multiple 5' UTR variants, while for eIF4A-independent mRNAs this was < 1% [@Rubio2014]. Translation of mRNAs with different 5' UTR variants has recently been implicated to drive cancer cell plasticity towards more "stem cell-like" phenotypes during hypoxia [@Jewer2020]. In addition, eIF4A inhibitors with different mechanisms of action have been shown to exert different impact on the translatome. Hippuristanol inhibits RNA interaction by selective binding to eIF4A [@Bordeleau2006;@Lindqvist2008]. In contrast rocaglates and their derivatives, e.g. silvestrol and CR-1-31-B, clamp eIF4A on polypurine sequences on the mRNA [@Iwasaki2016].
 
 Recently, a more nuanced picture for translation of mRNAs sensitive to inhibition of components of the eIF4F complex has been reported [@Gandin2016a]. Therein, a comparison of treatments with a mTOR and eIF4A inhibitors was evaluated. This revealed that non-TOP mTOR-sensitve mRNAs encompass two functionally distinct sets of mRNAs. These sets were characterised by different 5' UTR features. Here, one subset showed less sensitivity to eIF4A but more so to eIF4E (here after referred to as mTOR-eIF4E senstive mRNAs). mTOR-eIF4E sensitive mRNAs contain very short 5' UTRs (< 40 nt) and encode for proteins in metabolic functions. The other subset was sensitive to both eIF4E and eIF4A (hereafter referred to as mTOR-eIF4A sensitive). mTOR-eIF4A sensitive mRNAs were characterised by long and structured 5' UTRs that endoce for pro-surival proteins. The authors concluded that inhibiton of mTOR-eIF4A dependent programs would lead to cytotoxic effects. Whereas, inhibition of mTOR-eIF4E mRNAs would lead to metabolic dormancy and a cytostatic effect [@Gandin2016a]. The difference in cytostatic and cytoxic effects was attributed to that eIF4A inhibition leads to continuous expression of proteins involved in respiration. However, expression of proteins protecting mitochondrial integrity would be reduced. Thus, leading to apoptosis. In contrast, inhibition of mTOR activity would impact protein expression of both subset simultaneaously, leading to metabolic dormancy.
 
\subsection{The integrated stress response} 
The integrated stress response (ISR) is a pathway activated through kinases responding to various stress signals. These kinases include: Protein kinase R-like endoplasmic reticulum kinase (PERK) activated by misfolded peptides in the endoplasmatic reticulum (ER); Heme regulated eIF2alpha kinase (HRI) activated during heme deficiency; Protein kinase R (PKR) which is activated in response to certain viral infections by binding to double-stranded RNA (dsRNA); and general control nonderepressible 2 (GCN2) which is activated when cells are deprived of amino acids [@Kapur2017;@Guan2017;@Taniuchi2016;@Andreev2015;@Lemaire2005].

\subsubsection{Global and selective regulation of translation via the ISR} Similar to mTOR signalling egulation of translation via the ISR is achieved at a global and selective level. During the ISR the \(\alpha\) subunit of eIF2 is phosphorylated. Phosphorylated eIF2\(\alpha\) directly engages the guanine nucleotide exchange factor eIF2B and prevents conversion of inactive eIF2-GDP to  active eIF2-GTP needed for met-tRNAi incorporation in the TC. This reduces TC availability and causes a global downregulation of mRNA translation [@Sonenberg2009] (*see figure \ref{fig:initiation}*). While global translation is reduced upon ISR, translation of a selective subset of mRNA with upstream open reading frames (uORFs) is increased. A uORF is a reading frame that originates in the 5' UTR of an mRNA upstream of the coding ORF (cdsORF) (*see Figure \ref{fig:UTRFeat}*). uORFs can be out of frame with the cdsORF  and, when translated, lower the expression of the cdsORF [@Kozak1984]. Activating transcription factor 4 (ATF4), that regulates expression of stress response genes, contains multiple uORFs of which one partially overlaps with the cdsORF. Under normal conditions ATF4 translation is initiated at uORF1 and reinitiation at uORF2 occurs. The overlap of uORF2 with the cdsORF causes ribosomes to synthesise protein from uORF2 thereby inhibiting the translation of the cdsORF. Limitation of TC availability during ISR causes longer ribosome scanning times leading to that ribosomes scan past uORF2 and initiate at the cdsORF (i.e. delayed reinitiation) [@Pakos-Zebrucka2016].

 Ribosome profiling studies indicate that 50% of mammalian mRNAs harbour uORFs (*see section \ref{exptMethod} for detials on ribosome profiling*). mRNAs containing uORFs include oncogenes and transcripts important in differentiation and cell cycle [@Calvo2009;@Ingolia2011]. Apart from delayed reinitiation, uORF translation can also be regulated by "leaky scanning". The surrounding sequence of the uORF is important for initiation of translation. An AUG in the classical Kozak context (i.e. RNN**AUG**G) is most efficient for translation initiation due to better recognition by the met-tRNAi [@Kozak1986;@Calvo2009]. Unfavoured flanking sequences of the AUG can cause the ribosome to scan past the AUG, this process is called "leaky scanning". An example of this is DNA-inducible gene 34 (GADD34) which increases its translation upon ER stress, i.e. a condition where eIF2\(\alpha\) is phosphorylated. In humans, GADD34 contains two uORFs separated by 30 nucleotides [@Lee2009]. In contrast to the uORFs in ATF4, the uORFs in GADD34 are too close together to allow for reinitiation. Under basal conditions uORF2, which has a poor Kozak context, represses translation of the cdsORF. However, under stress causing eIF2\(\alpha\) phosphorylation, ribosomes scan past uORF2 to translate the cdsORF [@Young2015]. Lastly, structural elements in proximity of the uORF can also influence its translation [@Ruan1994]. 

\subsection{Regulation of mRNA translation by tRNAs} \label{tRNA}
\begin{wrapfigure}{r}{0.6\textwidth}
  \includegraphics{./figures/tRNA.pdf}
  \caption{ Schematic representations of (A) the tRNA cloverleaf structure with indicated anti-codon and amino acid acceptor sites;  (B)  the proliferation and differentiation mRNAs dependent on distinct tRNA subsets; (C) the U34 wobble position and the catalytic enzymes involved in this modification that is implied in tumor progression.
 \label{fig:tRNA}}
\end{wrapfigure}
 As touched upon earlier, tRNAs are an essential part of the translation machinery that carry the amino acids to the ribosome. In eukaryotes, tRNAs consist of a 76-90 long nucleotide sequence set into a "cloverleaf" structure forming several loops [@Sharp1985] (*see figure \ref{fig:tRNA}*). The acceptor stem binds the amino, while the anti-codon loop binds to the mRNA within the ribosome via classical Watson-Crick basepairing [@Watson1953]. Multiple codons can encode for the same amino acid (synonymous codons), however the availability of the tRNAs for different codons may vary which can influence elongation rates and thus protein synthesis. 

 This supply ( i.e. tRNA availability) and demand (i.e. codon composition) relationship has been found to vary across different cellular states, e.g. proliferation and differentiation. Gingold et. al. observed two different tRNA subsets. One subset is induced when proliferation is stimulated and is otherwise repressed. And the other subset increases expression when differentiation is induced and is repressed otherwise (*see figure \ref{fig:tRNA}*). Both tRNA subsets match the codon demand of the transcriptome under their respective cellular state [@Gingold2014]. This model has been disputed and it was proposed that the observed differences may be attributed to GC content in the mRNA [@Rudolph2016]. Nevertheless, abberant tRNA expression and codon usage have been reported in cancer [@Zhang2018]. Furthermore, a comprehensive study using small RNAseq (e.g. for identification of tRNAs) and protein samples across 17 tissues reported a tRNA signature stratified by proliferation marker Ki67 staining. The identified tRNA signature has implications for patient survival [@Hernandez-Alias2020]. Therefore, while a consensus on proliferation specific tRNA subsets might not have been reached, emerging evidence implicates a role thereof in cancer [@Hernandez-Alias2020;@Gingold2014;@Zhang2018]. For instance, increased expression of \(tRNA_{CCG}^{Arg}\) and \(tRNA_{UUC}^{Glu}\) has been observed in breast cancer cell lines and are proposed to drive metastasis [@Goodarzi2016]. 

 Others reported of a role for tRNAs in cancer attributed to tRNA modifications, specifically at the highly conserved U34 anti-codon (wobble) position [@Rapino2017;@ElYacoubi2012;@Lorent2019]. The ability to wobble was proposed by Francis Crick and refers to non-Watson-Crick base pairing of tRNA anti-codons [@Crick1966]. This enables a smaller set of tRNAs (41-55 in eukaryotes) to encode for the 64 possible codon combinations [@Goodenbour2006]. In mammals, the U34 modification catalytic cascade involves the acetyltransferase Elongator (ELP3), the methyltransferase TRM9-like domain of Alkylation repair homolog 8 (ALKBH8), and the urmylation (URM) pathway. The URM pathway includes the cytosolic thiouridylase homolog 1 and 2 (CTU1/CTU2) [@Kalhor2003;@Karlsborn2014]. These enzymes ultimately modify the U34 position into 5-methoxycarbonyl-methyl-2-thiouridine (\(mcm^5s^2U\)) which ensures cognate codon recognition. This modification is thought to occur in tRNAs with a U in the wobble position, e.g.  \(tRNA^{UUU}\), \(tRNA^{UUC}\), \(tRNA^{UUG}\), \(tRNA^{UCC}\), and \(tRNA^{UCU}\) (*see figure \ref{fig:tRNA}*). 

 Loss of the ability to modify U34 has been shown to reduce translation elongation rates with varying effects on protein expression [@Nedialkova2015;@Deng2015;@Zinshteyn2013]. While in some cases U34 dependent signalling led to ribosome stalling resulting in protein aggregates and increased stress [@Nedialkova2015;@Zinshteyn2013], others reported a subtle dowregulation of proteins encoded by mRNAs requiring U34-modified tRNAs [@Deng2015]. U34 modification dependent tRNAs have been shown to play a role in cancer. For example, ELP3 is important in tumor initiation in intestinal epithelia and promotes breast cancer invasion as well as progression to metastasis [@Ladang2015;@Delaunay2016].

\subsection{RNA binding proteins and trans-acting factors}
 The UTRs of an mRNA contain sequence elements to which small RNA and RNA binding proteins (RBPs) bind and exert translational regulation (*see figure \ref{fig:UTRFeat}*).
 
 \subsubsection{microRNAs}
 microRNAs (miRNA) are a class of small non-coding RNA. The precise role and mechanisms of regulation of translation by miRNAs are still under active investigation [@Oliveto2017]. However, miRNAs can directly bind to other mRNAs and silence them which is accomplished through translational repression or destabilisation [@Jonas2015]. Regulation of gene expression by miRNAs has been observed in cancer. Here, miRNAs have been implicated to promote tumorgenesis or act as tumor suppressors [@Sampson2007;@Muniyappa2009;@Tian2010;@Nagpal2015].
 
\subsubsection{RNA binding proteins}
RBPs are a class of proteins involved in many regulatory steps of gene expression and account for ~7.5% of the protein coding genes. RBPs bind to elements in the 3' UTRs, e.g. the poly (A) tail. One RBP binding the poly(A) tail is PABP. As explained in *section \ref{initiation}* PABP is involved in mRNA translation initiation and stabilisation of the mRNA [@Afonina2014;@Amrani2008] (*see also figure \ref{fig:initiation}*). 

Another site in the 3' UTR is the U-rich cytoplasmic polyadenylation (CPE) site to which cytoplasmic polyadenylation element binding proteins (CPEBs) can bind (*see Figure \ref{fig:UTRFeat}*). Studies in *Xenopus* oocytes indicate that CPEB associates with a non-canonical poly(A) polymerase (Gld2) and a poly(A) deadynelase (PARN). PARN has a higher activity than Gld2 and thus binding of CPEB to an mRNA leads to shortening of the Poly(A) tail [@Barnard2004]. However, hormonal stimulation leading to CPEB phosphorylation removes PARN from the complex. Removal of PARN promotes poly(A) tail elongation through Gld-2 [@Kim2006]. Furthermore, in *Xenopus* oocytes, CPEB associates with an eIF4E binding protein, i.e. maskin. Maskin bound to eIF4E prevents eIF4F complex formation which represses mRNA translation [@Stebbins-Boaz1999;@Ivshina2014]. Therefore, CPEBs can regulate translation by altering 3' UTR lengths and are involed in transaltional repression by blocking eIF4E association with the 5' cap in *Xenopus* oocytes. While the role of CPEB mediated regulation has been described mostly in *Xenopus* oocytes, dysregulation of CPEBs has been observed in glioblastoma, colorectal and pancreatic cancer [@Ortiz-Zapater2011; @Villanueva2017;@Chang2014].

Another important RBP implicated in regulation of translation is Human antigen R (HuR). HuR preferentially binds to AU-rich sequences in the 3' UTR, acts as a stabilising agent and is involved in RNA-processing [@Levine1993;@Baou2011;@Fan1998;@Peng1998]. In colorectal carcinoma cells HuR has been shown to enhance protein synthesis of p53 after exposure to short-wavelength UV light (UVC) by binding the 3' UTR [@Mazan-Mamczarz2003]. The enhanced effect on protein synthesis was not mediated by stability as HuR failed to stabilise p53 upon UVC exposure. In human bone osteosarcoma epithelial cells, overexpression of HuR led to a dose dependent increase in eIF4E protein levels. The increase in protein levels was atrributed to eIF4E transcript stabilisation through binding of HuR to AU-rich elements (AREs) in the 3' UTR of eIF4E [@Topisirovic2009]. Other known drivers of tumor progression targeted by HuR include hypoxia inducible factor 1 (HIF-1), VEGF, c-Myc [@Denkert2004;@LopezdeSilanes2003;@LopezdeSilanes2005]. Furthermore, studies in breast, colon and lung cancer observed correlation between HuR and malignancy [@Denkert2004;@LopezdeSilanes2003;@LopezdeSilanes2005].

\clearpage
\section{Expertimental methods to measure mRNA translation} \label{exptMethod}
\begin{wrapfigure}{r}{0.6\textwidth}
    \includegraphics{./figures/polyRibo.pdf}
  \caption{Polysome profiling and ribosome profiling workflows. In polysome profiling a fraction from whole cytoplasmic RNA is loaded onto a sucrose gradient on which they get separated by sedimentation into the sucrose gradient using ultra centrifugation. Fractions corresponding to efficiently translated mRNAs are collected and can be quantified with for example RNA sequencing (left). During ribosome profiling a fraction from the whole cytoplasmic RNA is exposed to a digestion agent which breaks the RNA. Ribosomes will protect fragments thereby creating ribosome protected fragments. These fragments are then isolated and can be sequenced. Note the differently coloured mRNAs and how both methods separate them. \label{fig:polyRibo}}
\end{wrapfigure}
Two methods are predominantly used for measuring mRNA translation to estimate changes in translation efficiencies across conditions, namely polysome profiling and ribosome profiling (**see figure \ref{fig:polyRibo}**). These methods capture the number of ribosomes an mRNA is associated with. The number of bound ribosomes is an adequate of estimator changes in translation efficiencies when initation is rate limiting. Indeed, as explained above this is a common scenario (*see section \ref{transEff}*).

\subsection{Polysome profiling} 
Polysome profiling allows for separation of polysomes from monosomes, ribosomal subunits and messenger ribonucleoprotein particles (mRNPs). During this assay, ribosomes are immobilized on the mRNAs using translation elongation inhibitors, e.g. cycloheximide (CHX). Cytoplasmic extracts sediment on a linear sucrose gradient (5-50%) using ultra centrifugation. The resulting gradient is fractionated and mRNAs with different numbers of bound ribosomes can be extracted and analyzed for mRNA content [@Gandin2014]. Typically fractions belonging to mRNAs with more than 3 bound ribosomes are pooled. A 3-ribosome cutoff has been chosen as it is thought to capture most biologically relevant changes in translation efficiency [@Gandin2016a].
\begin{wrapfigure}{r}{0.6\textwidth}
    \includegraphics[width=0.9\linewidth]{./figures/polysome_shifts.pdf}
  \caption{Polysome profliles -  (top left) Schematic representation of a polysome profile using linear sucrose gradient fractionation. Indicated in the polysome profiles are the 40S, 60S ribosomal subunits as well as the 80S monosome. H.P. indicates heavy polysome fractions.Between conditions (i.e. black an pink lines) distribution changes for mRNA abundance (grey and green; top right), translation (grey and red; bottom left) and translation within high polysome fractions (grey and red; bottom right) are illustrated. \label{fig:polysome}}
\end{wrapfigure}
An illustration of a polysome profile with peaks for the 40S, 60S subunits and 80S ribosome can be seen in (*Fig \ref{fig:polysome} top left*). Subsequent peaks along the frations indicate the mRNAs with 2 or more bound ribosomes. mRNAs are typically normally distributed along the fractions, i.e. in a pool of the same mRNA many will be associated with different numbers of ribosomes [@Gandin2016a]. Changes in mRNA abundance may lead to an overall increase in the amount of isolated polysome-associated mRNA without a shift of the distribution along the fractions (*Fig \ref{fig:polysome} top right*). This means that the translation efficiency per mRNA remains unchanged. In contrast, changes in translational efficiency can be observed by shifts  along the polysome fractions. For example, if the mRNAs shift from the light (inefficiently translated) towards the heavy (efficiently translated) polysome fractions or vice versa in the absence of changes in total mRNA levels (*Fig \ref{fig:polysome} bottom left*). Shifts within the heavy polysome fractions (i.e. with a mean distribution around 4 bound ribosomes to 7 bound ribosomes) can also occur (*Fig \ref{fig:polysome} bottom right*). Quantification of mRNA levels within each fraction can be assessed using Northern blotting or reverse transcription quantitative polymerase chain reaction (RT-qPCR). For transcriptome-wide studies, efficiently translated mRNAs are pooled and quantified using either DNA-microarrays or RNA sequencing.

 Pooling of mRNAs as well as collection of multiple fractions makes polysome profiling inconvenient when dealing with low amounts of input RNA or with large samples sizes. Therefore, an optimised sucrose gradient was developed where mRNAs bound to >3 ribosomes are collected on a sucrose cushion and thereby can be isolated from one single fraction [@Liang2018]. This optimised gradient allows for application of polysome profiling in small tissue samples where RNA quantity is limiting and reduces labor intensity of the assay. 

 Polysome-associated mRNA levels are subject to changes in translation efficiency as well as factors contributing to cytosolic mRNA levels. Cytosolic mRNA levels impact the pool of mRNAs that can associate with ribosomes. However, changes in cytosolic can occur due to, e.g. transcription or mRNA stability.  Therefore, to identify true changes in translation efficiency it is important to collect cytoplasmic mRNA and polysome-associated mRNA from the same sample in parallel to correct for such mechanisms during downstream analysis. 

\subsection{Ribosome profiling} \label{riboseq}
R17 bacteriophage ribosome protected RNA fragments (RPFs) have been obtained in the 1960s using ribonucleosases to trim away mRNA sequences not protected by ribosomes [@Steitz1969]. More recently ribosome profiling has been developed. Ribosome profling encompasses sequencing of RPFs on a transcriptome-wide scale [@Ingolia2009;@Ingolia2016]. Similar to polysome profiling. in the ribosome profiling assay ribosomes are immobilised on mRNAs using translation elongations inhibitors (e.g. CHX).

RPFs are obtained by RNAse treatment that degrades the links of RNA between ribosomes leaving single ribosomes covering a ~28 nucleotide long RNA fragment. However, as ribosomal RNAs (rRNA) are also degraded during this process they represent a major contaminant herein. The RPFs are then isolated using ultra centrifugation through a sucrose cushion. During this step other RNA fragments such as rRNAs,  non-coding RNAs (ncRNAs) or large ribonucleoprotein complexes can co-migrate and contaminate the sample. Typically RPFs with a size ranging from 25-30 nucleotides are selected for quantification. However, RBP-or 43S-protected RNAs, such as those localised in stress granules, may produce fragments of similar length. Furthermore, ribosomes undergoing conformational changes have been shown to protect fragments corresponding to a length of 21nt when translation elongation is blocked by e.g. CHX [@Lareau2014]. Therefore, size selection can distort the estimation of translation efficiency from ribosome profiling data [@Andreev2017]. From the resulting RPFs libraries are generated and quantified using RNA sequencing. During library construction additional biases due to enzyme sequence preferences can be introduced that can lead erroneus estimations of codon positions within the ribosome [@Artieri2014a]. 

Recently, the algorithm Ribo-seq Unit Step Transformation (RUST) has been developed [@OConnor2016]. RUST can reveal mRNA sequence features affecting RPF density globally. The authors applied RUST to 30 publicly available data sets and identified substanial sequence heterogeneity affecting RPF densities. Thus, sequence bias is prominent in ribosome profiling [@OConnor2016].

 Initially, fragmented total mRNA using alkaline hydrolysis of the same size were retrieved in paralell to RPFs. This was achieved by extraction of total mRNA from cell lysate followed by purification via recovery of polyadenylated messages or removal of ribosomal RNA [@Ingolia2009;@Brar2015]. The random fragmentation of total mRNA has been shown to result in experimental bias. Therefore, now sequencing of unfragmented total mRNA in parallel is preferred.

\subsection{Comparing ribosome and polysome profiling}
 Albeit both methods generate count data after quantification with RNA sequencing, there are some key aspects that differ between the techniques. Polysome profiling separates efficiently translated mRNAs from non- efficiently translated mRNAs along a sucrose gradient thereby creating an mRNA based perspective for analysing changes in translational efficiencies. In contrast, ribosome profiling determines translational efficiencies by counting the number of RPFs of both efficiently and non-efficiently translated mRNAs (*see figure \ref{fig:polyRibo}*). This can have implications for identifaction of transcript variants. For example,  if a ribosome would not protect a fragment differentiating two isoforms, information on the isoforms would be lost during ribosome profiling [@Floor2016]. In polysome profiling the integrity of the mRNA is preserved. This gives polysome profiling the advantage in cases where transcript variants with different 5' UTR lengths are of interest. This makes polysome profiling, but not ribosome profiling, compatible with methods to investigate UTRs (e.g. CAGE and 3P-seq) [@Takahashi2012;@Jan2011].

Shifts in ribosome association can be dramatic (e.g. near complete dissociation of ribosomes from an mRNA) or subtle (e.g. shifts from  2 to 4 ribosomes). These shifts can occur due to different properties of the mRNAs the ribosomes associate with [@Gandin2016a]. When dramatic and subtle changes in ribosome association are present in parallel, ribosome profiling is biased towards identification of shifts with a greater magnitude and masks the subtle ones [@Gandin2016a]. Gandin et. al. showed under mTOR inhibition ribosome profiling studies are likely to predominantly identify TOP mRNAs (i.e. heavy shifters). In contrast, polysome profiling identified both TOP and non-TOP mRNAs (i.e. more subtle shifters compared to TOP mRNAs) when mTOR is inhibited [@Gandin2016a]. Therefore, in scenarios where global mRNA translation is affected, application of ribosome profiling could lead to imprecise biological conclusions. The masking of subtle changes has been attrituted to the indirect estimation of translation efficiencies from counting RPFs for ribosome profiling, which is highly dependent on the abundance of mRNAs. In polysome profiling  this effect is much less pronounced as changes in translation efficiency are directly estimated from the mRNAs associated with heavy polysomes [@Masvidal2017]. Therefore, polysome profiling is more suitable than ribosome profiling in studies that aim to analyse global changes in translation efficiences. [@Gandin2016a;@Masvidal2017].

 An advantage of ribosome profiling is that it provides exact nucleotide positions occupied by ribosomes. This offers information at a single nucleotide level where the ribosome is located at the mRNA. Polysome profiling cannot reveal ribosome locations along the mRNA. The single nucleotide resolution of ribosome profiling is necessary in contexts studying events such as ribosomal frame shifts or uORF translation [@Rato2011;@Andreev2015]. One limitation of ribosome profiling to identify features such as uORFs, is the use of elongation inhibitors (e.g. CHX) as pre-treatment in the protocol. After CHX treatment elongation is not immediatly inhibited but continues for several cycles [@Hussmann2015]. Studies investigating stress in yeast showed that increases in ribosome occupancy were due to CHX treatment rather than stress [@Gerashchenko2014]. Therefore, CHX pre-treatment can introduce biases in ribosome profiling obscuring indentification of sequence features that are potentially important for translational regulation [@Gerashchenko2014;@Hussmann2015].

Both methods have their strengths and weaknesses. Thus, when designing an experiment each method should be considered and chosen depending on the underlying biological question.
\newline
\section{Modes for regulation of gene expression through mRNA translation} \label{modes}
\begin{wrapfigure}{r}{0.6\textwidth}
  \includegraphics{./figures/geneModes_MRNA.pdf}
  \caption{Modes for regulation of gene expression - Schematic representation of modes by which mRNA translation regulates gene expression. Indicated are: changes of the translation mode, i.e. an mRNA increases its ribosome association without a change in total mRNA levels; The mRNA abundance mode, i.e. changes in total mRNA levels paralleled by changes in translated mRNA levels; The translational buffering mode, i.e. changes in total mRNA levels without changes in translated mRNA levels. Also indicated is the expected outcome on the proteome for all three modes. \label{fig:modes}}
\end{wrapfigure}
From transcriptome-wide assessments of translation using ribosome or polysome profiling expression levels  for both cytoplasmic and polysome-associated mRNAs (or RPFs) are obtained. For simplicity, from now on, these RNA types will be referred to as total mRNA (i.e. cytoplasmic mRNA) and translated mRNA (i.e. polysome-associated mRNA or RPFs). The estimation of expression levels for both translated mRNA and total mRNA allows distinction between changes occuring at total mRNA level or translation and their integration. This is important because, the interplay of total mRNA with translated mRNA can give valuable insights about the underlying mechanisms that govern gene expression in the studied system.

When comparing perturbed systems to their corresponding control state three "modes" in which translated mRNA and total mRNA distinctly interact can be observed (*see figure \ref{fig:modes}*). These modes are referred to as "mRNA abundance", "translation" (i.e. changes in translation efficiencies leading to altered protein levels) and, "translational buffering" (i.e. changes in translation efficiencies ensuring stable protein levels across conditions). 

\subsection{mRNA Abundance mode} 
A change in mRNA abundance is observed when total mRNA levels change, by e.g. transcription or mRNA stability, but the translation efficiency of those mRNA is unaltered. (*see figure \ref{fig:modes}*). The change in total mRNA levels alters the number of mRNAs to compete for components of the translation machinery. Thus, their association to ribosomes can be explained by their abundance. While the translation efficiency for these mRNAs is unaltered, an alteration of protein synthesis from is expected as more or less mRNAs are translated.

\subsection{Translation mode} 
 A change in translation occurs when translated mRNA levels either increase or decrease, while corresponding total mRNA levels remain constant or change to a lesser extent (*see figure \ref{fig:modes}*).  The change in ribosome association indepedent of total mRNA levels is therefore a change in their translation efficiency. A prominent example of this mode can be observed for TOP mRNAs. Under conditions when mTOR is inhibited, TOP mRNAs show a near complete dissassociation from ribosomes [@Gandin2016a]. Furthermore, during the ISR, translation of ATF4 mRNA is altered due to eIF2\(\alpha\) phosphorylation. Similar to changes in mRNA abundance, mRNAs under the translation mode lead to changes in corresponding protein levels (*see figure \ref{fig:modes}*). 

\subsection{Translational buffering mode} \label{modeBuffering}

The third mode of regulation of gene expression is termed translational buffering. Under this mode, a change in total mRNA levels is observed, whereas polysome-associated mRNA levels remain constant between conditions. Translational buffering thus reflects a change in translation efficiency as the proportion of mRNAs associated with ribosomes is altered. In contrast to changes in translation and mRNA abundance, translational buffering has been shown to maintain similar protein levels across conditions rather than inducing alterations to them (*See figure \ref{fig:modes}*) [@Lorent2019;@McManus2014].

Current literature supports multiple forms of translational buffering. Translational buffering can compensate for differences in mRNA levels due to e.g. differences in gene dosages, so that protein levels remain similar [@Dassi2015;@Lalanne2018]. Furthermore, rather than compensating it can "offset" modulations of total mRNA levels at the level of translation to maintain corresponding protein levels [@Lorent2019]. 

Translational buffering in its compensation form has been observed at steady state levels. A study comparing co-evolution of transcription and translation across seven different organs and mammals showed overall greater divergence of transcription as compared to translation [@Wang2020]. Similar compensation at the level of translation has been observed between individuals, tissues and prokaryotes [@Cenik2015;@Perl2017;@Lalanne2018;@Dassi2015;@Artieri2014].

Compensation via translational buffering can also enforce equilibration of pathway or protein complex stoichometry [@Lalanne2018;@Li2014]. An example of this was observed in  evolutionary distant bacetria, i.e. *B. subtilis* and *E. coli*. In *B. subtilis* translation related factors rpsP and rplS are located in different operons, whereas in *E. coli* rpsP and rplS lie within the same operon as rimM and trmD. While *B. subtilis* can fine tune transcription at the different operons, in *E. coli* these will be transcribed together. However, in *E. coli* rimM and trmD are only required in low protein abundance, whereas rpsP and rplS are required in high abundance. *E. coli* compensates the transcriptional input at the translational level and thus equilibriates for requirements in pathway stoichometry [@Lalanne2018].
 
As mentioned above, a different form of translational buffering can be observed in perturbed systems that offset changes in total mRNA levels at the level of translation temporarily. For example, translational offsetting was observed in prostate cancer cells where a transcriptional program was induced under estrogen receptor \(\alpha\) (ER\(\alpha\)) depletion that showed no increase in polysome-associated mRNA. mRNAs whose transcription was translationally offset required the tRNA U34 modification. ER\(\alpha\) has been shown to regulate the expression of the catalytic enzymes required for the U34 modification [@Lorent2019]. Thus, depletion of ER\(\alpha\) led to impeded modification of tRNAs at the U34 position. Therefore the translation efficiency of mRNAs requiring the modification was reduced despite increased total mRNA levels. The authors confirmed that protein levels of translationally offset mRNAs were unaltered between conditions [@Lorent2019].
\newline
\section{Algorithms for analysis of changes in translation efficiencies}\label{algorithm}

As discussed above, the proteome can be reshaped via multiple modes for regulation of gene expression. Furthermore, these modes can have different underlying mechanisms. It is therefore warrented to distinguish them in analysis of translation efficienies. This section discusses methods to analyse polysome-profiling and ribosome profiling data to estimate changes in translation efficiencies across 2 or more conditions.

 Initially, analysis of transcriptome-wide translation studies used an approach called the translation efficiency score (TE-score) that applied the following equation: 
\[\varDelta TE = \frac{\frac{P_{c2}}{T_{c2}}} {\frac{P_{c1}}{T_{c1}}}\\\]

This score calculates the ratio of the ratios between polysome-associated mRNA levels (P) divided by total mRNA levels (T) within each condition, i.e. C1 and C2. The TE- score approach has been shown to be prone to spurious correlations [@Larsson2010]. Spurious correlations arise due to that the ratio of polysome-associated mRNA and total mRNA can systematically correlate with total mRNA levels which is not corrected for in this equation and leads to an elevated type-1 error.

 *Figure \ref{fig:TE}* gives an overview of the relationship between a change in TE-score and each gene expression mode (*see also figure \ref{fig:modes}*). Changes in mRNA abundance will lead to a \(\varDelta\)TE close to 0 in log space (i.e. no change in translation efficiency). This is due to that total mRNA and translated mRNA change with a similar magnitude. However, in the case of both the translation mode and the translational buffering mode, the nominator and denominator in the TE-score equation change leading to a \(\varDelta\)TE (TE < 0 or TE > 0). Therefore, the TE-score cannot distinguish between changes belonging to the translation mode from changes of the translational buffering mode. This can have drastic consequences for the interpretation of the results.
\clearpage
\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_TE.pdf}
  \caption{TE-scores for modes for regulation of gene expression -  Schematic representation of modes for regulation of gene expression in a fold-change scatter plot. The x-axis indicates the fold-change for total mRNA levels. The y-axis indicates the fold-change for translated mRNA levels. Indicated in red are changes in translation efficiency altering protein levels (i.e. translation mode), in green changes in mRNA abundance and in blue changes in translation efficiency leading to translational buffering/offsetting. The relationship for the TE-score and the modes for regulation of gene expression is indicated. \label{fig:TE}}
\end{wrapfigure}
 The TE-score approach was questioned when proposing the Analysis of Translation Activity (anota) algorithm which was developed for DNA-microarray data [@Larsson2010]. Anota combines analysis of partial variance (APV) [@Schleifer1993] with a random variance model (RVM) [@Wright2003]. RVM estimates gene variance using shared information across all genes to increase power for detection of differential expression [@Wright2003]. Anota uses a two-step process that firstly assesses the model assumptions for (i) absence of highly influential data points, (ii) samples classes sharing a common slope, (iii) homoscedasticity of residuals and (iv) normal distribution of per gene residuals. In the second step anota performs analysis of changes in translational activity using the following model:

\[y_{gi} = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 
  
here \(y_{gi}\) is the polysome associated mRNA expression, \(\beta_g^{RNA}\) describes the relationship to total RNA for the \(gth\) gene of the \(ith\) sample column of model matrix \(X\); \(\beta_g^{cond}\) represent the difference in intercept between treatment classes and \(\varepsilon_{gi}\) denotes the residual error. 
\clearpage
\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_anota_Larsson.pdf}
  \caption{anota gene models - Schematic representation of the anota analysis models. Translated mRNA expression is set out against total mRNA expression for each biological replicate and treatment condition. Top left shows the model of a gene that is differentially translated (i.e. change in translated but not total mRNA). The difference in the slope intercepts is used to estimate changes in translation efficiencies between conditions. Other gene models are shown; change in translation efficiency leading to altered protein levels with varying total mRNA levels (top right); change in mRNA abundance (bottom left) and translational buffering (bottom right).
  \label{fig:anota}}
\end{wrapfigure}

 Within anota, a common slope for the treatment classes that describes the relationship between translated mRNA and total mRNA is calculated. The difference between the slope intercepts is then interpreted as the change in translation efficiency. A simplified view of this model is provided in (*Figure \ref{fig:anota} top left*). Here, expression for translated mRNA and total mRNA are modeled over two sample classes that each has 4 replicates. Changes in translation efficiencies can also be observed when translated mRNA expression is modulated to a larger extent than the total mRNA levels (*Figure \ref{fig:anota} top right*). Identification of genes in this category can be a challenge, especially in highly variable data sets, as they resemble genes regulated at the level of mRNA abundance (*Figure \ref{fig:anota} bottom left*). Nevertheless, using the linear regression analysis anota accurately corrects changes in translated mRNA as can be seen in (*Figure \ref{fig:anota} bottom right*). Here, a change in total mRNA but not translated mRNA levels is observed. For this gene, the difference in slope intercepts between sample classes is small and will not be identified as differentially translated as would be the case in the TE-score approach. Anota was developed at a time where translational buffering was not considered in data sets. Naturally, the method lacks a setting to analyse translational buffering. This was addressed in anota's successor, anota2seq, and will be discussed in *Study 1*.

 Advances in experimental methods warrant for appropriate statistical approaches to analyse data resulting from them. DNA-microarray was the dominant platform to assess transcriptome-wide changes before the advent of RNA sequencing. DNA-microarrays measure intensity after hybridisation events which is an indicator of expression. In contrast, in RNA sequencing reads of constructed libraries are counted. Intensity data from DNA-microarray can be normalised and transformed (i.e. log transformation) to fulfill the requirements for application of linear models, whereas RNA sequencing harbours additional characteristics that need to be accounted for. Therefore, algorithms developed for analysis of DNA- microarray are not directly applicable to RNA sequencing data as is the case for the anota algorithm.

 RNA sequencing data shows variance that is greater than the mean which is commonly referred to as overdispersion. Count data from RNA sequencing have been initially approached using Poisson distributions which assumes that the variance is equal to the mean [@Lu2005]. Now established RNA sequencing analysis frameworks such as edgeR and DESeq2  use negative binomial distributions in combination with generalised linear models (GLMs) [@Robinson2010; @Love2014]. The negative binomial distribution uses a dispersion parameter to account for differences in the mean-variance relationship across the expression range [@McCarthy2012]. While analysis principles of DESeq2 and edgeR are similar, they differ in their normalisation method, dispersion estimation and information sharing across genes. In a simple differential expression analysis between two conditions with one RNA type the GLM model would be as in the following equation: 

\[log(y_{gi}) = \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 

here \(\beta_g^{cond}\ X_i^{cond}\) represent the condition (i.e. control and treatment) log2 fold change for the \(gth\) gene \(ith\) sample column of the model matrix X and \(\varepsilon_{gi}\) denotes the residual error. When analysing changes in translation effiencies additional parameters for RNA type (i.e. total mRNA or translated mRNA) and the interaction between the RNA type and condition are added so that:

\[log(y_{gi}) = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \beta_g^{RNA:cond}\ X_i^{interaction} + \varepsilon_gi\] 

 In this model, the interaction term is interpreted as the change in translation effiencies [@Chothani2019]. Other methods (i.e. Ribodiff [@Zhong2017], Riborex [@Li2017] and deltaTE [@Chothani2019]) borrow this analysis principle of an GLM with an interaction term by applying this exact model. A notable difference is that Ribodiff allows dispersion estimation for translated mRNA and total mRNA separately. Differences in within-RNA variances between RNA types can be expected due to varying experimental protocols [@Zhong2017; @Liang2018]. While the flexibility of GLMs allows for complex study designs involving 2 or more treatment conditions, Riborex and Ribodiff limit the study design to only two conditions. DeltaTE gives their users full flexibility of the DESeq2 GLM model. Xtail is a method developed for ribosome profiling that makes use of DESeq2 for RNAseq count normalisation [@Xiao2016]. Their assessment of differences in translation efficiencies relies on probability matrices for the ratio of translated mRNA over total mRNA within condition and a between condition ratio of these ratios. Babel was the first algorithm designed solely for analysis of differential translation and uses an error-in-variables regression analysis [@Olshen2013]. The error-in-variables regression allows accounting for variable total mRNA levels when assessing changes in translation.  Although these methods have distinct approaches to identify changes in translation efficiencies, their principle of analysis is similar to comparing a ratio of ratios (see TE-score equation above). Therefore these methods suffer from similar issues as the TE-score which will be discussed in **Study 1**.


