---
output: pdf_document
---

\chapter{Introduction}

\section{Gene expression}
\subsection{The central dogma of gene expression}
  The whole genetic code of an organisms i stored as deoxyribonucleic acid (DNA) molecules in a double stranded formation as chromosomes. Chromosomes hold the DNA in a condensed state using chromatin, a complex of DNA and proteins, structures. For transcription of DNA to occur chromatin is remodeled to expose promotor regions in the DNA to which fators assisting in transcription bind. One of these factors is DNA polymerase that unravels the double-stranded DNA and creates a single-stranded copy called ribonucleic acid (RNA) transcripts. This copy is less stable due to its single strandedness and therefore only temporary.  The RNA transcripts undergo processing by which multiple different transcript varaints coming from the same genomic region can be produced. The protein coding portion of these transcripts are called mRNAs. Once formed mRNAs are transported into the cytoplasm where they are either degraded or associate with ribosomes. These ribosomes translate the mRNAs into proteins by which the genetic information then is expressed. Synthesised proteins, if no longer needed in the cell, can be degraded by proteosomes (**see figure \ref{fig:geneExprPath}**). This flow of genetic information into expressed proteins is commonly referred to as the central dogma in molecular biology [@Crick1970].
  \clearpage
  \begin{figure}
  \includegraphics{./figures/geneExprPath_2.pdf}
  \caption{The gene expression pathway - DNA is transcribed in pre-mRNA containing a 5' cap (black oval) introns (teal boxes),exons (purple boxes) and a poly(A) tail. RNAs are processed into mRNAs consisting out ouf a 5' cap,exons and a poly(A) tail is then transported out of the cellular nucleus into the cytoplasm. Within the cytosplasm mRNAs can be degradated or translated  into proteins depending on cellular demands. Synthesised proteins can be degraded by proteosomes. \label{fig:geneExprPath}}
\end{figure}
\clearpage
\subsection{Contribution to gene expression}
  Proteins are the last product of the gene expression pathway and carry out the vast majority of all cellular functions. While it is apparent that modulation of protein levels will offer information on the changes in gene expression, it cannot completely answer the question as to why the levels change. In a disease context, protein levels alone might offer sufficient insight to explain phenotypic differences. However, how these differences arise machanistically, which often poses as target for therapeutic strategies in cancer, is obscured.
  
  Experimental methods to measure gene expression at different steps are often to referred as "omics" (i.e. proteomics for protein expression, transcriptomics for mRNA expression). These methods provide snapshots of the step under scrutiny in a specific context (steady state or perturbation) for a large portion of genes or proteins. Transcriptomics studies approach gene expression with the assumption that mRNA expression results in protein expression changes and can therefore be used as a proxy for them. However, this view got challanged by landmark studies that observed a poor mRNA to protein correlation and indicated a larger role of post transcriptional regulation in gene expression that previously assumed [@Lu2005,@Vogel2012,@deSousaAbreu2009,@Schwanhausser2011,@Silva2016].
  
  The debate on which step of the gene expression pathway contributes most is ongoing, nevertheless an understanding has been reached that the cellular context at is a major determinant. At steady state mRNA levels seem to explain protein abundance best, however in perturbed systems the contribution of transcript abundance is shifted away to other steps [@Liu2016].  For example in a study that challanged immune cells, protein levels were dependent on cellular transcript levels [@Jovanovic2015]. In contrast a study investigating cells under stress observed extensive modulation at the protein levels, whereas mRNA transcript abundance was only mildly affected [@Cheng2016].
  
  While the contribution of different steps of the gene expression is dependent on many different factors, e.g. cellular state, treatments, mRNA translation (synthesis of proteins) is an essential process of this pathway. Furthermore, dysregulation of mRNA translation has been observed in a plethora of diseases, ranging from neurological disorders to cancer[@Kapur2018,@Ruggero2013]. This thesis will focus on the role of mRNA translation in the context of cancer.
\clearpage

\section{mRNA translation}

 mRNA translation is the most energy demanding process in the cell and is therefore tightly linked to the energy metabolism [@Buttgereit1995]. Translation plays major roles in cellular functions such as prolifertion (increase cell count) and growth (expand cell size) and deregulation thereof is often found in cancer (**see section \ref{translationInCancer}**) but also associated with other diseases [@Graff2009,@Topisirovic2011,@Lee2021]. To make inferences on  differences in mRNA translation we capture the transcriptome available in the cytoplasm as well as the portion of mRNAs that is actively translated. This enables us to interrogate two prominent players of gene expression at once. 
    
\subsection{overview of an mRNA}
After transcription primary RNA transcripts are processed into mRNAs which is the product that will be translated into proteins. The coding region of an mRNA is flanked by untranslated regions (5' and 3' UTRs) that exert translational control over the mRNA (see \ref{UTR}). The 5' has a cap that is important for mRNA translation initiation[@Grifo1983], while the 3' end has a poly-A tail protecting the mRNA against degradation [@Wilusz2001]. Multiple different mRNAs (isoforms or transcript variants) from the same genomic region exist. These variants can arise due to a process called alternative splicing which alters the exon composition (i.e. coding region) of an mRNA. These variants can co exist and have distinct properties and can perform distinct functions [@JolyAnne-Laure2018]. 
\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/UTRFeatures.pdf}
  \caption{ An mRNA consists of a coding sequence (green), 5' (light orange) and 3' (purple) untranslated regions flanking the coding sequence, a 5' cap and a poly-A tail. Located within the 5' and 3' untranslated region are cis elements (light red boxes) that can exert translational control by interfering with ribosomal movement along the mRNA or interact with trans factors (blue and red) or recruit the 43s ribsome (dark green) to the mRNA (see section \ref{UTR}). The codon composition of the coding sequence also influences translation (see section \ref{tRNA}).   
 \label{fig:UTRFeat}}
\end{wrapfigure}
\clearpage
\subsection{Translation of an mRNA}
  For the vast majority of protein coding mRNAs, eukaryotic mRNA translation occurs in the cytoplasm, however a small subset of mRNAs is translated in the mitochondria. mRNA translation is a process that includes initiation, elongation, termination and ribosome recycling and is an essential process **see Figure \ref{fig:doodlemRNASteps}**. During the initiation phase a ribome will associate with the mRNA and starts scanning along the mRNA for a start codon to begin synthesis of the polypeptide chain by incorporating amino acids. The order by which amino acids are incorporated are dictated by subsequent codons of the open reading frame (ORF).  Redundancy in the codon availability allows that amino acids have multiple assigned codons, for example lysine is encoded by AAA and AAG.   Amino acids are transferred to the ribosome by specialised RNAs called transfer RNAs (tRNA) that can recognise the genetic code in the mRNA. The availability of tRNAs as well as their modification can influence the rate of elongation (see \ref{tRNA}) Once the ribosome encounters a stop codon translation will terminate and the polypeptide chain will be released. The ribosome then disassociates from the mRNA and the ribosome can be recycled to engage translation of the same or another mRNA. The following setions describe these processes in more detail. 

\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/doodleTranslation.pdf}
  \caption{mRNA translation initiation, elongation, termination and ribosome recycling steps - The ribosome binds to the mRNA and initiates scanning for a start codon. The elongation phase incorporates amino acids into a polypeptide chain (i.e. the protein product). Once the end of the coding sequence is detected, by recognition of stop codons, the ribosome terminates translation and releases the polypeptide chain. The ribosome can then be recycled to participate in the translation of another mRNA or reinitiate. \label{fig:doodlemRNASteps}}
\end{wrapfigure}
\clearpage

\subsection{initiation} \label{initiation}

For the initiation to commence, in eukaryotes, two complexes are required; the pre-initiation complex (PIC) and the eukaryotic initiation factor 4F (eIF4F) complex. Both these complexes are governed by signalling pathways that regulate their availability dependent on cellular cues. The PIC consists of the methionyl-initiatior transfer RNA (met-tRNAi) in a ternary complex (TC) with guanosine triphosphate (GTP) bound eIF2 [@Asano2000]. eIF4F is the translation initation complex containing three EIFs; eIF4E, the 5' cap binding protein, eIF4G a scaffold protein and eIF4A and RNA helicase [@Grifo1983,@Hinnebusch2006]. eIF4F recruites the PIC to the 5' cap of the mRNA after which scanning for a start codon (AUG) occurs. After AUG recognition  eIF2-GTP  is  hydrolyzed  forming  a  stable  48S  PIC.  Afterrelease  of  eIF2-GTP  the  60S ribosomal  subunit  joins  to  form  the  80S  ribosome  and  protein  synthesis can  commence (**Figure \ref{initiation}**).

\begin{wrapfigure}{o}{.6\textwidth}
  \includegraphics{./figures/initiation.jpg}
  \caption{Pathway of eukaryotic translation initiation via ribosomal scanning.
 \label{fig:initiation}}
\end{wrapfigure}

\subsection{elongation}

The 80S ribosome contains three sites important for decoding an mRNA; the acceptor (A), peptidyl (P) and Exit (E) sites. During elongation in eukaryotes animoacytelated tRNAs are delivered to the A-site in a ternary complex with eukaryotic elongation factor 1A (eEF1A). When the tRNA recognises its cognate codon and pairs, a bond between the amino acid and the polypeptide chain is formed. The formation of the bonds causes the ribosomal units to rotate in relation to each other [@Munro2007,@Moazed1989]. The rotation causes a shift of the tRNA acceptor ends from the A and P to the P and E sites, wheras the codon end remains in the A and P site. This is the "hyrbid" state of the tRNAs in the ribosome[@Dorner2006]. eEF2 then promotes the translocation by which the codon ends of the tRNA follow into the P and E sites. The deacytelated tRNA is then released from the ribosome. This process is repeated until a stop codon (UAA, UGA or UAG) is detected by the ribosome which terminates translation.

\begin{wrapfigure}{o}{.6\textwidth}
  \includegraphics{./figures/elongation.jpg}
  \caption{Model of the eukaryotic translation elongation pathway.Thomas E. Dever, and Rachel Green Cold Spring Harb Perspect Biol 2012;4:a013706. Reprinted with permission. ©2012 by Cold Spring Harbor Laboratory Press
 \label{fig:elongation}}
\end{wrapfigure}

\clearpage

\subsection{termination and recycling}
mRNa translation termination is  facilitated  by  two  release  factors (eRF), eRF2  and  eRF3-GTP[@Stansfield1995,@Alkalaeva2006]. A TC  containing  eRF2  and  eRF3-GTP binds to the A-site of the ribosome upon recognition of a stop codon. This causes an ATP hydrolysis event resulting in a conformational change and release of the polypeptide chain. eRF1 and the ATP binding cassette protein (ABCE1) together promote  the  splitting  of  the  60S  and  40S  subunits after which they can be recycled[@Pisarev2010,@Dever2012,@Hellen2018]. 

\subsection{Translation efficiency}

Each ribosome synthesises a single protein during translation of an mRNA from start to end. It has been known since the '60s that translation of an mRNA occurs via multiple bound ribosomes (polysomes) simultaneaously (**see figure \ref{fig:polysomes}**)[@Warner1962a,@Staehelin1963a]. The translation efficiency of an mRNA depends on the number of ribosomes it is associated with synthesising proteins and is frequently approached using the totally asymmetric simple exlcusion process (TASEP) [@MacDonald1968a,@Maniloff1969] (**See figure \ref{fig:polysomes}**). 
mRNA translation accounts for roughly 20% of the cellular energy expenditure [@Buttgereit1995]. While all steps of mRNA can affect the translation efficiency of an mRNA, it is most commonly regulated at the initiation step. Such regulation can be exerted for many transcripts at once (i.e. globally) or for mRNAs that share certain characteristics (selective) (**see section \ref{regmRNA}**).

\begin{wrapfigure}{l}{.6\textwidth}
  \includegraphics{./figures/polysome.pdf}
  \caption{Electromicrongraph of ribosomes extracted from different positions along a sucrose gradient used for polysome fractionation (A-C). For details on polysome fractionation see section \ref{exptMethod}. Reprinted with permission. DR. T. STAEHELIN et. al. Nature.1963 Aug 31;199:865-70.doi: 10.1038/199865a0. Copyright © 1963, Nature Publishing Group.
  Schematic of the TAPES method - A ribosome (A) can initiate with rate \(\alpha\) on an mRNA with a coding sequence with codons i= 1 ... L. The elongation rate at a specific codon is defined by \(\omega_i\) and \(\beta\) determines the termination rate once a stop codon is encountered. TASEP has been constantly modified to allows for correction of stalling ribosomes, i.e. a ribosome cannot initiate or elongate when the target codon is already occupied. Reprinted with permission. Juraj Szavits-Nossan and Martin R. Evans. 10.1103/PhysRevE.101.062404. ©2020 American Physical Society. 
 \label{fig:polysomes}}
\end{wrapfigure}

\section{Regulation of mRNA translation} \label{regmRNA}

Of the heretofore presented steps of mRNA translation the initiation is the most regulated step. From the perspective that mRNA translation contributes most to the cellular energy consumption, of which the initiation requires ATP, this process ought to be strictly regulated [@Buttgereit1995,@Jackson1991,@Sonenberg2009]. Nevertheless, translation can also be regulated at the elongation [@Richter2015] and termination[@Dever2012] phases albeit to a lesser extent. Dynamic modulation of mRNA translation can be achieved through signalling pathways as well as several distinct cis and trans elements in both untranslated regions (UTRs) of an mRNA (**Figure \ref{fig:UTRFeat}**). 

\subsection{Global regulation of mRNA translation} 

\paragraph{mTOR} \label{mTOR} is  a  conserved  Ser/Thr  kinase  and  is  found  in  two  structurally  and  functionally  distinct  complexes,  mTORC1  and mTORC2. [@Saxton2017,@Pearce2007]. mTOR activity is modulated via growth factor signalling (i.e. insulin and insulin-like growth factor; IGF1) as well as cellular metabolism. While mTORC1 down stream signalling impinges on differention and growth, via phosphorylation of 4E-BPs and S6k). mTORC2 promotes survival via signalling through protein kinase A (AKT). 



is centered around the energy status of the cell sensed through the ratio of adenosine-mono-phosphate (AMP) to adenosine-tri-phosphate (ATP) that actives the AMP-kinase (AMPK) [@Sanders2007,@Kimball2006]. Increase in AMPK activity leads to an inhibition of mTOR and therefore down regulation of mRNA translation initiation[@Inoki2003]. Nutrient availability (i.e. glucose and amino acids), oxygen levels are interwoven with cellular energy production by affecting the AMP/ATP ratios in the cell [@Heerlein2005].  The role of cellular metabolism, of which its deregulation is considered a hallmark of cancer, on mTOR activity suggests that mRNA translation plays an important role in this disease (see \ref{translationInCancer}.  

\begin{wrapfigure}{l}{6\textwidth}
  \includegraphics{./figures/mTORsignal.jpg}
  \caption{Schematic representation of mTOR signaling to the translational machinery. Philippe P. Roux, and Ivan Topisirovic Mol. Cell. Biol. 2018; doi:10.1128/MCB.00070-18. Reprinted with permission. Copyright © 2018, American Society for Microbiology
 \label{fig:mtorsignal}}
\end{wrapfigure}
\paragraph{The relationship between mTOR and metabolsim} 
\paragraph{Global regulation of translation via mTOR} \label{regmtor} is achieved by phosphorylation of 4E-BP1 which facilitates formation of the eIF4F mRNA translation complex by regulating eIF4E availability (**see section \ref{initiation}**). 
  
\paragraph{The integrated stress response} is a signalling pathway which can be activated through kinase signalling orginating from various stress signals. These kinases include Protein kinase R-like endoplasmic reticulum kinase (PERK) which is activated by misfolded peptides in the endoplasmatic reticulum (ER), Heme regulated eIF2alpha kinase (HRI) which is activated during oxidative stress, protein kinase R (PKR) which is activated in response to certain viral infections and GCN2 which is activated when cells are deprived of amino acids[@Kapur2017,@Guan2017,@Taniuchi2016,@Andreev2015]. During the integrated stress response the alpha subunit of eIF2 is phosphorylated. Upon eIF2alpha phosphorylation, eIF2 alpha directly engages the guanine nucleotide exchange factor eIF2beta and precents conversion of inactive eIF2-GDP to  active eIF2-GTP needed for met-tRNAi incorpration in the TC, therefore inhibiting translation by reducing PIC availability [@Sonenberg2009].

\paragraph{Regulation of translation via the ISR} is, similar to mTOR signalling, achieved at a global and selective mRNA level. Since the phosphorylation of eIF2 alpha limits ternary complex availability ribsome recruitment to the 5' cap is limited. While global translation is reduced upon ISR, translation of a selective subset of mRNA with upstream open reading frames (uORFs) is increased. A uORF is a reading frame that originates in the 5' UTR of an mRNA upstream of which the AUG precedes that of the coding sequence. uORFs are out of frame with the main ORF and when translated lower the expression of the mORF[@Kozak1984]. Ribosome profiling studies indicate that 50% of mammalian mRNAs harbour uORFs including oncogenes and transcripts important in differentiation and cell cycle [@Calvo2009,@Ingolia2011,@Morris1995]. The surrounding context of the uORF is important for its inhibitory effect through more efficient initiation, where the classical Kozak context is most efficient[@Kozak1986,@Calvo2009]. ATF4, a transcription factor for stress response genes, contains two uORFs of which one partially overlaps with the mORF. Under normal conditions ATF4 mRNA translation is initiated at uORF1 and reinitiation at uORF2 occurs. The close proximity of uORF2 to the mORF causes ribosomes to scan past the mORF start thereby inhibiting the translation of the coding sequence. Limitiation of TC availability during ISR causes longer ribosome scanning times leading to that ribosomes scan past uORF2 and initiate at the mORF[@Pakos-Zebrucka2016]. 

\subsection{Selective regulation of mRNA translation} \label{UTR}

\paragraph{mTOR sensitive} selective modulation of mRNA translation is dependent of 5' UTR characteristics of individual mRNAs. Transcripts with a terminal oligo pyrimidine (TOP) motif consisting of a C followed by a stretch of 4-15 pyrimidines directly after the 5' cap show near complete dissociation from ribosomes under conditions when mTOR is inhibited[@Yamashita2008]. TOP mRNA are enriched for genes encoding for parts of the translation machinery [@Thoreen2012]. Recent work indicates the importance of La ribonucleoprotein domain family member 1 (LARP1) herein. LARP1 is thought to bind to the 5' mRNA cap of TOP mRNAs via its DM15 domain and represses translation by obstructing eIF4E binding. mTORC1 physically interacts and phosphorylates LARP1. When phophorylation occurs close to the DM15 domain of LARP1 the inhibitory effect on mRNA translation of TOP mRNAs is abolished [@Jia2021]. Other instances of selective translation are for mRNAs that lack the TOP motif, but show sensitivity to mTOR activity. These mRNAs are, in addition to mTOR, dependent on either eIF4E or eIF4A (**see \ref{eif4a}**). mTOR-eIF4E sensitive mRNAs show extremely short 5' UTRs and encode for metabolic functions[@Gandin2016a]. 

\paragraph{eIF4A sensitive mRNAs} \label{eif4a}
  Progression of the ribosome is dependent of on eIF4A, the RNA helicase in the eIF4F complex, that unwinds structural elements the ribsome encounters[@Wolfe2014,@Rubio2014]. The importance of eIF4A's unwinding capacity was identified by treatment of KOPT-K1 cells, a lymphoma cell line, MDA-MB-231 ,a breast cancer cell line, with silvesterol of which translational control was evaluated using ribosome prolfing (**see section \ref{riboseq}**)[@Wolfe2014,@Rubio2014]. These studies identified silvesterol sensitive mRNAs that are characterised by long and structured 5' UTRs. In addition eIF4A dependent mRNAs were enriched for having multiple 5' UTR variants, while independet mRNAs were not [@Rubio2014]. Among eIF4A dependent mRNAs are genes important for proliferation and survival which is implications for oncogenic signaling[@Wolfe2014,@Gandin2016a]. In a different context of eIF4A inhibition using rocaglates polypurine sequences in the 5' UTR were of importance. Here, rocaglates would clamp eIF4A to the mRNA causing a road block for ribosome scanning and premature translation initiation [@Iwasaki2016].

\subsection{RNA binding proteins and trans-acting factors}

The UTRs of an mRNA contain sequence elements to which RNA and RNA binding proteins (RBPs) bind and exert translational regulation. For instance, MicroRNAs, a small class of non coding RNA, can directly bind to other RNAs and silence them accomplished through translational repression or, more often, destabilisation [Jonas2015]. RBPs are a class of proteins involved in many regulatory steps of gene expression and account for 7.5% of the protein coding genes. Poly-A-binding-protein (PABP) is thought to form a closed loop complex of the 3' end to the 5' by interacting with eIF4G. This closed loop should promote translation and prevent mRNA decay [@Afonina2014,@Amrani2008]. An RBP of particular interest in **study 3** is Human antigen R (HuR). HuR preferentially binds to AU-rich sequences in the 3' UTR and acts as a stabilizing agent and is involved in RNA-processing [@Levine1993,@Baou2011,@Fan1998,@Peng1998]. Studies in breast, colon and lung cancer observed correlation between HuR and malignancy. Among HuR targets are HIF-1, VEGF (important for angiogenesis) and the oncogene Myc [@Denkert2004,@LopezdeSilanes2003,@LopezdeSilanes2005].
  
  
\subsection{Regulation of mRNA translation by tRNAs} \label{tRNA}

is a balance between supply and demand of charged tRNAs of actively translated codons. This concept is also referred to as codon optimality. 

\section{mRNA translation in cancer} \label{translationInCancer}



\section{Regulatory modes of gene expression}
 EXPAND EVOLUTION - NATURE PAPER COMENSATORY EFFECTS ETC
In transcriptome-wide studies of translation efficiencies the interplay between total mRNA and translated mRNA levels are interrogated. Traditionally it was thought that changes in translation efficiencies lead to altered protein levels. A change in translation efficiency is observed for mRNAs whose polysome-association is altered whereas their total mRNA does not change to a similar magnitude as the polysome-association (I.e. change in translation). An example thereof is TOP mRNA translation under conditions where mTOR is stimulated [@Masvidal2017]. 
\begin{wrapfigure}{l}{0.6\textwidth}
  \includegraphics{./figures/geneModes_MRNA.pdf}
  \caption{Regulatory modes of gene expression - Schematic representation of regulatory modes of translation efficiency in a fold-change scatter plot. Indicated in red are changes in translation (i.e. changes in translated mRNA but not total mRNA), in green changes in mRNA abundance (i.e. congruent changes between total mRNA and translated mRNA) and in blue translational buffering (i.e. changes in total mRNA levels but not translated mRNA levels). TE changes as the TE-score would estimate them are indicated.\label{fig:mRNA}}
\end{wrapfigure}
In recent years, evidence emerged where translation efficiencies of mRNAs can be altered to compensate for changes in total mRNA levels. Within this newly identified mode of regulation of mRNA translation “translational buffering”, mRNA translation is altered such that changes in total mRNA levels do not influence their corresponding protein levels [@Oertlin2019,@McManus2014,@Lorent2019]. Translational buffering is observed to compensate for inter-tissue, inter-species and inter-individual difference [@Artieri2014,@Cenik2015,@Perl2017]. Furthermore, in bacteria translational buffering maintains protein complex stoichiometry as well as protein levels for conserved pathway across species [@Li2014,@Lalanne2018]. Recently translational buffering has been observed under conditions where estrogen receptor alpha (ERalpha) is depleted. ERalpha modulates activity of specific tRNA modification enzymes. These enzymes are needed for the U34 tRNA modification. Loss of ERalpha led to reduced U34 tRNA modification thereby hindering translation of mRNAs requiring such modified tRNAs.  For these mRNAs, even though their total mRNA levels were induced across conditions, their protein levels remained constant [@Lorent2019]. Given these multiple roles of mRNA translation to regulate the proteome it is critical to distinguish them as their underlying mechanisms can have different biological implications.



\section{Expertimental methods to measure mRNA translation} \label{exptMethod}

\subsection{Polysome profiling} is a technique to measure changes in translational efficiencies of mRNAs between two or more conditions. Polysome profiling allows for separation of polysomes from monosomes, ribosomal subunits and messenger ribonucleoprotein particles (mRNPs). During the assay, ribosomes are immobilized on the mRNAs using translation elongation inhibitors (e.g. cycloheximide). Cytoplasmic RNA extracts are then sedimented on a linear sucrose gradient (5-50%) using ultra centrifugation. 
\begin{wrapfigure}{r}{0.6\textwidth}
    \includegraphics[width=0.9\linewidth]{./figures/polysome_shifts.pdf}
  \caption{Polysome profliles -  (top left) Schematic representation of a polysome profile using linear sucrose gradient fractionation. Indicated in the polysome profiles are the 40S, 60S ribosomal subunits as well as the 80S monosome. H.P. indicates heavy polysome fractions.Between conditions distribution changes for mRNA abundance (top right), translation (bottom left) and translation within high polysome fractions (bottom right) are illustrated. \label{fig:polysome}}
\end{wrapfigure} 
The resulting gradient is fractionated and mRNAs with different number of bound ribosomes can be extracted and analyzed for changes in translational efficiency [@Gandin2014]. An illustration of a polysome profile with peaks for the 40S, 60S subunits and 80S ribosome can be seen in (**Fig \ref{fig:polysome} top left**). Subsequent peaks along the frations indicate the mRNAs with 1 or more bound ribosome. mRNAs are typically normally distributed along the fractions, i.e. a pool of the same mRNA will be associated with 1- n number of ribosomes. Changes in mRNA abundance will lead to an overall increase in the amount of isolated polysome-associated mRNA without a shift of the distribution along the fractions (**Fig \ref{fig:polysome} top right**). This means that the translation efficiency per mRNA remains unchanged. Changes in translational efficiency  can be observed by shifts of polysome association for mRNAs from the light (inefficiently translated) towards the heavy (efficiently translated) polysome fractions or vice versa (**Fig \ref{fig:polysome} bottom left**). Shift within the heavy polysome fractions (i.e. 3 bound ribosome to 7 bound ribosome) can also occur (**Fig \ref{fig:polysome} bottom right**). These shift remain undetected in cases where the distribtion of polysome-associated mRNAs does not sufficiently shift across the fractions and is a limitation of polysome profiling. Quantification of mRNA levels within each fraction can be assessed using Northern blotting or reverse transcription quantitative polymerase chain reaction (RT-qPCR).
For transcriptome wide studies, pooling of efficiently translated mRNAs (mRNAs with >3 bound ribosomes) followed by quantification using either DNA-microarrays or RNA sequencing is common. Pooling of mRNAs as well as collection of multiple fractions makes polysome profiling inconvenient when dealing with large samples sizes or experiments with low amounts of input RNA. Therefore, an optimized sucrose gradient was developed where most efficiently translated mRNAs are collected on a sucrose cushion and thereby can be isolated from one single fraction [@Liang2018]. This optimized gradient allows for application of polysome profiling in small tissue samples where RNA quantity is limiting and reduces labor intensity of the assay.
Polysome-associated mRNA levels are subject to changes in translation efficiency as well as factors contributing to cytosolic mRNA levels. Mechanisms such as transcription (i.e. in the case of mRNA abundance) or mRNA stability can affect cytosolic mRNA levels which impacts the pool of mRNAs that can be associated to polysomes. Therefore, to identify bona fide changes in translation efficiency it is important to collect cytoplasmic mRNA levels in parallel to polysome-associated mRNA to correct for such mechanisms (e.g. transcription or mRNA stability) during downstream analysis [@Gandin2014,@Oertlin2019]. 

\subsection{Ribosome profiling} \label{riboseq}
is a technique that enables sequencing of ribosome protected mRNA fragments (RPFs). In the assay ribosomes are immobilized on the mRNAs using, similar to polysome profiling, translation elongations inhibitors (e.g. cyclohexamide) [@Ingolia2010,@Ingolia2016]. One limitation with the use of translation elongation inhibitors is the distortion of ribosome distributions especially at translation initiation sites. These introduced artefacts need to be accounted for in the downstream analysis when assessing ribosome position along the mRNA. Following the translation elongation inhibitor treatment, cells ought to be immediately flash frozen using liquid nitrogen. Alternatively, using only flash freezing has been seen as a robust approach in a wide range of diverse organisms [@Brar2015]. Generation of RPFs is achieved by RNAse treatment breaking the links of RNA between ribosomes leaving single ribosomes with a ~28 nucleotide long RNA fragment within each ribosome. RPFs are then isolated using ultra centrifugation through a sucrose cushion. Co-migration of RNA fragments such as structured non-coding RNAs or large ribonucleoprotein complexes within the sucrose gradient can be a cause of contamination and thereby can provide false readouts of translation. A polyacrylamide gel loaded with RPFs and a reference ladder is used to select RPFs of the right size. Typically, RPFs with lengths of 25 to 30 nucleotides are selected. The RPFs can then be pooled if sample specific barcodes are used. After size selection a pre-adenylated DNA is ligated to the RPFs. This RNA-DNA construct is then used as template for reverse transcription. Through gel-based purification, full-length products of the reverse transcription are selected and circularized. Following circularization, a double stranded DNA library is constructed and PCR amplified. This library is suitable for quantification using RNAseq. In parallel to RPF selection, randomly fragmented total mRNA of the same size is also retrieved. This is achieved by extraction of total mRNA from cell lysate followed by purification via recovery of polyadenylated messages or removal of ribosomal RNA. Fragmentation of total RNA is done using an alkaline fragmentation buffer [@Ingolia2010,@Brar2015].

\subsection{Comparing ribosome and polysome profiling}
Albeit both methods generate count data after quantification with RNAsequencing, there are some key aspects that differ between the techniques. Polysome profiling separates efficiently translated mRNAs from non- efficiently translated mRNAs along a sucrose thereby creating an mRNA based perspective for analyzing changes in translational efficiencies. In contrast, ribosome profiling determines translational efficiencies by counting the number of RPFs of both efficiently and non-efficiently translated mRNAs. Changes in translational efficiencies, e.g. shifts between the polysomal fractions, can be dramatic (I.e. near complete dissociation of ribosomes from an mRNA) or subtle (shifts from 2 to 4 ribosomes) [@Livingstone2015]. Ribosome profiling has been shown to be biased towards identification of dramatic shifts of associated ribosomes to mRNAs, whereas subtle shifts are masked which can lead to false biological conclusions. Polysome profiling is affected by this to a much lesser extent, thereby more robust in identifying such changes [@Masvidal2017]. RPFs in ribosome profiling provide exact nucleotide positions occupied by ribosomes thereby offering single nucleotide resolution. Polysome profiling cannot reveal ribosome locations along the mRNA. However, polysome profiling allows access to full-length mRNAs that includes the UTRs. The single nucleotide resolution of ribosome profiling is necessary in contexts studying local translation events such as ribosomal frame shifts [@Rato2011] or uORF translation [@Andreev2015]. Higher sensitivity in detecting changes in translational efficiencies on a global scale makes polysome profiling more suitable for transcriptome-wide studies [@Gandin2016]. Both methods have their strengths and weaknesses and therefore each method should be considered depending on the underlying biological question of each experiment.

\section{Algorithms for analysis of changes in translation efficiencies}

In polysome-profiling and ribosome profiling translated mRNAs (i.e. polysome-associated mRNAs and RPFs) are separated in parallel from their total mRNA counterpart. Estimating translation efficiencies requires that changes in translated mRNA are corrected for changes in total mRNA to accurately identify the regulatory modes of gene expression (i.e. translation, mRNA abundance and translational buffering)[@Oertlin2019]. Here we will discuss methods that analyse polysome-profiling and ribosome profiling data to estimate changes in translation efficiencies across 2 or more conditions and how these methods identify different regulatory modes of gene expression.

Initially analysis of transcriptome-wide translation studies used an approach called the translation efficiency (TE-score) that uses the following equation: 
\[\varDelta TE = \frac{\frac{P_{c2}}{T_{c2}}} {\frac{P_{c1}}{T_{c1}}}\\\]

This score calculates the ratio of the ratios between polysome-associated mRNA levels (P) divided by total mRNA levels (T) within each condition (i.e. C1 and C2). The TE- score approach has been shown to be prone to spurious correlations [@Larsson2010]. Spurious correlations arise due to that the ratio of polysome-associated mRNA and total mRNA can systematically correlate with total mRNA levels which is not corrected for in this equation and leads to an elevated type-1 error.
\clearpage
\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_TE.pdf}
  \caption{TE scores for regulatory modes of gene expression -  Schematic representation of regulatory modes of translation efficiency in a fold-change scatter plot. Indicated in red are changes in translation efficiency altering protein levels, in green changes in mRNA abundance and in blue changes in translation efficiency leading to translational buffering/offsetting. The shifts for the translation efficiency (TE) score are indicated. \label{fig:TE}}
\end{wrapfigure}
**Figure \ref{fig:TE}** gives an overview of the relationship between a change in  TE) and each regulatory mode of gene expression. Changes in mRNA abundance will lead to a \(\varDelta\)TE close to 0 in log space (i.e. no change) as total mRNA and translated mRNA change with a similar magnitude. However, in the case of both translation and translational buffering, terms in the TE-score equation change leading to a \(\varDelta\)TE (TE < 0 or TE > 0) and thereby identification of both changes in translation and translational buffering simultaneously. Therefore, the TE-score method fails to differentiate between changes in translation and translational buffering which can have drastic consequences for the biological interpretation of the results [@Oertlin2019].

The TE-score approach was challenged by the Analysis of Translation Activity (anota) algorithm which was developed for DNA-microarray data [@Larsson2011]. anota combines analysis of partial variance (APV)[@Schleifer1993] with a random variance model (RVM)[@Wright2003]. RVM estimates gene variance using shared informatio across all genes to increase power for detection of differential expression[@Wright2003]. anota uses a two-step process that firstly assesses the model assumptions for (i) absence of highly influential data points, (ii) common slopes of sample classes, (iii) homoscedasticity of residuals and (iv) normal distribution of per gene residuals. In the second step then performs analysis of changes in translational activity using the following model:

\[log(y_{gi}) = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 
  
here \(\beta_g^{RNA}\) described the relationship to total RNA  \(gth\) gene \(ith\) sample of model matrix \(X\); \(\beta_g^{cond}\) represent the log2 fold change for treatment classes and \(\varepsilon_{gi}\) denotes the residual error. 

\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_anota_Larsson.pdf}
  \caption{anota gene models - Schematic representation of the anota analysis models. Translation mRNA expression is set out against total mRNA expression for each biological replicate and treatment condition. Top left shows the model of a gene that is differentially translated (i.e. change in translated but not total mRNA). The difference in the slope intercepts are used to estimate changes in translation efficiencies between conditions i.e. dTE. Other gene models are shown; change in translation efficiency with varying total mRNA levels (top right); change in mRNA abundance (bottom left) and translational buffering (bottom right).
  \label{fig:anota}}
\end{wrapfigure}

Within anota a common slope for the treatment classes that describes the translated mRNA to total mRNA relationship is calculated. The difference between the slope intercepts is then interpreted as the \(\varDelta\) TE. A simplified view of this model can be seen in (**Figure \ref{fig:anota} top left**). Here expression for translated mRNA and total mRNA are modeled over two sample classes with each 4 replicates. Furthermore, changes in translation efficiencies can also be observed when translated mRNAs shift to a larger extent than the total mRNA levels (**Figure \ref{fig:anota} top right**). Identification of genes in this categorie can be a challenge, especially in highyl variable data set, as they resemble mRNA abundance genes (**Figure \ref{fig:anota} bottom left**). Nevertheless, Using the linear regression analysis anota accurately corrects changes in translated mRNA as can be seen in (**Figure \ref{fig:anota} bottom right**) where a change in total mRNA   but not translated mRNA levels is observed. For this gene the difference in slope intercepts is small and will not be identified as difference in translation as would be the case in the TE-score approach. anota was developed at a time where translational buffering was uncommonly seen in data sets. Naturally, the methods lacks a setting to analyse translational buffering. This was addressed in anota's successor, anota2seq, and will be discussed in **Study 1**.

Advances in experimental methods warrant for appropriate statistical methods to analyse data resulting from them. DNA- microarray was the dominant platform to assess genome-wide changes before the advent of RNA sequencing. In DNA- microarray RNA hybridizes probes on a chip and generate a signal of which the measured intensity is an indicator of expression, whereas in RNA sequencing reads from RNAs are counted. Intensity data from DNA microarray can be normalised and transformed (i.e. log transformation) to fulfill the requirements for application of linear models, whereas RNA sequencing harbours additional characteristics that need to be accounted for. Therefore, algorithms developed for analysis of DNA- microarray are not directly applicable to RNA sequencing data as is the case for the anota algorithm.

RNA sequencing data shows variance that is greater than the mean which is commonly referred to as overdispersion. Count data from RNA sequencing have been initially approached using Poisson distributions which assumes that the variance is equal to the mean[@Lu2005]. Now established RNA sequencing analysis frameworks such as edgeR and DESeq2  use negative binomial distributions in combination with generalized linear models (GLMs) [@Robinson2010, @Love2014]. The negative binomial distribution uses a dispersion parameter to account for overdispersion [@McCarthy2012]. While analysis principles of DESeq2 and edgeR are similar they differ in their normalisation method, dispersion estimation and information sharing across genes. In a simple differential expression analysis between two conditions with one RNA type the GLM model would be as in the following equation: 

\[log(y_{gi}) = \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 

here \(\beta_g^{cond}\ X_i^{cond}\) represent the condition (i.e. control and treatment) log2 fold change for the \(gth\) gene \(ith\) sample of the model matrix X and \(\varepsilon_{gi}\) denotes the residual error. When analysing changes in translation effiencies additional parameter for RNA type (i.e. total mRNA or translated mRNA) and the interaction between the RNA type and condition are added so that:

\[log(y_{gi}) = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \beta_g^{RNA:cond}\ X_i^{interaction} + \varepsilon_gi\] 

In this model the interaction term is interpreted as the change in translation effiencies [@Chothani2019]. Other methods (i.e. Ribodiff[@Zhong2017], Riborex[@Li2017] and deltaTE [@Chothani2019]) borrow this analysis principle of an GLM with an interaction term by often applying this exact model. A noteable difference is that Ribodiff allows dispersion estimation for translated mRNA and total mRNA separetly as variance differences between the RNA types can be expected due to varying experimental protocols [@Zhong2017, @Liang2018]. While the flexibility of GLMs allows for complex study designs involving 2 or more treatment conditions, Riborex and Ribodiff limit the study design to only two conditions. DeltaTE gives their users full flexibility of the DESeq2 GLM model. Xtail is a method developed for ribosome profiling that makes use of DESeq2 for RNAseq count normalisation [@Xiao2016]. Their assessment of differences in translation efficiencies relies on probability matrices for the ratio of translated mRNA over total mRNA within condition and a between condition ratio of these ratios. Babel was the first algorithm designed solely for analysis of differential translation and uses an error-in-varaibles regression analysis [@Olshen2013]. The error-in- variables regression allows accounting for variable total mRNA levels when assessing changes in translation.  Although these methods have distinct approaches to identify changes in translation efficiencies, their principle of analysis is similar to comparing a ratio of ratios. Therefore these methods suffer from similar issues as the TE-score which will be discussed in **Study 1**.


