---
output: pdf_document
---

\chapter{Introduction}

\section{Gene expression}
\subsection{The central dogma of gene expression}
  The whole genetic code of an organisms i stored as deoxyribonucleic acid (DNA) molecules in a double stranded formation as chromosomes. Chromosomes hold the DNA in a condensed state using chromatin, a complex of DNA and proteins, structures. For transcription of DNA to occur chromatin is remodeled to expose promotor regions in the DNA to which fators assisting in transcription bind. One of these factors is DNA polymerase that unravels the double-stranded DNA and creates a single-stranded copy called ribonucleic acid (RNA) transcripts. This copy is less stable due to its single strandedness and therefore only temporary.  The RNA transcripts undergo processing by which multiple different transcript variants coming from the same genomic region can be produced. The protein coding portion of these transcripts are called mRNAs. Once formed mRNAs are transported into the cytoplasm where they are either degraded or associate with ribosomes. These ribosomes translate the mRNAs into proteins by which the genetic information then is expressed. Synthesised proteins, if no longer needed in the cell, can be degraded by proteosomes (**see figure \ref{fig:geneExprPath}**). This flow of genetic information into expressed proteins is commonly referred to as the central dogma in molecular biology [@Crick1970].

  \clearpage
  \begin{figure}
  \includegraphics{./figures/geneExprPath_2.pdf}
  \caption{The gene expression pathway - DNA is transcribed in pre-mRNA containing a 5' cap (black oval) introns (teal boxes),exons (purple boxes) and a poly(A) tail. RNAs are processed into mRNAs consisting out ouf a 5' cap,exons and a poly(A) tail is then transported out of the cellular nucleus into the cytoplasm. Within the cytosplasm mRNAs can be degraded or translated  into proteins depending on cellular demands. Synthesised proteins can be degraded by proteosomes. \label{fig:geneExprPath}}
\end{figure}
\clearpage
\subsection{Contribution to gene expression}
  Proteins are the last product of the gene expression pathway and carry out the vast majority of all cellular functions. While it is apparent that modulation of protein levels will offer information on the changes in gene expression, it cannot completely answer the question as to why the levels change. In a disease context, protein levels alone might offer sufficient insight to explain phenotypic differences. However, how these differences arise mechanistically, which often poses as target for therapeutic strategies in cancer, is obscured.
  
  Experimental methods to measure gene expression at different steps are often to referred as "omics" (i.e. proteomics for protein expression, transcriptomics for mRNA expression). These methods provide snapshots of the step under scrutiny in a specific context (steady state or perturbation) for a large portion of genes or proteins. Transcriptomics studies approach gene expression with the assumption that mRNA expression results in protein expression changes and can therefore be used as a proxy. However, this view got challenged by landmark studies that observed a poor mRNA to protein correlation and indicated a larger role of post transcriptional regulation in gene expression than previously assumed [@Lu2005,@Vogel2012,@deSousaAbreu2009,@Schwanhausser2011,@Silva2016].
  
  The debate on which step of the gene expression pathway contributes most is ongoing, nevertheless an understanding has been reached that the cellular context is a major determinant. At steady state mRNA levels seem to explain protein abundance best, however in perturbed systems the contribution of transcript abundance is shifted away to other steps [@Liu2016].  For example in a study that challenged immune cells, protein levels were dependent on cellular transcript levels [@Jovanovic2015]. In contrast a study investigating cells under stress observed extensive modulation at the protein levels, whereas mRNA transcript abundance was only mildly affected [@Cheng2016].
  
  While the contribution of different steps of the gene expression is dependent on many different factors, e.g. cellular state or treatments, mRNA translation (synthesis of proteins) is an essential process of this pathway. Furthermore, dysregulation of mRNA translation has been observed in a plethora of diseases, ranging from neurological disorders to cancer which warrants for a comprehensive understanding of this process [@Kapur2018,@Ruggero2013,@Lee2021,@Graff2009]. This thesis will focus on the role of mRNA translation in gene expression in the context of cancer.
\clearpage

\section{mRNA translation}
\subsection{Overview of an mRNA}
After transcription primary RNA transcripts are processed into mRNAs which is the product that will be translated into proteins. The coding region of an mRNA is flanked by untranslated regions (5' and 3' UTRs) that exert translational control over the mRNA (see \ref{regmRNA}). The 5' has a cap that is important for mRNA translation initiation[@Grifo1983], while the 3' end has a poly-A tail protecting the mRNA against degradation [@Wilusz2001]. Multiple different mRNAs (isoforms or transcript variants) from the same genomic region exist. These variants can arise due to a process called alternative splicing which alters the exon composition (i.e. coding region) of an mRNA. These variants can co-exist at the same time and have distinct properties and can perform distinct functions [@JolyAnne-Laure2018]. 
\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/UTRFeatures.pdf}
  \caption{ An mRNA consists of a coding sequence (green), 5' (light orange) and 3' (purple) untranslated regions flanking the coding sequence, a 5' cap and a poly-A tail. Located within the 5' and 3' untranslated region are cis elements (light red boxes) that can exert translational control by interfering with ribosomal movement along the mRNA or interact with trans factors (blue and red) or recruit the 43s ribsome (dark green) to the mRNA (see section \ref{regmRNA}). The codon composition of the coding sequence influences translation elongation rates (see section \ref{tRNA}).   
 \label{fig:UTRFeat}}
\end{wrapfigure}
\clearpage
\subsection{Translation of an mRNA}
  For the vast majority of protein coding mRNAs, eukaryotic mRNA translation occurs in the cytoplasm, however a small subset of mRNAs is translated in the mitochondria. mRNA translation is a process that includes initiation, elongation, termination and ribosome recycling and is an essential process **see Figure \ref{fig:doodlemRNASteps}**. During the initiation phase a ribosome will associate with the mRNA and starts scanning along the mRNA for a start codon to begin synthesis of the polypeptide chain by incorporating amino acids. Amino acids are transferred to the ribosome by specialised RNAs called transfer RNAs (tRNA) that can recognise the genetic code in the mRNA. The availability of tRNAs as well as their modification can influence the rate of elongation (see \ref{tRNA}). The order by which amino acids are incorporated is dictated by the order of the codons of the open reading frame (ORF).  Redundancy in the codon availability allows that amino acids are encoded by codons, for example lysine is encoded by AAA and AAG.    Once the ribosome encounters a stop codon translation will terminate and the polypeptide chain will be released. The ribosome then disassociates from the mRNA and the ribosome can be recycled to engage translation of the same or another mRNA. The following sections describe these processes in more detail. 

\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/doodleTranslation.pdf}
  \caption{mRNA translation initiation, elongation, termination and ribosome recycling steps - The ribosome binds to the mRNA and initiates scanning for a start codon. The elongation phase incorporates amino acids into a polypeptide chain (i.e. the protein product). Once the end of the coding sequence is detected, by recognition of stop codons, the ribosome terminates translation and releases the polypeptide chain. The ribosome can then be recycled to participate in the translation of another mRNA or reinitiate. \label{fig:doodlemRNASteps}}
\end{wrapfigure}
\clearpage

\subsection{Initiation} \label{initiation}

For the initiation to commence, in eukaryotes, two complexes are required; the pre-initiation complex (PIC) and the eukaryotic initiation factor 4F (eIF4F) complex. Both these complexes are governed by signalling pathways that regulate their availability dependent on cellular cues (**see section \ref{regmRNA}**). The PIC consists of the methionyl-initiatior transfer RNA (met-tRNAi) in a ternary complex (TC) with guanosine triphosphate (GTP) bound eIF2 [@Asano2000]. eIF4F is the translation initation complex containing three eIFs; eIF4E, the 5' cap binding protein, eIF4G a scaffold protein and eIF4A and RNA helicase [@Grifo1983,@Hinnebusch2006]. eIF4F recruites the PIC to the 5' cap of the mRNA after which scanning for a start codon (AUG) occurs. After AUG recognition  eIF2-GTP  is  hydrolyzed  forming  a  stable  48S  PIC.  After release  of  eIF2-GTP  the  60S ribosomal  subunit  joins  to  form  the  80S  ribosome  and  protein  synthesis can  start (**Figure \ref{initiation}**).

\subsection{Elongation}

The 80S ribosome contains three sites important for decoding an mRNA; the acceptor (A), peptidyl (P) and Exit (E) sites. During elongation in eukaryotes aminoacytelated tRNAs are delivered to the A-site in a ternary complex with eukaryotic elongation factor 1A (eEF1A). When the tRNA recognises its cognate codon and pairs, a bond between the amino acid and the polypeptide chain is formed. The formation of the bonds causes the ribosomal units to rotate in relation to each other [@Munro2007,@Moazed1989]. The rotation causes a shift of the tRNA acceptor ends from the A and P to the P and E sites, whereas the codon end remains in the A and P site. This is the "hyrbid" state of the tRNAs in the ribosome[@Dorner2006]. eEF2 then promotes the translocation by which the codon ends of the tRNA follow into the P and E sites. The deacytelated tRNA is then released from the ribosome. This process is repeated until a stop codon (UAA, UGA or UAG) is detected by the ribosome[@Hellen2018]].

\begin{figure}[ht]
\includegraphics{./figures/initiation.jpg} 
  \caption{Pathway of eukaryotic translation initiation via ribosomal scanning.
  \label{fig:initiation}}
\end{figure}

\clearpage

\subsection{Termination and recycling} 
\begin{wrapfigure}{r}{.4\textwidth}
  \includegraphics{./figures/polysome.pdf}
  \caption{Electromicrongraph of ribosomes extracted from different positions along a sucrose gradient used for polysome fractionation (A-C). For details on polysome fractionation see section \ref{exptMethod}. Reprinted with permission. DR. T. STAEHELIN et. al. Nature.1963 Aug 31;199:865-70.doi: 10.1038/199865a0. Copyright © 1963, Nature Publishing Group.
  Schematic of the TAPES method - A ribosome (A) can initiate with rate \(\alpha\) on an mRNA with a coding sequence with codons i= 1 ... L. The elongation rate at a specific codon is defined by \(\omega_i\) and \(\beta\) determines the termination rate once a stop codon is encountered. TASEP has been constantly modified, e.g. to allow for correction of initiation or elongation when the following codon is already occupied. Reprinted with permission. Juraj Szavits-Nossan and Martin R. Evans. 10.1103/PhysRevE.101.062404. ©2020 American Physical Society. 
 \label{fig:polysomes}}
\end{wrapfigure}
mRNa translation termination is  facilitated  by  two  eukaryotic release  factors (eRF), eRF2  and  eRF3-GTP[@Stansfield1995,@Alkalaeva2006]. A TC  containing  eRF2  and  eRF3-GTP binds to the A-site of the ribosome upon recognition of a stop codon. This causes an ATP hydrolysis event resulting in a conformational change and release of the polypeptide chain. eRF1 and the ATP binding cassette protein (ABCE1) together promote  the  splitting  of  the  60S  and  40S  subunits after which they can be recycled[@Pisarev2010,@Dever2012,@Hellen2018]. 

\subsection{Translation efficiency}
Each ribosome synthesises a single protein during translation of an mRNA assuming it is not prematurely terminated. It has been known since the '60s that translation of an mRNA occurs via multiple bound ribosomes (polysomes) simultaneaously (**see figure \ref{fig:polysomes}A-C**)[@Warner1962a,@Staehelin1963a].

The translation efficiency of an mRNA depends on the number of ribosomes it is associated with synthesising proteins. While all steps of mRNA can affect the translation efficiency of an mRNA, it is most commonly regulated at the initiation step[@Richter2015,@Dever2012,@Jackson2010a]. 

Assessment which step of mRNA translation is affected is often done using experimental methods (e.g. polysome profiling or by measuring ribosome transit time). Furthermore, there are efforts to model translation kinetics such as totally asymmetric simple exclusion process (TASEP) to obtain a better understanding of ribosome movements along the mRNA [@MacDonald1968a,@Maniloff1969] (**See figure \ref{fig:polysomes}D**). A model that assessed ribosome traffic under steady state and high initiation rates indicates traffic is not only dependent on the densities of ribosomes but also codon specific elongation rates and their distribution along the mRNA [@Szavits-Nossan2020b]. The next section will go further into detail how mRNA translation can be regulated so that initiation rates, but also elongation, are influenced thereby altering translation efficiencies of mRNAs.

\section{Regulation of mRNA translation} \label{regmRNA}

mRNA translation is the most energy consuming process in the cell, for example in rat it was estimated to account for  ~20% of the cellular energy consumption [@Buttgereit1995]. The role of mRNA translation in gene expression and high energy consumption therefore translational control is paramount, especially in cancer where deregulated metabolism is common[@Hanahan2011].

A strong feature of translational control of initiation rates or elongation rates is that it can affect mRNA distinct populations differently. When one or more components of the translation machinery are affected (e.g. initiation factors, ribosomal proteins, tRNA availability) translation of a large set of mRNAs is affected (i.e. global regulation). Whereas translational control acting on characteristics of mRNAs, e.g. through cis elements in the UTRs or RNA binding proteins (RBP), selective regulation occurs. In some cases mRNAs escape global translational control. 

Major signalling pathways, e.g. the PI3K/AKT/mTOR pathway and the integrated stress response, regulate translation at a global level. Furthermore, the codon usage of mRNAs can be used to regulate translation globally dependent on the available tRNA pool or tRNA modifications. Selective regulation of mRNA translation occurs via characteristics a limited population of mRNA share found in the 5' and 3' UTRs of an mRNA [@Leppek2018] (**see figure \ref{fig:UTRFeat}**).


\subsection{mTOR} \label{mTOR} mTOR is  a  conserved  Ser/Thr  kinase  and  is  found  in  two  structurally  and  functionally  distinct  complexes,  mTORC1  and mTORC2 [@Saxton2017,@Pearce2007]. In a growth promoting environment mTOR switches cell metabolism to increased production of protein, lipids and nucleotides, while suppressing catabolic pathways. mTORC2 promotes survival via signalling through protein kinase A (AKT) [@Sarbassov2005]. 

mTOR activity is modulated via growth factor signalling (i.e. insulin and insulin-like growth factor; IGF1) as well as cellular metabolism. Growth factor signalling es mediated through the phosphoinositide 3-kinase (PI3K) / AKT and Ras/mitogen-activated protein kinase (MAPK) pathways. Both these signalling pathways are involved in oncogenic signalling and under investigation as therapeutic targets in cancer[@Hilger2002,@Yang2019]. In several cancers (e.g. breast, lung, prostate and colon) the gene encoding the p110\(\alpha\) subunit of PI3K is frequently mutated or amplified [@Levine2005,@Samuels2004,@Lee2005]. The E545K mutation which leads to a reduced inhibition of p85 at the  p110\(\alpha\) subunit [@Jiang2018]. In **study 3** we investigate oncogenic signalling via the PI3K pathway activated by insulin and the role of mTOR in mediating the resulting effects on gene expression in the MCF7 breast cancer cell line that harbours the E545K mutation [@Schneck2013]. Furthermore, hyperactivity of PI3K/AKT and Ras/MAPK signalling pathways has been reported in multiple cancers and been linked anti- cancer therapy resistance [@Populo2012,@Tan2013, @Salaroglio2019]. mTOR, as a downstream actor of these pathways integrating their signals, has therefore become a focus of anti cancer therapy by either targeting mTORC1 or using dual inhibitors for PI3K and mTOR [@Bhat2015].

\begin{wrapfigure}{r}{0.6\textwidth}
  \includegraphics{./figures/mTORsignal.jpg}
  \caption{Schematic representation of mTOR signaling to the translational machinery. Philippe P. Roux, and Ivan Topisirovic Mol. Cell. Biol. 2018; doi:10.1128/MCB.00070-18. Reprinted with permission. Copyright © 2018, American Society for Microbiology
 \label{fig:mtorsignal}}
\end{wrapfigure}
mTOR fulfills a central role in metabolic signalling where it integrates amino acids availability, glucose and cellular oxygen levels to form an appropriate response. For example, amino acid availability induces relocalisation of mTOR into proximity of Rag GTPases leading to its activation [@Sancak2008]. Furthermore, Glucose deprivation and oxygen availability regulate mTOR activity via adenosine-mono-phosphate kinase (AMPK) signalling. AMPK is activated when a shift in the cellular AMP to adenosine-tri-phophate (ATP) ratio is sensed [@Sanders2007,@Kimball2006]. While hypoxia, deprivation of oxygen, inhibits protein synthesis in normal cells, in breast cancer protein synthesis was not inhibited during hypoxia attributed to uncontrolled mTOR signalling [@Connolly2006]. 

Given the central of mTOR governing proliferation, growth and metabolism, which are often deregulated in cancer, it is vital to comprehensively understand mTORC1 signalling herein [@Hanahan2011].

\subsubsection{Global regulation of translation via mTOR} mTOR regulates mRNA translation mainly by mediating availability of eIF4E by phosphorylating 4E binding proteins (4E-BPs)[@Gingras1999]. eIF4E is required for mRNA 5' cap binding and formation of the mRNA translation eIF4F complex. Therefore, inhibition of mTOR leads to a down regulation of cap dependent mRNA translation initiation. eIF4E has overexpression is commonly found cancers and targetted using 4E-antisense oligonucleotides as anti-cancer therapy [@Hong2011]. Other downstream targets of mTOR such as S6k which has been shown to regulate components of the translation machinery, e.g. by activation of programmed cell death protein 4(PDCD4) which disrupts eIF4G-eIF4A binding through competition[,@Dorrello2006,@Goke2002].  

\subsubsection{Selective or "mTOR sensitive" regulation of translation} translation involves transcripts with a terminal oligo pyrimidine (TOP) motif consisting of a C followed by a stretch of 4-15 pyrimidines directly after the 5' cap show near complete dissociation from ribosomes under conditions when mTOR is inhibited [@Meyuhas2000,@Yamashita2008]. TOP mRNA are enriched for genes encoding for parts of the translation machinery [@Meyuhas2000,@Thoreen2012]. Recent work indicates the importance of La ribonucleoprotein domain family member 1 (LARP1) in regulation of TOPs. LARP1 is thought to bind to the 5' mRNA cap of TOP mRNAs via its DM15 domain and represses translation by obstructing eIF4E binding. mTORC1 physically interacts and phosphorylates LARP1. When phophorylation occurs close to the DM15 domain of LARP1 the inhibitory effect on mRNA translation of TOP mRNAs is abolished [@Jia2021]. Other instances of selective translation are for mRNAs that lack the TOP motif, but show sensitivity to mTOR activity. These mRNAs are, in addition to mTOR, dependent on either availability of eIF4E or activity of eIF4A (**see \ref{eif4a}**). mTOR-eIF4E sensitive mRNAs show extremely short 5' UTRs and encode for metabolic functions[@Gandin2016a]. 

\clearpage
\subsection{The integrated stress response} The integrated stress response is a signalling pathway which can be activated through kinase signalling originating from various stress signals. These kinases include Protein kinase R-like endoplasmic reticulum kinase (PERK) which is activated by misfolded peptides in the endoplasmatic reticulum (ER), Heme regulated eIF2alpha kinase (HRI) which is activated during oxidative stress, protein kinase R (PKR) which is activated in response to certain viral infections and GCN2 which is activated when cells are deprived of amino acids[@Kapur2017,@Guan2017,@Taniuchi2016,@Andreev2015]. During the integrated stress response the alpha subunit of eIF2 is phosphorylated. Upon eIF2alpha phosphorylation, eIF2 alpha directly engages the guanine nucleotide exchange factor eIF2beta and prevents conversion of inactive eIF2-GDP to  active eIF2-GTP needed for met-tRNAi incorporation in the TC, therefore inhibiting translation by reducing PIC availability [@Sonenberg2009] (**see also figure \ref{fig:initiation}**).

\subsubsection{Global and selective regulation of translation via the ISR}Global and selective regulation of translation via the ISR is, similar to mTOR signalling, achieved at a global and selective mRNA level. Phosphorylation of eIF2 alpha limits ternary complex availability, therefore ribosome recruitment to the 5' cap is limited which results in a reduction of translation initiation. While global translation is reduced upon ISR, translation of a selective subset of mRNA with upstream open reading frames (uORFs) is increased. A uORF is a reading frame that originates in the 5' UTR of an mRNA upstream of which the AUG precedes that of the coding sequence. uORFs are out of frame with the main ORF and when translated lower the expression of the mORF[@Kozak1984]. Ribosome profiling studies indicate that 50% of mammalian mRNAs harbour uORFs including oncogenes and transcripts important in differentiation and cell cycle [@Calvo2009,@Ingolia2011,@Morris1995]. The surrounding context of the uORF is important for its inhibitory effect through more efficient initiation, where the classical Kozak context (i.e. [A,G]..**ATG**G) is most efficient[@Kozak1986,@Calvo2009]. ATF4, a transcription factor for stress response genes, contains two uORFs of which one partially overlaps with the mORF. Under normal conditions ATF4 mRNA translation is initiated at uORF1 and reinitiation at uORF2 occurs. The close proximity of uORF2 to the mORF causes ribosomes to scan past the mORF start thereby inhibiting the translation of the coding sequence. Limitation of TC availability during ISR causes longer ribosome scanning times leading to that ribosomes scan past uORF2 and initiate at the mORF[@Pakos-Zebrucka2016]. 

\subsection{Regulation of mRNA translation by tRNAs} \label{tRNA}

As touched upon earlier, tRNAs are an essential part of the translation machinery that carry the amino acids, to be incorporated into the polypeptide chain during the elongation process, to the ribosome. tRNAs consist of a 76-90 long nucleotide sequence set into a "cloverleaf" structure forming several loops [@Sharp1985] (** see figure tRNA**). The acceptor stem binds the amino acid carried by the tRNA, while the anti-codon loop binds to the mRNA within the ribosome via classical watson-crick pairing [@Watson1953]. Multiple codons can encode for the same amino acid (synonymous codons), however the availability of the tRNAs for different codons may vary which can influence elongation rates. 

\begin{wrapfigure}{o}{0.6\textwidth}
  \includegraphics{./figures/tRNA.pdf}
  \caption{ (A) Schematic representation of the tRNA cloverleaf structure with indicated anti-codon and amino acid acceptor sites.  (B) Schematic representation of the proliferation and differentiation mRNAs dependent on distinct tRNA subsets. (C) Schematic representation of the U34 wobble position and the catalytic enzymes involved in this modification that is implied in tumor progression.
 \label{fig:tRNA}}
\end{wrapfigure}

This supply ( i.e. tRNA availability) and demand (i.e. codon in expressed mRNA transcripts) relationship has been found to vary across different cellular states, e.g. proliferation and differentiation. In this model two distinct subsets of tRNAs are observed; A tRNA subset induced under proliferation that is otherwise repressed and a subset with similar regulation under differentiation of which the supply matches the codon demand of the transcriptome [@Gingold2014]. Therefore, in this model differentiation and proliferation would underlie two distinct translational programs dependent on tRNA expression. This model has been disputed and that the observed differences would be attributed to GC content in the mRNA [@Rudolph2016]. Nevertheless, abbarent tRNA expression and codon usage have been reported in cancer[@Zhang2018]. Furthermore, a comprehensive study including small RNAseq (i.e. for identification of tRNAs) and protein samples across 17 tissues obtained from the The Cancer Genome Atlas (TCGA) reported a tRNA signature stratified by Ki67 (a proliferation marker) staining with implications for patient survival[@Hernandez-Alias2020]. Therefore, while a consensus on proliferation specific tRNA subsets might not have been reached, emerging evidence implicates a role thereof in cancer[@Hernandez-Alias2020,@Gingold2014,@Zhang2018].

Other reports of translational interference by tRNAs in cancer are attributed to tRNA modifications, specifically at the U34 anti- codon (or wobble) position which is highly conserved [@Rapino2017,@ElYacoubi2012]. Wobbling was proposed by Francis Crick and refers to the ability of non-watson-crick base pairing of tRNA anti codons[@Crick1966]. This enables a smaller set of tRNAs (41-55 in eukaryotes) to encode for the 64 possible codon combinations [@Goodenbour2006]. In mammals, the U34 modification catalytic cascade involves the acetyltransferase Elongator (ELP3), the methyltransferase TRM9-like domain of Alkylation repair homolog 8 (ALKBH8), and the urmylation (URM) pathway, that includes the cytosolic thiouridylase homolog 1 and 2 (CTU1/CTU2)[@Kalhor2003,@Karlsborn2014]. These enzymes ultimately modify the U34 modification into 5-methoxycarbonyl-methyl-2-thiouridine (\(mcm^5s^2U\)) which ensures cognate codon recognition. This modification is thought to occur for a small subset of tRNAs, namely  \(tRNA^{UUU}\), \(tRNA^{UUC}\), \(tRNA^{UUG}\), \(tRNA^{UCC}\), and \(tRNA^{UCU}\). 

Loss of the ability to modify U34 has been shown to reduce translation elongation rates with varying effects on protein expression [@Nedialkova2015,@Deng2015,@Zinshteyn2013]. While in some cases U34 dependent signalling led to ribosome stalling resulting in protein aggregates and increased stress [@Nedialkova2015,@Zinshteyn2013]. Other reported a subtle dowregulation of proteins encoded by mRNAs requiring U34-modified tRNAs [@Deng2015]. U34 modification dependent tRNAs have been shown to play a role in cancer. For example, ELP3 is important in tumor initiation in the intestine and promotes breast cancer invasion as well as progression to metastasis[@Ladang2015,@Delaunay2016]. Furthermore, loss of U34 modification in a prostate cancer model lead to an adaptive transcriptional response for mRNAs whose translation was dependent on the U34 modification [@Lorent2019].

\subsection{eIF4A sensitive mRNAs} \label{eif4a}
  Progression of the ribosome through the 5' UTR is dependent on eIF4A, the RNA helicase in the eIF4F complex, that unwinds structural elements the ribsome encounters[@Wolfe2014,@Rubio2014]. The importance of eIF4A's unwinding capacity was identified by treatment of KOPT-K1 cells, a lymphoma cell line, MDA-MB-231 ,a breast cancer cell line, with silvesterol of which translational control was evaluated using ribosome profiling (**see section \ref{riboseq}**)[@Wolfe2014,@Rubio2014]. These studies identified silvesterol sensitive mRNAs that are characterised by long and structured 5' UTRs. In addition eIF4A dependent mRNAs were enriched for having multiple 5' UTR variants, while independent mRNAs were not [@Rubio2014]. Among eIF4A dependent mRNAs are genes important for proliferation and survival which has implications for oncogenic signaling[@Wolfe2014,@Gandin2016a]. Among others, these studies indicate a strong therapeutic value in targeting "sensitive mRNAs" in diseases. In **study 2** we aimed to exploit eIF4A sensitive mRNA translation as a therapeutic strategy in pancreatic cancer.

\subsection{RNA binding proteins and trans-acting factors}

The UTRs of an mRNA contain sequence elements to which RNA and RNA binding proteins (RBPs) bind and exert translational regulation. For instance, MicroRNAs, a small class of non coding RNA, can directly bind to other RNAs and silence them accomplished through translational repression or, more often, destabilisation [Jonas2015]. RBPs are a class of proteins involved in many regulatory steps of gene expression and account for ~ 7.5% of the protein coding genes. Poly-A-binding-protein (PABP) is thought to form a closed loop complex of the 3' end to the 5' by interacting with eIF4G. This closed loop should promote translation and prevent mRNA decay [@Afonina2014,@Amrani2008] (**see also figure \ref{fig:initiation}**). An RBP of particular interest in **study 3** is Human antigen R (HuR). HuR preferentially binds to AU-rich sequences in the 3' UTR and acts as a stabilizing agent and is involved in RNA-processing [@Levine1993,@Baou2011,@Fan1998,@Peng1998]. Studies in breast, colon and lung cancer observed correlation between HuR and malignancy. Among HuR targets are HIF-1, VEGF (important for angiogenesis) and the oncogene Myc [@Denkert2004,@LopezdeSilanes2003,@LopezdeSilanes2005].

\section{Expertimental methods to measure mRNA translation} \label{exptMethod}
\begin{wrapfigure}{r}{0.6\textwidth}
    \includegraphics{./figures/polyRibo.pdf}
  \caption{Polysome profiling and ribosome profiling workflows. In polysome profiling a fraction from whole cytoplasmic RNA is loaded onto a sucrose gradient on which they get separated by sedimentation using ultra centrifugation. Fractions corresponding to efficiently translated mRNAs are collected and can be quantified with for example RNA sequencing (left). During ribosome profiling a fraction from the whole cytoplasmic RNA is exposed to a digestion agent which disturbs the RNA. The ribosomes will protect fragments thereby creating ribosome protected fragments. These fragments are then isolated and can be sequenced.  \label{fig:polyRibo}}
\end{wrapfigure}
Methods that measure mRNA translation try to capture the number of ribosomes an mRNA is associated with on the prinicple that this is directly correlated with their translation efficiency. There are two methods that are predominantly used  for measuring mRNA translation, or changes in translation efficiencies across conditions, namely polysome profiling and ribosome profiling (**see figure \ref{fig:polyRibo}**).
 
\clearpage

\subsection{Polysome profiling} is a technique to measure changes in translational efficiencies of mRNAs between two or more conditions. Polysome profiling allows for separation of polysomes from monosomes, ribosomal subunits and messenger ribonucleoprotein particles (mRNPs). During the assay, ribosomes are immobilized on the mRNAs using translation elongation inhibitors (e.g. cycloheximide). A portion of cytoplasmic RNA extracts then sediment on a linear sucrose gradient (5-50%) using ultra centrifugation.
\begin{wrapfigure}{r}{0.6\textwidth}
    \includegraphics[width=0.9\linewidth]{./figures/polysome_shifts.pdf}
  \caption{Polysome profliles -  (top left) Schematic representation of a polysome profile using linear sucrose gradient fractionation. Indicated in the polysome profiles are the 40S, 60S ribosomal subunits as well as the 80S monosome. H.P. indicates heavy polysome fractions.Between conditions (i.e. black an pink lines) distribution changes for mRNA abundance (grey and green; top right), translation (grey and red; bottom left) and translation within high polysome fractions (grey and red; bottom right) are illustrated. \label{fig:polysome}}
\end{wrapfigure} 
The resulting gradient is fractionated and mRNAs with different number of bound ribosomes can be extracted and analyzed for changes in translational efficiency [@Gandin2014]. An illustration of a polysome profile with peaks for the 40S, 60S subunits and 80S ribosome can be seen in (**Fig \ref{fig:polysome} top left**). Subsequent peaks along the frations indicate the mRNAs with 1 or more bound ribosome. mRNAs are typically normally distributed along the fractions, i.e. a pool of the same mRNA will be associated with 1- n number of ribosomes. Changes in mRNA abundance will lead to an overall increase in the amount of isolated polysome-associated mRNA without a shift of the distribution along the fractions (**Fig \ref{fig:polysome} top right**). This means that the translation efficiency per mRNA remains unchanged. Changes in translational efficiency  can be observed by shifts of polysome association for mRNAs from the light (inefficiently translated) towards the heavy (efficiently translated) polysome fractions or vice versa (**Fig \ref{fig:polysome} bottom left**). Shift within the heavy polysome fractions (i.e. 3 bound ribosome to 7 bound ribosome) can also occur (**Fig \ref{fig:polysome} bottom right**). These shift remain undetected in cases where the distribution of polysome-associated mRNAs does not sufficiently shift across the fractions and is a limitation of polysome profiling. Quantification of mRNA levels within each fraction can be assessed using Northern blotting or reverse transcription quantitative polymerase chain reaction (RT-qPCR).

For transcriptome wide studies, pooling of efficiently translated mRNAs (mRNAs with >3 bound ribosomes) followed by quantification using either DNA-microarrays or RNA sequencing is common. The 3-ribosome cut off has been chosen as it is thought to capture most biologically relevant changes in translation efficiency. Pooling of mRNAs as well as collection of multiple fractions makes polysome profiling inconvenient when dealing with large samples sizes or experiments with low amounts of input RNA. Therefore, an optimized sucrose gradient was developed where most efficiently translated mRNAs are collected on a sucrose cushion and thereby can be isolated from one single fraction [@Liang2018]. This optimized gradient allows for application of polysome profiling in small tissue samples where RNA quantity is limiting and reduces labor intensity of the assay. 

Polysome-associated mRNA levels are subject to changes in translation efficiency as well as factors contributing to cytosolic mRNA levels. Mechanisms such as transcription (i.e. in the case of mRNA abundance) or mRNA stability can affect cytosolic mRNA levels which impacts the pool of mRNAs that can be associated to polysomes. Therefore, to identify true changes in translation efficiency it is important to collect cytoplasmic mRNA levels in parallel to polysome-associated mRNA to correct for such mechanisms (e.g. transcription or mRNA stability) during downstream analysis [@Gandin2014]. 

\subsection{Ribosome profiling} \label{riboseq}
Ribosome profiling is a technique that enables sequencing of ribosome protected mRNA fragments (RPFs). In the assay ribosomes are immobilized on the mRNAs using, similar to polysome profiling, translation elongations inhibitors (e.g. cyclohexamide) [@Ingolia2009,@Ingolia2016]. One limitation with the use of translation elongation inhibitors is the distortion of ribosome distributions especially at translation initiation sites. These introduced artefacts need to be accounted for in the downstream analysis when assessing ribosome position along the mRNA. Following the translation elongation inhibitor treatment, cells are immediately flash frozen using liquid nitrogen. Alternatively, using only flash freezing has been seen as a robust approach in a wide range of diverse organisms [@Brar2015].

RPFs are obtained by RNAse treatment that breaks the links of RNA between ribosomes leaving single ribosomes with a ~28 nucleotide long RNA fragment within each ribosome. The RPFs are then isolated using ultra centrifugation through a sucrose cushion. During this step other RNA fragments such as  non-coding RNAs or large ribonucleoprotein complexes can co-migrate and contaminate the sample. Typically RPFs with a size ranging from 25-30 nucleotides are selected for quantification.

In parallel to RPF selection, randomly fragmented total mRNA of the same size is also retrieved. This is achieved by extraction of total mRNA from cell lysate followed by purification via recovery of polyadenylated messages or removal of ribosomal RNA.  [@Ingolia2009,@Brar2015].

\subsection{Comparing ribosome and polysome profiling}
Albeit both methods generate count data after quantification with RNAsequencing, there are some key aspects that differ between the techniques. Polysome profiling separates efficiently translated mRNAs from non- efficiently translated mRNAs along a sucrose thereby creating an mRNA based perspective for analyzing changes in translational efficiencies. In contrast, ribosome profiling determines translational efficiencies by counting the number of RPFs of both efficiently and non-efficiently translated mRNAs. This gives polysome profiling the advantage in cases of transcript variants with important features in their 5´UTR. Such information, if not protected by a ribosome, would be lost in ribosome profiling.

Changes in translational efficiencies, e.g. shifts between the polysomal fractions, can be dramatic (I.e. near complete dissociation of ribosomes from an mRNA) or subtle (shifts from 2 to 4 ribosomes) [@Livingstone2015]. Ribosome profiling has been shown to be biased towards identification of dramatic shifts of associated ribosomes to mRNAs, whereas subtle shifts are masked which can lead to false biological conclusions. Polysome profiling is affected by this to a much lesser extent, thereby more robust in identifying such changes [@Masvidal2017]. Higher sensitivity in detecting changes in translational efficiencies on a global scale makes polysome profiling more suitable for genome-wide studies [@Gandin2016].

An advantage of ribosome profiling is that it provides exact nucleotide positions occupied by ribosomes. This offers information at a single nucleotide level where the ribosome sits. Polysome profiling cannot reveal ribosome locations along the mRNA. The single nucleotide resolution of ribosome profiling is necessary in contexts studying local translation events such as ribosomal frame shifts [@Rato2011] or uORF translation [@Andreev2015]. 

Both methods have their strengths and weaknesses and therefore each method should be considered depending on the underlying biological question of each experiment.

\section{Modes for regulation of gene expression in mRNA translation} \label{modes}

\begin{wrapfigure}{l}{0.6\textwidth}
  \includegraphics{./figures/geneModes_MRNA.pdf}
  \caption{Regulatory modes of gene expression - Schematic representation of regulatory modes of translation efficiency in a fold-change scatter plot. Indicated in red are changes in translation (i.e. changes in translated mRNA but not total mRNA), in green changes in mRNA abundance (i.e. congruent changes between total mRNA and translated mRNA) and in blue translational buffering (i.e. changes in total mRNA levels but not translated mRNA levels). TE changes as the TE-score would estimate them are indicated.\label{fig:modes}}
\end{wrapfigure}

As explained in the previous section, from genome-wide assessments of translation using ribosome or polysome profiling expression levels for both cytoplasmic and polysome-associated mRNAs (or RPFs). For the sake of simplicity , from now on, these RNA types will be referred to as total mRNA (i.e. cytoplasmic mRNA) and translated mRNA (i.e. polysome-associated mRNA or RPFs). The estimation of expression levels for both translated mRNA and total mRNA allows for interrogation at two steps of the gene expression pathway and their interaction. The interaction of total mRNA with translated mRNA can give valuable insights for the underlying mechanisms that govern gene expression in the studied system.

When comparing perturbed systems to their corresponding control state we typically observe three "modes" in which translated mRNA and total mRNA distinctly interact that impact gene expression (**See figure \ref{modes}**). We refer to these modes as "translation", "mRNA abundance" and "translational buffering". 

\subsection{Translation} A change in "translation" occurs when  total mRNA levels remain unaltered, however translated mRNA levels either increase or decrease resulting in a change of their translation efficiency. A prominent example of this mode can be observed to TOP mRNAs. These mRNAs, under conditions when mTOR is inhibited, show a near complete dissassociation from ribosomes[@Gandin2016a]. mRNAs under the translation mode are expected to reshape the proteome (**See figure \ref{modes}**). 

\subsection{mRNA Abundance} Another mode that impacts the proteome is a change in mRNA abundance here the translated mRNA level changes to a similar magnitude as the total mRNA level. For these mRNAs the translation efficiency is unaltered, as the change in total mRNA levels explains the change in translated mRNA levels. Nevertheless, since there are more (or less) mRNAs that are being translated an effect the proteome is expected. The underlying biological implication for this mode mRNAs is often related to transcription (**See figure \ref{modes}**).

\subsection{Translational buffering} \label{modeBuffering}
In recent years, evidence emerged where translation efficiencies of mRNAs can be altered to compensate for changes in total mRNA levels. This mode is characterised by changes in total mRNA levels that are not matched by translated mRNA levels. The expected result on the proteome is that it maintaining (i.e. keeping levels constant) rather than reshaping it [@McManus2014,@Lorent2019] (**See figure \ref{modes}**).

This mode was named "Translational buffering" and currently the literature supports multiple different instances where translation "buffers" changes in transcription to retain a constant proteome. At steady state, translation compensates for inter-individual, inter-species or inter-tissue differences 

At steady state a compensatory aspect of translational buffering is observed when corrections are made, for e.g. gene dosage or transcriptional noise, at an inter-species or inter-individual level [@Artieri2014,@Cenik2015,@Perl2017,@Wang2020,@Li2014,@Lalanne2018]. Which is consistent that protein abundance is overall more conserved across species [@Laurent2010]. 

An example of such compensation was observed when comparing evolutionary distant bacteria species, i.e. B. subtilis, E. coli and S. cerevisiae. It was found that while extensive remodeling of promotors and terminators diverged transcript abundance, post-transcriptional regulation was altered to maintain a preferred stoichiometry of pathways[@Lalanne2018]. In B. subtilis translation related factors rpsP and rplS are transcriptionally fine tuned, whereas in E. coli they lie within an operon together with rimM and trmD which are only required in low abundance at the protein level. Therefore, E. coli compensates the transcriptional input at the translational level[@Lalanne2018]. 

A different form of translational buffering can be observed at in perturbed systems. For example, in prostate cancer cells a transcriptional program was induced under estrogen receptor\(\alpha\) (ER\(\alpha\)) depletion that was offset at the level of translation. mRNAs whose transcription was induced required the tRNA u34 modification, of which the catalytic enzymes were dependent on ER\(\alpha\), for efficient translation [@Lorent2019]. In **study 3** we discuss the occurrence of translational buffering in its "offset" context. 


\section{Algorithms for analysis of changes in translation efficiencies}\label{algorithm}

Given these multiple roles of mRNA translation to regulate the proteome it is critical to distinguish them as their underlying mechanisms can have different biological implications. In this section we will discuss methods that analyse polysome-profiling and ribosome profiling data to estimate changes in translation efficiencies across 2 or more conditions and how these methods identify different modes of gene expression.

Initially analysis of transcriptome-wide translation studies used an approach called the translation efficiency (TE-score) that uses the following equation: 
\[\varDelta TE = \frac{\frac{P_{c2}}{T_{c2}}} {\frac{P_{c1}}{T_{c1}}}\\\]

This score calculates the ratio of the ratios between polysome-associated mRNA levels (P) divided by total mRNA levels (T) within each condition (i.e. C1 and C2). The TE- score approach has been shown to be prone to spurious correlations [@Larsson2010]. Spurious correlations arise due to that the ratio of polysome-associated mRNA and total mRNA can systematically correlate with total mRNA levels which is not corrected for in this equation and leads to an elevated type-1 error.
\clearpage
\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_TE.pdf}
  \caption{TE scores for regulatory modes of gene expression -  Schematic representation of regulatory modes of translation efficiency in a fold-change scatter plot. Indicated in red are changes in translation efficiency altering protein levels, in green changes in mRNA abundance and in blue changes in translation efficiency leading to translational buffering/offsetting. The shifts for the translation efficiency (TE) score are indicated. \label{fig:TE}}
\end{wrapfigure}
**Figure \ref{fig:TE}** gives an overview of the relationship between a change in  TE and each regulatory mode of gene expression (**see also figure \ref{fig:modes}**). Changes in mRNA abundance will lead to a \(\varDelta\)TE close to 0 in log space (i.e. no change) as total mRNA and translated mRNA change with a similar magnitude. However, in the case of both translation and translational buffering, terms in the TE-score equation change leading to a \(\varDelta\)TE (TE < 0 or TE > 0) and thereby identification of both changes in translation and translational buffering simultaneously. Therefore, the TE-score method fails to differentiate between changes in translation and translational buffering which can have drastic consequences for the biological interpretation of the results [@Oertlin2019] (**see also section \ref{modes}**).

The TE-score approach was challenged by the Analysis of Translation Activity (anota) algorithm which was developed for DNA-microarray data [@Larsson2011]. anota combines analysis of partial variance (APV)[@Schleifer1993] with a random variance model (RVM)[@Wright2003]. RVM estimates gene variance using shared information across all genes to increase power for detection of differential expression[@Wright2003]. anota uses a two-step process that firstly assesses the model assumptions for (i) absence of highly influential data points, (ii) common slopes of sample classes, (iii) homoscedasticity of residuals and (iv) normal distribution of per gene residuals. In the second step then performs analysis of changes in translational activity using the following model:

\[log(y_{gi}) = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 
  
here \(\beta_g^{RNA}\) described the relationship to total RNA  \(gth\) gene \(ith\) sample of model matrix \(X\); \(\beta_g^{cond}\) represent the log2 fold change for treatment classes and \(\varepsilon_{gi}\) denotes the residual error. 

\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_anota_Larsson.pdf}
  \caption{anota gene models - Schematic representation of the anota analysis models. Translation mRNA expression is set out against total mRNA expression for each biological replicate and treatment condition. Top left shows the model of a gene that is deferentially translated (i.e. change in translated but not total mRNA). The difference in the slope intercepts are used to estimate changes in translation efficiencies between conditions i.e. dTE. Other gene models are shown; change in translation efficiency with varying total mRNA levels (top right); change in mRNA abundance (bottom left) and translational buffering (bottom right).
  \label{fig:anota}}
\end{wrapfigure}

Within anota a common slope for the treatment classes that describes the translated mRNA to total mRNA relationship is calculated. The difference between the slope intercepts is then interpreted as the \(\varDelta\) TE. A simplified view of this model can be seen in (**Figure \ref{fig:anota} top left**). Here expression for translated mRNA and total mRNA are modeled over two sample classes with each 4 replicates. Furthermore, changes in translation efficiencies can also be observed when translated mRNAs shift to a larger extent than the total mRNA levels (**Figure \ref{fig:anota} top right**). Identification of genes in this category can be a challenge, especially in highyl variable data set, as they resemble mRNA abundance genes (**Figure \ref{fig:anota} bottom left**). Nevertheless, Using the linear regression analysis anota accurately corrects changes in translated mRNA as can be seen in (**Figure \ref{fig:anota} bottom right**) where a change in total mRNA   but not translated mRNA levels is observed. For this gene the difference in slope intercepts is small and will not be identified as difference in translation as would be the case in the TE-score approach. anota was developed at a time where translational buffering was uncommonly seen in data sets. Naturally, the methods lacks a setting to analyse translational buffering. This was addressed in anota's successor, anota2seq, and will be discussed in **Study 1**.

Advances in experimental methods warrant for appropriate statistical methods to analyse data resulting from them. DNA- microarray was the dominant platform to assess genome-wide changes before the advent of RNA sequencing. In DNA- microarray RNA hybridizes probes on a chip and generate a signal of which the measured intensity is an indicator of expression, whereas in RNA sequencing reads from RNAs are counted. Intensity data from DNA microarray can be normalised and transformed (i.e. log transformation) to fulfill the requirements for application of linear models, whereas RNA sequencing harbours additional characteristics that need to be accounted for. Therefore, algorithms developed for analysis of DNA- microarray are not directly applicable to RNA sequencing data as is the case for the anota algorithm.

RNA sequencing data shows variance that is greater than the mean which is commonly referred to as overdispersion. Count data from RNA sequencing have been initially approached using Poisson distributions which assumes that the variance is equal to the mean[@Lu2005]. Now established RNA sequencing analysis frameworks such as edgeR and DESeq2  use negative binomial distributions in combination with generalized linear models (GLMs) [@Robinson2010, @Love2014]. The negative binomial distribution uses a dispersion parameter to account for differences in the mean-variance relationship across the expression range [@McCarthy2012]. While analysis principles of DESeq2 and edgeR are similar they differ in their normalisation method, dispersion estimation and information sharing across genes. In a simple differential expression analysis between two conditions with one RNA type the GLM model would be as in the following equation: 

\[log(y_{gi}) = \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 

here \(\beta_g^{cond}\ X_i^{cond}\) represent the condition (i.e. control and treatment) log2 fold change for the \(gth\) gene \(ith\) sample of the model matrix X and \(\varepsilon_{gi}\) denotes the residual error. When analysing changes in translation effiencies additional parameter for RNA type (i.e. total mRNA or translated mRNA) and the interaction between the RNA type and condition are added so that:

\[log(y_{gi}) = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \beta_g^{RNA:cond}\ X_i^{interaction} + \varepsilon_gi\] 

In this model the interaction term is interpreted as the change in translation effiencies [@Chothani2019]. Other methods (i.e. Ribodiff[@Zhong2017], Riborex[@Li2017] and deltaTE [@Chothani2019]) borrow this analysis principle of an GLM with an interaction term by often applying this exact model. A notable difference is that Ribodiff allows dispersion estimation for translated mRNA and total mRNA separately as variance differences between the RNA types can be expected due to varying experimental protocols [@Zhong2017, @Liang2018]. While the flexibility of GLMs allows for complex study designs involving 2 or more treatment conditions, Riborex and Ribodiff limit the study design to only two conditions. DeltaTE gives their users full flexibility of the DESeq2 GLM model. Xtail is a method developed for ribosome profiling that makes use of DESeq2 for RNAseq count normalisation [@Xiao2016]. Their assessment of differences in translation efficiencies relies on probability matrices for the ratio of translated mRNA over total mRNA within condition and a between condition ratio of these ratios. Babel was the first algorithm designed solely for analysis of differential translation and uses an error-in-variables regression analysis [@Olshen2013]. The error-in- variables regression allows accounting for variable total mRNA levels when assessing changes in translation.  Although these methods have distinct approaches to identify changes in translation efficiencies, their principle of analysis is similar to comparing a ratio of ratios. Therefore these methods suffer from similar issues as the TE-score which will be discussed in **Study 1**.


