---
output: pdf_document
---


\chapter{Results and discussion}

\section{Study 1 - Generally applicable transcriptome-wide analysis of translation using anota2seq}

Initially changes in translation efficiencies were estimated using the TE-score approach as outlined in section \ref{algorithm}. However, this method was being shown to be prone to identification of spurious correlations leading to elevated false positive identification that can result in false biological conclusions [@Larsson2010]. The identification of spurious correlations, when using the TE-score, can be attributed the inadequate correction for changes in total mRNA levels when estimating translation efficiencies [@Larsson2010,@Larsson2011]. The Analysis of Translation Activity (anota) algorithm facilitates analysis of translational efficiencies that are corrected for changes in total mRNA levels and therefore is not prone to spurious correlations[@Larsson2011]. 

anota was developed for analysis of transcriptome-wide analysis for data quantified by DNA- microarrays [@Larsson2010]. However, advances in experimental methodologies lead to the development in RNA sequencing. RNA sequencing and DNA microarray data have distinct characteristics that need to be accounted for before analysis (**see section \ref{algorithm}**). Therefore, while the statistical framework of anota had been shown as an adequate approach for analysis of translational efficiencies for data from DNA microarrays it was not directly applicable to RNA sequencing data. The characteristic of RNA sequencing data, that makes applying anota directly not possible, is the mean variance relationship. This encompasses that the counts for lower expressed genes show higher variability than counts for higher expressed genes even after log transformation. Efforts have been made to make RNA sequencing data more DNA- microarray like so that algorithms developed for intensity based microarray data can be applied to count based RNA sequencing data [@Law2014,@Love2014]. Anota2seq, the algorithm developed in this study, allows for transformation and normalisation of RNA sequencing data so that the anota statistical frame work can be applied for analysis of count data. 

Another feature of anota2seq is that it allows for statistical analysis of translational buffering. The need for the analysis of translational buffering, or the uncoupling of transcription from translation, has been noted before anota2seq's development by comparing 20 translatomes and transcriptomes with different underlying  stimuli in mammalian cells [@Tebaldi2012]. The same authors proposed a framework, called tRanslatome, that combines several methodologies for analysis of differential transcription and translation efficiencies, including anota, for a comprehensive analysis of transcription and translation as well as their underlying mechanisms [@Tebaldi2014].

\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_anota2seq.pdf}
  \caption{anota2seq gene model for analysis of translational buffering /offsetting - Total mRNA expression is set out against translated mRNA expression for each biological replicate and treatment condition. The model shows total mRNA changes that are independent of translated mRNA changes which is classified as translational buffering. It is important to distinguish between the gene modes as their regulation could be due to different underlying biological mechanisms (see section \ref{modes}).
  \label{fig:anota2seq}}
\end{wrapfigure}

Nevertheless, commonly observed in polysome and ribosome profiling data sets are three gene expression modes, translation, translational buffering and mRNA abundance. While anota can be used to identify genes among the translation and mRNA abundance mode, analysis for translational buffering was not implemented therein (*See Figure \ref{fig:anota}*). Therefore, one would need to rely on the integration of several methods to efficiently analyse transcriptome-wide studies of translation efficiences. Anota2seq addresses this issue by changing the analysis model as described in section \ref{algorithm} to analyse changes in total mRNA levels corrected for changes in translated mRNA levels (i.e. translational buffering, **see figure \ref{fig:anota2seq}**). 

Appliance of anota2seq has successfuly identified translational buffering to which biological mechanisms could be linked,e.g. as mentioned earlier translationally bufferring under ER\(\alpha\) depletion in prostate cancer (**see section \ref{modeBuffering}**) [@Lorent2019]. Furthermore, in **study 2** translational buffering can be observed as a compensating mechanisms in "healthy" cells upon treatment with an eIF4A inhibitor and in **study 3** we identify  mTOR dependent translational buffering for mRNAs with certain 3' UTR charactersitics.

The aim of this study is to compare anota2seq's performance to other established algorithms (i.e. DESeq2, RiboDiff, babel, TE-score and Xtail) for analysis of translation efficiencies, specifically their ability to distinguish the three prominent modes of gene expression. To achieve we used a simulated data based approach. While it is arguable to what extent conclusion drawn from simulated data can be extended towards empircal data it allows for a controlled environment where true positive changes are known in advance. Furthermore, the mean-variance relationship in the simulated data is based on a real polysome profiling data set to increase confidence that drawn conclusions are also applicable to emprical data [@Guan2017].

The simulated data consisted of four replicates for translated mRNA and total mRNA with a "control" and a "treatment" condition. Furthermore, the data sets contained a combination of the following gene sets:

"Unchanged": For this simulation category we drew reads from the same NB distrubution for both the control and treatment conditions in both the translated and total mRNA. This catergory reprents genes that would be unaffected by e.g. a stimulus of between cellular states.

"mRNA abundance": For this category the control condition for both the translated mRNA and total mRNA were drawn from the same NB distribution. The NB distribution for *both translated mRNA and total mRNA* of the treatment condition was altered so that values would be drawn corresponding to a fold change (negative or positive) ranging between 1.5 to 3.0. The directionality of the fold changes (i.e. up or down regulation) was the same for translated mRNA and total mRNA.

"translation": For this category the control condition for both the translated mRNA and total mRNA were drawn from the same NB distribution. The NB distribution for *translated mRNA only* of the treatment condition was altered so that values would be drawn corresponding to a fold change (negative or positive) ranging between 1.5 to 3.0.

"buffering": For this category the control condition for both the translated mRNA and total mRNA were drawn from the same NB distribution. The NB distribution for *total mRNA only* of the treatment condition was altered so that values would be drawn corresponding to a fold change (negative or positive) ranging between 1.5 to 3.0.

As a first step we tested whether the methods could properly control for type-1 errors (i.e. false positive identifcation). For this we simulated a data set with genes belonging only to the "unchanged" category. This revealed that babel, but to an even greater extent Xtail, were unable to control their type-1 error as these methods assigned low p-values and FDRs when no real changes were present. This indicated a limited applicability of Xtail and babel for statistical analysis of translatomes.

From the comparative analysis  of the analysis for changes in translation efficiencies affecting protein levels we concluded that anota2seq outperforms all other methods. This was assessed by comparing the area under the curve from Receiver operating characteristics (ROC) and precision recall curves. The ROC curves showed a, albeit slightly, better performance for detecting changes in translation. However, the precision recall was much higher for anota2seq which can be accredited to that the analysis principle of the other methods is based on identifying changes regardless of whether the change is in the translated mRNA or total mRNA (*as explained in section \ref{algorithm}*). Nevertheless, when comparing the performance using simulatd data in the absence of genes belonging to the "buffering" category anota2seq still showed superior performance. 

Next to the a prior knowledge of the introduced changes in the simulation data, it also allowed us to modify parameters to investigate the robustness of the methods to increased variance, overall sequencing depth and differing sequencing depth between samples. Here, all methods showed robustness against variance and sequencing depth differences between samples as long as a minimum of 5 million counts per sample was reached. 

A short coming in the simulation study is that we did not assess the effects of systematic batch effects. Batch effects can be introduced e.g. during experimental design and there are many methods that try to correct for these[@Johnson2007,@Leek2014,@Zhang2020]. Other ways to correct for batch effects is their inclusion in the analysis model, which can be supplied to the analysis model in DESeq2, edgeR as well as anota2seq. Indeed analysis of a dataset with prominent batch effects showed that batch effects can dampen the efficiency of the anota2seq algorithm to identify changes but can be effectively corrected for in the algorithm.

In this study we developed an analysis algorithm for efficient transcriptome-wide analysis of translation efficiencies applicable to DNA-microarrays and RNA seq. Furthermore, anota2seq has been successfully applied to broaden the knowledge around mRNA translation in various different contexts [@Lorent2019,@Chan2019,@Hipolito2019,@Chaparro2020]. 


\section{Study 2 - eIF4A supports an oncogenic translation program in pancreatic ductal adenocarcinoma}

Pancreatic cancer is considered a lethal malignancy and has limitited treatment options. A study on predicitons for european cancer mortality rates for 2021 concluded that health efforts should focus on pancreatic cancer. While other cancers (e.g. ovary, breast and stomach) showed a decline in mortality rates, no major overall decline was observed for pancreatic cancer in the period of 1970-2021 [@Carioli2021].

Pancreatic ductal adeno carcinoma (PDAC) accounts for over 90% of exocrine pancreatic cancer, whereas non ductal pancreatic cancers e.g. acinar cell carcinomas are uncommon [@Feldmann2007,@Jun2016]. It is estimated the 60-70% of the PDACs arise in the head [@Luchini2006]. So far treatment options are mostly limited to surgical removal, which can be complicated due to the anatomical location of the pancreas head. 

With the increasing understanding of tumor heterogeneity, i.e. tumors with similar tissue origin are not necessarily identical at a molecular level,  anti cancer therapy improved [@Biankin2011]. In breast cancer stratification by histological,molecular and gene expression features lead to identification of several breast cancer subtypes for which different treatment options exist, e.g. \(ER^+\) breast cancer subtypes respond to endocrine therapy whereas \(ER^-\) do not[@Andre2006,@Parker2009]. While breast canacer treatment strategies benefit from a rather well established understanding of the molecular subtypes, in pancreatic cancer trascriptomic based subtyping is still ongoing [@Bailey2016,@Puleo2018,@Collisson2001,@Moffit2015,@Collisson2019]. Therefore, it is warranted to extend current knowledge around pancreatic cancer to advance therapeutic strategies in this lethal disease.

Nevertheless, commonly shared among PDAs are oncogenic mutations in KRAS as well as inactivation of tumor supressors e.g. TP53 [@Jones2008]. Furthermore, PDAs have been shown to be dependent on increased protein synthesis mediated by Kras [@Chio2016]. This indicates an important role of mRNA translation in PDA. 

The aim of *study 2* was to investigate the therapeutic effects of targetting eIF4A in a three dimensional PDA organoid cell culture with mutations in the \(Kras^{LSL-G12D}\), \(Trp53^{LSL-R172H}\) and Pdx1-cre alleles that has been shown to recapitulate PDA tumor progression [@Boji2015]. The inhibition of eIF4A was done using a synthetic rocaglate, CR-1-31B (CR-31). Rocaglates have been shown to inhibit eIF4A helicase activity and displayed anti tumor activity [@Cencic2009]. 

We first wanted to establish the therapeutic validity of targetting eIF4A in PDA. In vitro experiments comparing treated PDA organoids (KP) to their normal (N) counter parts revealed hightened sensitivity of PDA organoids to CR-31 treatment. OP-puromycin incorporation showed reduced protein synthesis in KP, whereas N were affected to a lesser extent. Furthermore, similar effects were found in vivo for PDA tumours where also CR-31 reduced protein synthesis (assessed by SUnSET assay), tumor growth (assessed by ultra sound imaging) and increased survival. The effect on protein synthesis was not due to inhibition of oncogenic signalling pathways which was evaluated western blot assessing the phosphorylation of e.g. AKT, mTOR and 4E-BP1. From these findings we concluded that there is therapeutic validity in targetting eIF4A in PDA that acts downsteam of oncogenic signalling pathways.

Using polysome profiling we then sought to decipher the mechanisms explaining the increased sensivity to CR-31 in KP organoids.  First we investigated the differences in gene expression between untreated KP and N organoids. Analysis of changes in translation efficiencies using anota2seq revealed massive modulation at both transcription and translation indicative of the underlying differences in e.g. genomic stability and enhanced oncogenic signalling impinging on protein synthesis. Consistent with the in vitro OP-puromycin incorporation and in vivo SUNsET experiments, CR-31 had only little effect on translation in N organoids, whereas in KP organoids exclusively changes in translation were detected.

We then compared the mRNAs affected by CR-31 in KP to the acquired translational profile of untreated KP and N organoids. In order to achieve this we visualised the mRNAs affected by CR-31 in KP onto the data of the untreated KP vs N organoids comparison. This revealed that the acquired translational program of KP organoids is reversed when treated with CR-31. 

Performing a similar analysis, we visualised mRNAs sensitive to CR-31 in KP onto the data of the CR-31 treated N organoids. mRNAs affected by CR-31 in KP were translationally buffered in CR-31 treated N organoids. Translational buffering, as an adaptive response to treatment, has been shown to maintain protein homeostasis by inducing a transcriptional response increasing the pool of mRNAs that can be translated [@Lorent2019]. The ability for N organoids in increase transcription for mRNAs affected by CR-31, whereas KP cannot, could partially explain as to why protein synthesis is not reduced to a similar extent in N as in KP.

We then assessed 5' UTR characteristics of the translationally regulated mRNAs upon CR-31 treatment in KP organoids. It was reported that eIF4A-senstive mRNAs showed overall and more structured 5' UTRs (e.g. containing G-quadruplexes) [@Rubio2014,@Wolfe2014,@Gandin2016a]. Furthermore, a mechanisms by which rocaglates would clamp eIF4A to mRNAs with [A,G] repeats in their 5 UTR was described [@Iwasaki2016]. However, mRNAs sensitive to CR-31 treatment herien showed overall shorter 5' UTRs that were more structured when corrected for their length  without enrichment for 4G-quadruplexes or  [A,G] repeats. From the polysome profiling and UTR analysis we concluded that eIF4A supports an oncogenic translation program in PDA cells for mRNAs with shorter but structured 5' UTRs.

Shorter 5' UTRs have been shown to be sensitive to eIF4E expression and enconde for metabolic functions (see section \ref{regmRNA}). When we compared an eIF4E overexpression signature in the KP vs N and CR-31 treated KP we observed that in KP organoids translationally regulated mRNAs under eIF4E overexpression were also translationally activated. This observation is consistent with reports of 4E-BP1 loss in pancreatic cancer [@Martineau2014]. CR-31 treatment in KP reversed the translational profile for these mRNAs. Therefore, while rocaglates have been shown to directly target eIF4A [@Cencic2009]. eIF4A inhibition in tumors resistant to mTOR inhibiton by loss of 4E-BP1 has been shown to circumvent this resistance [@Muller2019]. Therefore, eIF4F complex formation could be disrupted by reduced eIF4A availability to an extent that also cap dependent translation is affected in PDA organoids herein. 

When further inspecting the regulated gene sets in treated and untreated KP compared to their normal counterparts we could see an enrichtment in metabolic pathways, e.g. Oxidative phosphorylation. This pathway was upregulated in untreated KP compared to N, whereas in KP CR-31 treatment reversed the translational profile of this pathway. These findings were experimentally validated and shown to disrupt the mitochondrial respiration (measured by oxygen consumption rates). 

A way to counter loss of energy production through oxidative phosphorylation is to increase activity of other anaerobic metabolic pathways, i.e. glycolysis. However, in CR-31 treated KP we could not detect a upregulation of glycolysis measured by \(U-C^{13}\) glucose labeling and extra cellular acidification rates nor did CR-31 treatement affect  expression of glycolytic enzymes (e.g. HK1,HK2, LDHA, SLCA1, SLCA3). Furthermore, glucose deprivation did not further sensitise to CR-31 treatment. This was rather unexpected, given that increased glycolysis is characteristic for cancers as described by Otto walburg [@Warburg1927]. However, the polysome prolfing data revealed translational downregulation and subsequent reduction of protein expression for the glucose transporter Slc2a6. Indeed, perturbation of Slc2a6 using \(sgRNA^{Slc2a6}\) in N and KP organoids revealed a decrease in glucose uptake. From this we concluded that glycolytic compensation of KP is diminished by translational regulation of the glucose transporter Slc2a6 upon CR-31 treatment.

Among the translationally activated genes in the CR-31 treated KP organoids where mRNAs involved in the glutamine metabolism (i.e. Slc1a5 and Gls1). Furthermore, glutamine levels were elevated in patient derived PDA cell lines treated with CR-31. Glutamine can be converted into \(\alpha\)-ketoglutarate and funneled into the krebs cycle and therefore can serve to increase energy production [@Xiao2016a]. Indeed, using gas chromatography mass spectrometry (GC/MS) to quantify downstream metabolites after culturing PDA cells in \(C_5^{13}\)- glutamine, we identified a shift towards reductive carboxylation of \(\alpha\)-ketoglutarate obtained from \(C_5^{13}\)- glutamine to produce acetyl-CoA. Notably, the glutamine metabolism was not elevated in N organoids.

A combined treatment of CR-31 with glutaminase inhibitors (BPTES or CB839) could sensitise to CR-31 treatment patient-derived PDA cells to CR-31 treatment. Therefore, our study suggests an eIF4A dependent translational program in PDA that can act as a theurapeutic target in PDA. Furthermore, a recently published ribosome profiling study of a CR-31 treated human pancreatic cancer cell line (PANC1) observed the same therapeutic effect of CR-31 treatment in vivo on survival and tumor volume [@Signh2012]. This underlines the significance of our study in identifying eIF4A as therapeutic target in PDA. 

Nevertheless, the same study indicated differences on the underlying regulated mRNA subsets. They report, in line with the literature, that eIF4A depdent mRNAs show long and structured 5' UTRs containing G-quadruplexes [@Singh2021,@Wolfe2014]. While the 5' UTRs of the mRNAs identified herein where overall shorter, they were overall more structured. Furthermore, the mRNAs identifed in Singh et. al. were involved in KRAS signalling, which was unaltered in our study.

This raises some questions about the differences between experimental setups and their potential influence on biological outcomes. For instance, Singh et. al. performed ribosome profiling on a PANC1 cell line culture treated with 25nM CR-31, wheres herein we performed polysome profiling on a 3D-organoid culture treated with 10nM CR-31. The differences between ribosome and polysome profiling have been discussed extensively (see section \ref{Exptmethod}). However, by measuring IC50 concentrations for CR-31 in a panel of pancreatic cancer cell lines, the autors show a ~6-fold difference in susceptability to CR-31. PANC1 cells were most affected by CR-31. Dosage dependent viability experiments of patient derived PDA cells in our study revealed that at 10nM viability was reduced by ~30%, whereas treatment with 25nM reduced viability by > 50%. Furthermore, a combination treatment of CR-31 and CB839 showed no difference at 25nM CR-31, whereas at 12.50nM a difference was observed. Therefore, combining the findings of these two studies indicate that CR-31 treatment in PDA indeed has a therapetuic effect, however the underlying mechanisms that are observed in the transcriptome wide analysis of translation efficiencies are likeley dependent on the cell culture and treatment dosages.


\section{Study 3 - mTOR-dependent translational buffering overrides transcriptome alterations leading to
maintained proteome composition}


