---
output:
  pdf_document: default
---

\chapter{Results and discussion}

\section{Study 1 - Generally applicable transcriptome-wide analysis of translation using anota2seq}

 Initially, changes in translation efficiencies were estimated using the TE-score approach as outlined in section \ref{algorithm}. However, this method was shown to be prone to spurious correlations. This leads to elevated false positive identification that can result in erroneous biological conclusions [@Larsson2010]. When using the TE-score, spurious correlations can be attributed to the inadequate correction for changes in total mRNA levels when estimating translation efficiencies [@Larsson2010;@Larsson2011]. The Analysis of Translation Activity (anota) algorithm facilitates analysis of translational efficiencies that are corrected for changes in total mRNA levels [@Larsson2011]. 

 Anota was developed for analysis of transcriptome-wide analysis for data quantified by DNA- microarrays [@Larsson2010]. However, advances in experimental methodologies lead to the development of RNA sequencing. RNA sequencing and DNA microarray data have distinct characteristics that need to be accounted for before analysis (**see section \ref{algorithm}**). Therefore, while the statistical framework of anota was shown as an adequate approach for analysis of translational efficiencies DNA microarray data, it was not directly applicable to RNA sequencing data (*see section \ref{algorithm}*). Efforts have been made to make RNA sequencing data more "DNA-microarray-like" so that algorithms developed for intensity based microarray data can be applied to count based RNA sequencing data [@Law2014;@Love2014]. Anota2seq, the algorithm developed in this study, allows for transformation and normalisation of RNA sequencing data so that the anota statistical frame work can be applied for analysis of count data. 

 Another feature of anota2seq is that it allows for statistical analysis of translational buffering. The need for the analysis of translational buffering, or the uncoupling of total mRNA levels from translation, has been noted before anota2seq's development by comparing 20 translatomes and transcriptomes with different underlying  stimuli in mammalian cells [@Tebaldi2012]. The same authors proposed a framework, called tRanslatome, that combines several methodologies for analysis of differential transcription and translation efficiencies, including anota, for a comprehensive analysis of transcription and translation as well as their underlying mechanisms [@Tebaldi2014].

\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_anota2seq.pdf}
  \caption{anota2seq gene model for analysis of translational buffering /offsetting - Total mRNA expression is set out against translated mRNA expression for each biological replicate and treatment condition. The model shows total mRNA changes that are independent of translated mRNA changes, i.e. translational buffering (see section \ref{modes}).
  \label{fig:anota2seq}}
\end{wrapfigure}

 Nevertheless, commonly observed in polysome and ribosome profiling data sets are three gene expression modes, translation, translational buffering and mRNA abundance. While anota can be used to identify genes among the translation and mRNA abundance mode, analysis of translational buffering was not implemented therein (*See Figure \ref{fig:anota}*). Therefore, one would need to rely on the integration of several methods to efficiently analyse transcriptome-wide studies of translation efficiences. Anota2seq addresses this issue by changing the analysis model as described in section \ref{algorithm} to analyse changes in total mRNA levels corrected for changes in translated mRNA levels (i.e. translational buffering, *see figure \ref{fig:anota2seq}*). 

 Application of anota2seq has successfully identified translational buffering to which biological mechanisms could be linked, e.g. as mentioned earlier translationally bufferring under ER\(\alpha\) depletion in prostate cancer (*see section \ref{modeBuffering}*) [@Lorent2019]. Furthermore, in **study 2** translational buffering can be observed as a compensating mechanism in "healthy" cells upon treatment with an eIF4A inhibitor and in **study 3** we identify  mTOR dependent translational buffering for mRNAs with certain 3' UTR characteristics.

 The aim of this study was to compare anota2seq's performance to other established algorithms (i.e. DESeq2, RiboDiff, babel, TE-score and Xtail) for analysis of translation efficiencies. More specifically, we were interested in their ability to distinguish the three prominent modes of gene expression (*see figure \ref{fig:modes}*). To achieve this we used simulated data. While it is arguable to what extent conclusion drawn from simulated data can be extended towards empirical data, it allows for a controlled environment where true positive changes are known in advance. Furthermore, the mean-variance relationship in the simulated data is based on a real polysome profiling data set to increase confidence that drawn conclusions are also applicable to empirical data [@Guan2017]. Furthermore, during testing of our simulation we used an additional data set to estimate parameters from to generate data sets. Using these simulated data to compare the performance of the above mentioned algorithms showed almost identical results. 

 The simulated data consisted of four replicates for translated mRNA and total mRNA with a "control" and a "treatment" condition. Furthermore, the data sets contained a combination of the following gene sets:
 
*Unchanged*: For this simulation category we sampled reads from the same NB distribution for both the control and treatment conditions in both the translated and total mRNA. This category represents genes that would be unaffected by e.g. a stimulus.

*mRNA abundance*: For this category the control condition for both the translated mRNA and total mRNA were sampled from the same NB distribution. The NB distribution for **both translated mRNA and total mRNA** of the treatment condition was altered so that values would be drawn corresponding to a fold change (negative or positive) ranging between 1.5 to 3.0. The directionality of the fold changes (i.e. up or down regulation) was the same for translated mRNA and total mRNA.

*translation*: For this category the control condition for both the translated mRNA and total mRNA were sampled from the same NB distribution. The NB distribution for **translated mRNA only** of the treatment condition was altered so that values would be drawn corresponding to a fold change (negative or positive) ranging between 1.5 to 3.0.

*buffering*: For this category the control condition for both the translated mRNA and total mRNA were sampled from the same NB distribution. The NB distribution for **total mRNA only** of the treatment condition was altered so that values would be drawn corresponding to a fold change (negative or positive) ranging between 1.5 to 3.0.

 As a first step, we tested whether the methods could properly control for type-1 errors (i.e. false positive identification). For this we simulated a data set with genes belonging only to the "unchanged" category. This revealed that babel, but to an even greater extent Xtail, were unable to control their type-1 error as these methods assigned low p-values and FDRs when no real changes were present. DESeq2 was marginally affected by this also. This indicated a limited applicability of Xtail and babel for statistical analysis of translatomes.

 From the comparative analysis  of the analysis for changes in translation efficiencies affecting protein levels we concluded that anota2seq outperforms all other methods. This was assessed by comparing the area under the curve from Receiver operating characteristics (ROC) and precision recall curves. The ROC curves showed a, albeit slightly, better performance for detecting changes in translation. However, the precision recall was much higher for anota2seq. This can be accredited to that the analysis principle of the other methods is similar to the TE-score (*as explained in section \ref{algorithm}*). Nevertheless, when comparing the performance using simulated data in the absence of genes belonging to the "buffering" category anota2seq still showed superior performance. 

 Knowing the simulated true changes in advance allowed us to modify parameters to investigate the robustness of the methods to increased variance, overall sequencing depth and differing sequencing depth between samples. Here, all methods showed robustness against variance,sequencing depth, and differences in sequencing depth between samples as long as a minimum of 5 million counts per sample was reached. 

 A shortcoming in the simulation study is that we did not assess the effects of systematic batch effects for all methods. Batch effects can be introduced during experimental design and can reduce statistical power. Currently there are many methods that try to correct for batch effects when present in the data [@Johnson2007;@Leek2014;@Zhang2020]. Other ways to correct for batch effects is to include them in the analysis model. For instance, Anota2seq, edgeR and DESeq2 allow for incorporation of batch effects in their analysis models. Indeed, analysis of a dataset with prominent batch effects showed that batch effects can dampen the efficiency of the anota2seq algorithm to identify changes. However, including a batch correction in the analysis model of anota2seq increased statistical power to detect changes drastically.

 In this study, we developed an analysis algorithm for efficient transcriptome-wide analysis of translation efficiencies applicable to DNA-microarrays and RNA seq. Furthermore, anota2seq has been successfully applied to broaden the knowledge around mRNA translation in various different contexts [@Lorent2019;@Chan2019;@Hipolito2019;@Chaparro2020]. Furthermore, more recently anota2seq has been used to compare mRNA levels between cytoplasmic mRNA and mRNA stored in P-bodes showcasing that anota2seq is generally applicable beyond analysis of translation efficiencies [@Bearss2021].
\newline
\section{Study 2 - eIF4A supports an oncogenic translation program in pancreatic ductal adenocarcinoma}

 Pancreatic cancer is considered a lethal malignancy and has limited treatment options. While other cancers (e.g. ovary, breast and stomach) showed a decline in mortality rates, no major overall reduction in mortality was observed for pancreatic cancer in the period of 1970-2020 [@Carioli2021].

 Pancreatic ductal adenocarcinoma (PDAC) accounts for over 90% of exocrine pancreatic cancer, whereas non ductal pancreatic cancers e.g. acinar cell carcinomas are uncommon [@Feldmann2007;@Jun2016]. It is estimated the 60-70% of the PDACs arise in the head of the pancreas [@Luchini2016]. So far treatment options are mostly limited to surgical removal, which often is impossible due to the anatomical location of the pancreas head. The 5-year survival rate for this disease is less than 10% [@Rawla2019]. A Dutch nationwide study indicated that in cases were surgical removal was possible 5-year survival only increased from 9.1% to 16.5% [@Latenstein2020].

 With the increasing understanding of tumor heterogeneity anti-cancer therapy improved [@Biankin2011]. For example, in breast cancer stratification by histological, molecular and gene expression features identified several breast cancer subtypes for which different treatment options exist, e.g. \(ER^+\) breast cancer subtypes respond to endocrine therapy, whereas \(ER^-\) do not [@Andre2006;@Parker2009]. While breast cancer treatment strategies benefit from a rather well established understanding of the molecular subtypes, in pancreatic cancer transcriptomic based subtyping is still ongoing [@Bailey2016;@Puleo2018;@Collisson2011;@Moffitt2015;@Collisson2019]. Therefore, insufficient understanding of molecular mechanisms that underpin PDAC hinder development of more efficient therapeutic approaches.
 
 While intricacies of molecular subtypes are still being investigated, research has shown that  oncogenic mutations in KRAS as well as inactivation of tumor suppressor TP53 are commonly shared among PDACs [@Jones2008]. Furthermore, PDACs have been shown to be dependent on increased protein synthesis mediated induced via KRAS mutations [@Chio2016]. This indicates an important role of mRNA translation in PDAC.  

 The aim of *study 2* was to investigate the therapeutic effects of targeting eIF4A in a murine three dimensional PDAC organoid cell culture with mutations in the \(Kras^{LSL-G12D}\), \(Trp53^{LSL-R172H}\) and Pdx1-cre alleles that has been shown to recapitulate PDAC tumor progression [@Boj2015]. Pancreatic and duodenal homeobox 1 (PDX1), is an important factor for pancreatic differentiation. PDX1 knock out mice failed to develop a pancreas [@Hale2005]. The inhibition of eIF4A was carried out using a synthetic rocaglate, CR-31. Rocaglates have been shown to inhibit eIF4A helicase function and display anti-tumor activity [@Cencic2009]. 

 We first wanted to establish the therapeutic validity of targeting eIF4A in PDAC. *In vitro* experiments comparing treated PDAC organoids (KP) to their normal (N) counter parts revealed heightened sensitivity of KP organoids to CR-31 treatment relative to N organoids. OP-puromycin incorporation showed reduced protein synthesis in KP organoids, whereas N organoids were affected to a lesser extent. Furthermore, similar effects were found *in vivo* for PDAC tumours. Here CR-31 reduced protein synthesis (assessed by SUnSET assay), tumor growth (assessed by ultra sound imaging) and increased survival of mice. The effect on protein synthesis was not due to inhibition of oncogenic signalling pathways which was evaluated via western blot assessing the phosphorylation of e.g. AKT, mTOR and 4E-BP1. From these findings we concluded that there is therapeutic validity in targeting eIF4A in PDAC.

 Using polysome profiling, we then sought to decipher the mechanisms explaining the increased sensitivity to CR-31 in KP organoids.  First we investigated the differences in gene expression between untreated KP organoids and N organoids. Analysis of changes in translation efficiencies using anota2seq revealed massive modulation at both total mRNA levels and translation indicative of underlying differences in e.g. genomic stability and enhanced oncogenic signalling impinging on protein synthesis reported in PDAC [@Boj2015]. Consistent with the *in vitro* OP-puromycin incorporation and *in vivo* SUNsET experiments, CR-31 strongly impacted global protein synthesis in KP organoids, while only exerting a modest effect in N organoids.

 We then compared the translatomes of untreated KP organoids to that of N organoids. This revealed differences at both total mRNA levels as well as translation. Treatment of KP organoids with CR-31 reversed the changes in untreated KP organoids as compared to N organoids. mRNAs affected by CR-31 in KP organoids showed modulated total mRNA levels in N organoids that were offset at the level of translation. Translational offsetting has been shown to keep protein levels constant despite alterions in total mRNA levels [@Lorent2019]. The ability for N organoids to modulate mRNAs levels affected by CR-31 that can be translationally offset, whereas KP cannot, could partially explain as to why protein synthesis is not reduced to a similar extent in N as in KP.

 We then assessed 5' UTR characteristics of the mRNAs whose translation was affected upon CR-31 treatment in KP organoids. It was reported that eIF4A-senstive mRNAs showed overall and more structured 5' UTRs (e.g. containing G-quadruplexes) [@Rubio2014;@Wolfe2014;@Gandin2016a]. Furthermore, a mechanisms by which rocaglates would clamp eIF4A to mRNAs with [A,G] repeats in their 5 UTR was described [@Iwasaki2016]. However, mRNAs sensitive to CR-31 treatment herein had short 5' UTRs that were more structured when corrected for their length  without enrichment for 4G-quadruplexes or  [A,G] repeats. Therefore, CR-31 sensitive mRNAs in KP organoids show 5' UTR characteristics different from those reported in the literature (*see section \ref{sel4F}*). However, based on our polysome profiling and 5' UTR analysis, we concluded that eIF4A supports an oncogenic translation program in PDAC cells for mRNAs with shorter but more structured 5' UTRs as compared to our background set (i.e. all non-regulated mRNAs).

 Translation of mRNAs harbouring shorter 5' UTRs has been shown to be sensitive to eIF4E expression and encode for metabolic functions [@Gandin2016a]. When we compared an eIF4E overexpression signature in the KP vs N and CR-31 treated KP we observed that in KP organoids translationally regulated mRNAs under eIF4E overexpression were also translationally activated. This observation is consistent with reports of 4E-BP1 loss in pancreatic cancer and consequently increased ability of eIF4E to engage in the eIF4F complex [@Martineau2014]. eIF4A inhibition in tumors resistant to mTOR inhibiton by loss of 4E-BP1 has been shown to circumvent this resistance [@Muller2019]. Indeed, CR-31 treatment in KP organoids reversed the translational profile for eIF4E-sensitive mRNAs. 

 We further inspected the regulated gene sets in treated and untreated KP organoids compared to N organoids. Here we could see an enrichment in metabolic pathways, e.g. Oxidative phosphorylation. This pathway was upregulated at the polysome associated mRNA levels in untreated KP compared to N, whereas in KP organoids CR-31 treatment reversed the translational profile of this pathway. Furthermore, CR-31 treatment in KP organoids lead to reduced oxygen consumption rates, whereas N organoids where affected to a lesser extent. While measuring oxygen consumption rates do not rule out that non-mitochondrial sources are affected, we attributed the observed decrease in oxygen consumption to defective oxydative phosphorylation. 

 A way to counter loss of energy production through oxidative phosphorylation is to increase activity of other metabolic pathways, i.e. glycolysis. However, in CR-31 treated KP porganoids we could not detect an upregulation of glycolysis measured by \(U-C^{13}\) glucose labeling and extra cellular acidification rates nor did CR-31 treatment affect  expression of glycolytic enzymes (e.g. HK1,HK2, LDHA, SLCA1, SLCA3). Furthermore, glucose deprivation did not further sensitise to CR-31 treatment. However, the polysome profiling data revealed translational downregulation and subsequent reduction of protein expression for the glucose transporter Slc2a6. Indeed, perturbation of Slc2a6 using \(sgRNA^{Slc2a6}\) in N and KP organoids revealed a decrease in glucose uptake. From this we concluded that glycolytic compensation of KP is diminished by translational regulation of the glucose transporter Slc2a6 upon CR-31 treatment.

 Among the translationally activated genes in the CR-31 treated KP organoids where mRNAs involved in the glutamine metabolism (i.e. Slc1a5 and Gls1). Furthermore, glutamine levels were elevated in patient derived PDAC cell lines treated with CR-31. Glutamine can be converted into \(\alpha\)-ketoglutarate which can undergo reductive carboxylation to produce citrate [@Xiao2016a]. Indeed, using gas chromatography mass spectrometry (GC/MS) to quantify metabolites after labelling PDAC cells in \(C_5^{13}\)-glutamine, we identified a shift towards reductive carboxylation of \(\alpha\)-ketoglutarate obtained from \(C_5^{13}\)-glutamine to produce citrate. Notably, the reductive glutamine metabolism was not elevated in N organoids.

 A combined treatment of CR-31 with glutaminase inhibitors (BPTES or CB839) could sensitise to CR-31 treatment patient-derived PDAC cells to CR-31 treatment indicating that glutamine utlisation is important therein. Therefore, our study suggests an eIF4A dependent translational program in PDAC that can act as a theurapeutic target in PDAC. Furthermore, a recently published ribosome profiling study of a CR-31 treated human pancreatic cancer cell line (PANC1) observed the same therapeutic effect of CR-31 treatment *in vivo* on survival and tumor volume [@Singh2021]. This underlines the significance of our study in identifying eIF4A as therapeutic target in PDAC. 

 Nevertheless, the same study indicated differences on 5' UTR characteristics of eIF4A-sensitive mRNA subsets [@Singh2021]. They report, in line with the literature, that eIF4A dependent mRNAs show long and structured 5' UTRs containing \(GGC_4\) sequence motifs they propose to form G-quadruplexes [@Singh2021]. The 5' UTR characteristics of the mRNAs proposed in our study resembled more those of mTOR-eIF4E sensitive mRNAs proposed by Gandin et al., i.e. shorter 5' UTRs as compared to our background [@Gandin2016]. This raises some questions about the differences between experimental setups and their potential influence on biological outcomes and interpretation thereof. For instance, Singh et. al. performed ribosome profiling on a PANC1 cell line culture treated with 25nM CR-31, wheres herein we performed polysome profiling on a 3D-organoid culture treated with 10nM CR-31. The differences between ribosome and polysome profiling have been discussed extensively (see section \ref{exptMethod}). Furthermore, by measuring IC50 concentrations for CR-31 in a panel of pancreatic cancer cell lines, Singh et. al. show a ~6-fold difference in susceptability to CR-31 between the cell lines of which PANC1 cells were most sensitive to CR-31. Dosage dependent viability experiments of patient derived PDAC cells in our study revealed that at 10nM CR-31 treatment cell viability was reduced by ~30%, whereas treatment with 25nM reduced viability by > 50%. Furthermore, in patient derived PDAC at 25nm CR-31 a combination treatment of CR-31 and CB839 did not alter cell viability compared to CR-31 treatment alone. However, at 12.50nM CR-31 treatment, combined treated with CR-31 and CB839 further reduced viability. Therefore, combining the findings of these two studies indicate that CR-31 treatment in PDAC indeed has a therapeutic effect. However the underlying mechanisms that are observed in the transcriptome wide analysis of translation efficiencies could be dependent on the experimental method to assess mRNA translation, the model system and drug concentrations.
\newline
\section{Study 3 - Insulin signalling gene expression landscapes distinguish non-transformed vs. BCa cells}

 An important factor to consider in breast cancer treatment are life style and other health related issues that could impact cancer progression or response to treatment, e.g. obesity. Studies in the 1970 observed unfavorable prognosis for breast cancer in obese women [@Abe1976]. Obesity can pose an increased risk to develop metabolic disorders such as metabolic syndrome or type 2 diabetes that can lead to hyperinsulinemia, i.e. elevated physiological insulin levels [@Saltiel2001]. 

 The role of insulin in the body is to regulate glucose and lipid metabolism as well as protein synthesis of which dysregulation are hallmarks of cancer [@Saltiel2001;@Hanahan2011]. Insulin binds to insulin receptor (IR) A and B that activate the PI3K/AKT/mTOR and mitogen activated protein kinase (MAPK) signalling pathways. A role of insulin in cancer progression has initially been observed in long term tissues cultures where it increased metabolism as well as growth [@Osborne1976]. IGFs (i.e. IGF1 and IGF2) carry out similar roles as insulin and bind to insulin-like growth factors receptor 1 (IGFR1). Insulin and IGFs can bind to either IRs or IGF1R. Furthermore, IRs and IGF1R have been shown to form homo- and heterodimers, e.g. IR-A/IGF1R dimers. Because they act on same receptors, insulin and IGFs activate the same signaling pathways albeit with different affinities [@Pollak2008;@Boucher2010]. Additionally, IGF1 plays a role in cancer progression and its levels are elevated upon hyperinsulinemia [@Christopoulos2015;@Bailyes1997;@Molinaro2019;@Gallagher2010]. 

 The importance of both IGF1 and insulin signalling in cancer is well-established and led to development of therapeutic strategies by e.g. targeting both IGF1R and IR or the PI3K signalling pathway [@Kuijjer2013;@Mayer2017]. Yet, to this day the full mechsanism of insulin/IGF action in cancer remains poorly understood. This study aims to bridge this gap in knowledge by elucidating the effects of insulin signalling on gene expression using a multi-omics (including transcriptomics, translatomics and proteomics) approach to capture multiple steps of the gene expression pathway simultaneously. Furthermore, we assess insulin signalling in cancer cells as well as non-transformed epithelial cells.

 We first investigated the effects of insulin on gene expression in a luminal A breast cancer cell line, i.e. MCF7 cells. MCF7 cells harbour the E545K PI3KC mutation and are senstive to insulin stimulation. Polysome-profiling revealed a strong modulation of total mRNA levels and translational response upon insulin stimulation. Among the translationally activated mRNAs were mRNAs with short 5' UTRs that harboured TOP motifs which is in accordance that insulin signalling leads to activation of mTORC1 and phosphorylation of its downstream targets. When visualising the mRNAs in the data where MCF7 cells were stimulated with insulin in the presence of torin1 we observed that the changes in mRNA translation were almost fully reversed. Torin1 is an mTOR active site inhibitor. This led to the conclusion that the effects of insulin on gene expression are to a great extent dependent on mTOR activity. 

 What was surprising to us was the observation that a subset of mRNAs exhibited translational buffering upon insulin stimulation while mTOR is inhibited. For these mRNAs, the total mRNA levels were increased, whereas the polysome-association was unaltered between conditions Thus, changes in total mRNA levels where offset at the level of translation. Using HiRIEF LC/MS we show that translationally offset mRNAs maintain constant protein expression across conditions [@Branca2014]. The ability of translational offsetting to maintain constant protein levels across conditions despite alterions in total RNA levels has been shown by others before [@Lorent2019]. These findings indicate that mTOR can act as a gatekeeper for transcriptional programs to fine tune translation in response to extra cellular or intra cellular signals.

 MCF7 cells are of epithelial origin and therefore not "classical" insulin-sensitive such as adipose tissue or muscle cells. The strong response of MCF7 cells to insulin prompted us to investigate whether this could be an adapted mechanism in cancer. To assess this we chose to compare the effects found in MCF7 to a non-malignant epithelial cell type, i.e. HMEC cells. We found that insulin alone was not sufficient to stimulate the PI3K/AKT/mTOR pathway in HMEC to a similar extent as in MCF7. However, a combination treatment of insulin and IGF1 in HMEC cells elicited a similar reponse as insulin treatment in MCF7 alone. We therefore opted to compare the combined insulin + IGF1 treatment in HMEC to that of MCF7 assuming their signalling cascades are nearly identical as proposed in the literature [@Pollak2008;@Boucher2010].

 Polysome profiling of the insulin + IGF1 stimulated HMEC cells revealed a strong translational response which was similar to MCF7 cells. Translationally activated mRNAs in HMEC showed 5' UTR features similar to those in MCF7 cells. Consequently, their translational activation was dependent on mTOR signalling evident from their translational suppression under conditions when mTOR was inhibited during insulin + IGF stimulation. Furthermore, comparing the mRNA signatures of HMEC in the expression data origating from MCF7, and *vice versa*, we suggested that changes at the level of translation were almost fully in accord across cell types.

 In contrast to MCF7 cells, HMEC did not elicit a strong strong modulation of total mRNA levels as shown by the small number of changes in mRNA abundance. When comparing a recently described transcription signature induced after IR translocation to the nucleus, we could see that in both HMEC and MCF7 these mRNA show changes in total mRNA levels but of differing magnitude [@Hancock2019]. A possible explanation for the different response in total mRNA levels between the cell clines could be due to differences in, e.g. chromosome instability that expose different parts of the DNA to trans acting factors. While not assessed herein, a transcription factor analysis paired with chromatin immunoprecipitation (ChIP) sequencing could provide insight into this [@Solomon1988;@Park2009]. Assuming a consequence of having a weaker modulation of total mRNA leves, HMEC cells did not elicit translational offsetting upon insulin + IGF1 stimulation when mTOR was inhibited. Thus the effects of insulin and IGF1 signalling on mRNA translation are foremost mTOR dependent, however total mRNA responses differ between malignant and non-malignant epithelial cells.

 The translational offsetting in MCF7 drove us to investigate this phenomenon more. To assess differences dependent on mRNA characteristics we defined two subsets. The "reversed" and the "uncoupled" (that is translationally buffered when mTOR is inhibited) subsets that only differed in their total mRNA response when mTOR was inhibited during insulin stimulation. To rule out that the observed effects on total mRNA are technical artifacts we validated total mRNA levels for two genes from each subset. The differences in regulation of gene expression were not dependent on codon usage which has been described before in a different context [@Lorent2019]. However, overall uncoupled mRNAs had shorter 3' UTRs with a higher GC content and were depleted for HuR binding sites. 

 The depletion of HuR binding sites in the 3' UTRs of the uncoupled subset prompted us to investigate mRNA stability differences. Using a time series experiment under actinomycin D treatment to block transcription quantified using nanoString, we found significant longer mRNA half lifes for the uncoupled subset as compared to the reversed subset. Based on these data, we hypothesised that there are different underlying mechanisms that regulate gene expression under conditions where mRNA translation is dampened between these subsets in MCF7 cells. Under this hypothesis: On one hand, the reversed subset could be regulated through mRNA stability. Therefore, when translation is inhibited total RNA levels would be reduced by destabilisation of the mRNA. On the other hand the uncoupled subset cannot be de-stabilised and has longer half lives than the reversed subset. Therefore, their increased total mRNAs levels require to be translationally offset under conditions when mRNA translation is inhibited.

 The involvement of HuR in this cannot be fully supported with our current data as the analysis only supports a correlation between the occurrence of HuR binding sites and the 3' UTRs of the reversed subset. The effect on stability could be due to other trans acting factors, e.g. miRNAs and other RBPs [@ValinezhadOrang2014]. A way to increase confidence is to investigate HuRs involvement experimentally, e.g. we could use a small interfering (siRNA) to silence HuR and measure total mRNA levels of the reversed and uncoupled mRNAs previously validated by qPCR. If HuR is involved, we would expect that the reversed mRNAs retain higher total mRNA levels under the condition where mTOR is inhibited during insulin stimulation in MCF7 cells as compared to control. Furthermore, while the differences in total mRNA levels between the insulin and the insulin and torin1 treated conditions in the RNAseq data imply a treatment effect on the mRNA stability we did not observe this in the time chase experiment quantified by nanoString. This raises the question whether the transcription block induced by actinomycin D could influence the regulation of mRNA stability between treatments. We could address this by including actinomycin D in the siHuR experiment and see if effects thereof differ or setup an experiment independent of siHuR. Presence of an effect of actinomycin D on mRNA stability could indicate a cross-talk between transcription and regulation of mRNA stability. 

 Since the translational offsetting identified herein was only observed in insulin treated cancer cells, we wondered whether this is only specific to this system. It has been shown that cancer cells can become more "stem-cell-like" [@Jewer2020; @Wahl2017; @Quail2012]. Therefore, we reasoned that a system were we study gene expression of stem cells could give some insight whether cancer cells obtained stem cell features that normal epithelial cells do not have. Furthermore, we wanted to investigate whether other means of mTOR inactivation, e.g. hypoxia, would lead to similar effects on gene expression. To assess these aspects we cultured H9 stem cells in medium with insulin present in normoxia and hypoxia. This experimental setup differs to that of MCF7 and HMEC cells as these were serum starved (i.e. no insulin in medium) prior to induction with insulin.

 Studying this system in H9, we observed changes for all three modes for regulation of gene expression. Most notably, we observed a large fraction of translationally buffered mRNAs with similar 3' UTR characteristics to that of the uncoupled subset in MCF7 cells. Using publicly available data on mRNA stability we saw that the translationally buffered mRNA were overall more stable as compared to their background. Furthermore, visualising the reversed and uncoupled subset identified in MCF7 in the H9 data we observe differences in their regulation of total mRNA levels, indicating that these subset underlie different modes of regulation even across these two models. Furthermore, these data argue for that translational buffering observed in insulin treated MCF7 cells during mTOR inhibition is not limited to that system but can also occur under more other conditions where mTOR is inhibited. 

 Here, we present an unprecedented and comprehensive investigation of the effects on insulin on gene expression in cancer cells and non- transformed epithelial cells across multiple steps of the gene expression pathway. Our results indicate that cancer cells have acquired an increased sensitivity to insulin signalling as compared to non-transformed epithelial cells that is largely dependent on mTOR in both cell types. Furthermore, we observed that cancer cells have the ability to translationally buffer mRNAs which is a feature they share with stem cells.














