---
output:
  bookdown::pdf_book:
    includes:
      in_header: preamble.tex
    keep_tex: yes
    pandoc_args: [ "--csl", "_apa.csl" ] # apa citation style
  latex_engine: pdflatex
  pdf_document:
    fig_caption: yes
    number_sections: yes
fontsize: 12pt
linestretch: 1.25
toc-depth: 3
secnumdepth: 5
site: "bookdown::bookdown_site"
documentclass: book
classoption: openany # This removes the blank page after ToC
geometry: "left=4cm, right=3cm, top=3cm, bottom=3cm"
header-includes:
- \usepackage{placeins}
- \usepackage{titlesec}
- \newcommand{\sectionbreak}{\clearpage}
- \usepackage{amsmath}
- \usepackage{wrapfig}
- \usepackage{caption}
- \captionsetup[figure]{font=scriptsize}
- \usepackage{float}

subparagraph: yes
bibliography: library/thesis.bib

---

\cleardoublepage
\pagestyle{fancy}
\fancyhf{}
\renewcommand{\headrulewidth}{0pt}
\fancyfoot[LE,RO]{\thepage}
\renewcommand{\floatpagefraction}{.9} <!-- This specifies that only pages with more than 90% of floats will become a figure only page -->
\setcounter{page}{9} 

# Abbreviations {-}

```{r, echo=FALSE, warning=FALSE,message=F,prompt=F}
library(tidyverse)
library(knitr)

tibble(
  Abbreviation = c("RPF","TE","UTR","TOP","uORF"),
  Term = c("Ribosome Protected Fragment","Translation Efficiency","Untranslated region","Terminal oligopyrimidine","upstream open reading frame")) %>%
  arrange(Term) %>% # i.e. alphabetical order by Term
  kable(booktab = T, linesep = c("")) # linesep command suppresses the standard 5 row chunking
```

<!-- using inkscape filename.svg --without-gui --export-pdf=filename.pdf to export from Inkscape svg files to pdf -->

<!--chapter:end:index.Rmd-->

---
output: pdf_document
---

\chapter{Introduction}

\section{Gene expression}
The whole genetic code of an organisms is stored in long deoxyribonucleic acid (DNA) molecules called chromosomes. Chromosomes hold the DNA in a condensed state using chromatin, a complex of DNA and proteins, structures. The genetic code holds information on ~21 000 protein coding genes located within < 2% of the whole genome. While the rest of genome (i.e. 98%) historically has been labeled as "junk" DNA it has become more and more clear that it contains regularory elements of gene expression such as long non-coding RNAs (lcRNAs). DNA transcription can occur when chromatin is remodeled to expose promotor regions to which transcription factors, and other assisting factors, can bind. Resulting from DNA transcription are primary ribonucleic acid (RNA) transcripts. The RNA transcripts undergoe processing to remove non protein coding sequences (introns) from the primary transcript leaving the messenger RNA (mRNA). mRNAs carry the protein coding genomic information stored in the DNA. Once formed mRNAs are transported into the cytoplasm where they are either degraded or associate with ribosomes. These ribosomes translate the mRNAs into proteins by which the genetic information then is expressed. Synthesised proteins, if no longer needed in the cell, can be degraded by proteosomes (**see figure \ref{fig:geneExprPath}**). This flow of genetic information into expressed proteins is commonly referred to as the central dogma in molecular biology.

The rates at which these four steps regulate gene expression dependend on the underlying mechanisms at play. In the case of transcription chromating remodeling plays an important by opening or closing certain parts of the chromosome, whereas mRNA translatio is heavily regulated by sequence features of the mRNA. However regulation that couples 2 or more layers of the gene expression pathway has been observed as well [@Komili2008,@Pleiss2007,@Lorent2019]. While the relative contribution of each step of the gene expression pathway to shape the proteome is tissue and context dependent. Other have observed that transcription determines tissue specific gene expression (i.e. which genes are expressed in a given tissue), while mRNA translation is the chief regulator shaping the proteome within the same cells (to what extent are these genes expressed) [@Kim-Hellmuth2020, @Schwanhausser2011, @Eraslan2019]. Furthermore, mRNA translation allows for rapid adaptation to cellular or environmental cues (< 30 minutes ), whereas changes in transcription are comparatively slow ( > 4 hours).
Translational control (i.e. regulation of mRNA translation) therefore is an important aspect for regulation of gene expression. mRNA translation  is the most energy demanding process in the cell and is therefore tightly linked to the energy metabolism [@Buttgereit1995]. Translation plays major roles in cellular functions such as prolifertion (increase cell count) and growth (expand cell size) and deregulation thereof is often found in cancer cancer (**see section \ref{translationInCancer}**)[@Graff2009,@Topisirovic2011,@Lee2021]. This thesis focusses on the role of mRNA translation in gene expression, specifically in the context of cancer.
\clearpage

\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/geneExprPath_2.pdf}
  \caption{The gene expression pathway - DNA is transcribed in pre-mRNA containing a 5' cap (black oval) introns (teal boxes),exons (purple boxes) and a poly(A) tail. Via RNA-splicing different mRNA isoforms can be formed by splicing variations of intron / exon combination. The mature mRNA, consisting out ouf a 5' cap,exons and a poly(A) tail is then transported out of the cellular nucleus into the cytoplasm. Within the cytosplasm mRNAs can follow multiple paths such as storage, degradation or transaltion depending on cellular demands. mRNAs are translated into proteins (i.e. protein synthesis). Synthesised proteins can be degraded by proteosomes. \label{fig:geneExprPath}}
\end{wrapfigure}

\section{mRNA translation}

\paragraph{overview of an mRNA}
After transcription primary RNA transcripts are processed into mRNAs which is the products that will be translation. The coding region of an mRNA is flanked by untranslated region that exert translational control over the mRNA (see \ref{UTR}). The 5' has a cap that is important for mRNA translation initiation[@Grifo1983], while the 3' end has a poly-A tail protecting the mRNA against degradation [@Wilusz2001]. Multiple different mRNAs (isoforms or transcript variants) from the same genomic region exist. These variants arise due to a process called alternative splicing which alters the exon composition (i.e. coding region) of an mRNA with distinct functions [@JolyAnne-Laure2018]. 

For the vast majority of protein coding mRNAs, eukaryotic mRNA translation occurs in the cytoplasm, however a small subset of mRNAs is translated in the mitochondria. mRNA translation is a process that includes initiation, elongation, termination and ribosome recycling and is an essential process **see Figure \ref{fig:doodlemRNASteps}**. During the initiation phase a ribome will associate with the mRNA and starts scanning along the mRNA for a start codon to begin synhtesis of the polypeptide chain by incorporating amino acids. The order by which amino acids are incorporated are dictated by subsequent codons of the open reading frame (ORF).  Redudancy in the codon availability allows that amino acids have multiple assigned codons, for example lysine is encoded by AAA and AAG.   Amino acids are transferred to the ribosome by specialised RNAs called transfer RNAs (tRNA) that can recognise the genetic code in the mRNA. The availability of tRNAs as well as their modification can influence the rate of elongation (see \ref{tRNA}) Once the ribosome encounters a stop codon translation will terminate and the polypeptide chain will be released. The ribosome then disassociates from the mRNA and the ribosome can be recycled to engage translation of the same or another mRNA. The following setions describe these processes in more detail. 

\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/doodleTranslation.pdf}
  \caption{mRNA translation initiation, elongation, termination and ribosome recycling steps - The ribosome binds to the mRNA and initiates scanning for a start codon. The elongation phase incorporates amino acids into a polypeptide chain (i.e. the protein product). Once the end of the coding sequence is detected, by recognition of stop codons, the ribosome terminates translation and releases the polypeptide chain. The ribosome can then be recycled to participate in the translation of another mRNA or reinitiate. \label{fig:doodlemRNASteps}}
\end{wrapfigure}
\clearpage

\subsection{initiation} \label{initiation}

Of the three steps in mRNA translation the initiation steps is commonly regarded as the rate limiting step of translation[@Asano1997]. While initiation requires many small sequential modifications outlined in figure **REFFIG** there are broader important key aspects. For the initiation to commence, in eukaryotes, two complexes are required; the pre-initiation complex (PIC) and the eukaryotic initiation factor 4F (eIF4F) complex. Both these complexes are governed by signalling pathways that regulate their availability dependent on cellular cues (see \ref{signalling}). The PIC consists of the methionyl-initiatior transfer RNA (met-tRNAi) in a ternary complex (TC) with guanosine triphosphate (GTP) bound eIF2 [@Asano2000]. eIF4F is the translation initation complex containing three EIFs; eIF4E, the 5' cap binding protein, eIF4G a scaffold protein and eIF4A and RNA helicase [@Grifo1983,@Hinnebusch2009]. eIF4F recruites the PIC to the 5' cap of the mRNA after which scanning for a start codon (AUG) occurs. After AUG recognition  eIF2-GTP  is  hydrolyzed  forming  a  stable  48S  PIC.  Afterrelease  of  eIF2-GTP  the  60S ribosomal  subunit  joins  to  form  the  80S  ribosome  and  protein  synthesis can  commence. Next  to  this  scanning mechanism, mRNA translation can also be initiated via alternative cap independent mechanisms[@Wurth2015] (see also \ref{UTR}).

INSTERT FIGURE WITH PERMISSION

\subsection{elongation}

The 80S ribosome contains three sites important for decoding an mRNA; the acceptor (A), peptidyl (P) and Exit (E) sites. During elongation in eukaryotes animoacytelated tRNAs are delivered to the A-site in a ternary complex with eukaryotic elongation factor 1A (eEF1A). When the tRNA recognises its cognate codon and pairs, a bond between the amino acid and the polypeptide chain is formed. The formation of the bonds causes the ribosomal units to rotate in relation to each other [@Munro2007,@Moazed1989]. The rotation causes a shift of the tRNA acceptor ends from the A and P to the P and E sites, wheras the codon end remains in the A and P site. This is the "hyrbid" state of the tRNAs in the ribosome[@Dorner2006]. eEF2 then promotes the translocation by which the codon ends of the tRNA follow into the P and E sites. The deacytelated tRNA is then released from the ribosome. This process is repeated until a stop codon (UAA, UGA or UAG) is detected by the ribosome which terminates translation.

\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/elongation.jpg}
  \caption{Model of the eukaryotic translation elongation pathway. In this model the large ribosomal subunit is drawn transparent to visualize tRNAs, factors, and mRNA binding to the decoding center at the interface between the large and small subunits and tRNAs interacting with the peptidyl transferase center in the large subunit. Starting at the top, an eEF1A·GTP·aminoacyl-tRNA ternary complex binds the aminoacyl-tRNA to the 80S ribosome with the anticodon loop of the tRNA in contact with the mRNA in the A site of the small subunit. Following release of eEF1A·GDP, the aminoacyl-tRNA is accommodated into the A site, and the eEF1A·GDP is recycled to eEF1A·GTP by the exchange factor eEF1B. Peptide bond formation is accompanied by transition of the A- and P-site tRNAs into hybrid states with the acceptors ends of the tRNAs moving to the P and E sites, respectively. Binding of eEF2·GTP promotes translocation of the tRNAs into the canonical P and E sites, and is followed by release of eEF2·GDP, which unlike eEF1A does not require an exchange factor. The ribosome is now ready for the next cycle of elongation with release of the deacylated tRNA from the E site and binding of the appropriate eEF1A·GTP·aminoacyl-tRNA to the A site. Throughout, GTP is depicted as a green ball and GDP as a red ball; also, the positions of the mRNA, tRNAs, and factors are drawn for clarity and are not meant to specify their exact places on the ribosome. Thomas E. Dever, and Rachel Green Cold Spring Harb Perspect Biol 2012;4:a013706. Reprinted with permission. ©2012 by Cold Spring Harbor Laboratory Press
 \label{fig:elongation}}
\end{wrapfigure}

\clearpage

\subsection{termination and recycling}

mRNa translation termination is  facilitated  by  two  release  factors (eRF), eRF2  and  eRF3-GTP[@Stansfield 1995,@Alkalaeva2006]. A TC  containing  eRF2  and  eRF3-GTP binds to the A-site of the ribosome upon recognition of a stop codon. This causes an ATP hydrolysis event resulting in a conformational change and release of the polypeptide chain. eRF1 and the ATP binding cassette protein (ABCE1) together promote  the  splitting  of  the  60S  and  40S  subunits after which they can be recycled[@Pisarev2010,@Dever2012,@Hellen2018].



\section{Regulation of mRNA translation}

Of the heretofore presented steps of mRNA translation the initiation is the most regulated step. From the perspective that mRNA translation contributes most to the cellular energy consumption, of which the initiation requires ATP, this process ought to be strictly regulated [@Buttgereit1995,@Jackson1991,@Sonenberg2009]. Nevertheless, translation can also be regulated at the elongation [@Richter2015] and termination[@Dever2012] phases albeit to a lesser extent. Dynamic modulation of mRNA translation can be achieved through signalling pathways as well as several distinct cis and trans elements in both untranslated regions (UTRs) of an mRNA. Regulation of mRNA translation can be exerted at a global level i.e. reduction of protein synthesis for a large portion of the transcriptome. Furthermore, a more selective regulation of mRNA translation can be achieved through various mechanisms, which increases the complexity of the regulation of mRNA translation greatly.

\subsection{Signalling pathways impinging on mRNA translation} \label{signalling}

\paragrah{the mTOR pathway} \label{mTOR}

mTOR  is  a  conserved  Ser/Thr  kinase  and  is  found  in  two  structurally  and  functionally  distinct  complexes,  mTORC1  and mTORC2. [@Saxton2017,@Pearce2007]. mTOR activity is modulated via growth factor signalling as well as cellular metabolism. While mTORC1 down stream signalling impinges on differention and growth, via phosphorylation of 4E-BPs and S6k). mTORC1 promotes survival via signalling through protein kinase A (AKT). 



In  regards  to  regulation  of  mRNA  translation,  mTORC1  is  a  key  player  in  regulation  of  translation  initiation through  facilitating  the  release  of  eIF4E (**see section \ref{initiation})  from  4E-BPs via  phosphorylation  of  4E-BPs  by  mTOR  [@Hsieh2010]. Additional downstream targets of mTOR include signal transducer of transcrption 3 (STAT3) that regulates proliferation and growth as well as HIF1\alpha, a transcription factor showing the involvement of mTOR across the gene expression pathway

\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/mTORsignal.jpg}
  \caption{Schematic representation of mTOR signaling to the translational machinery. Growth factors stimulate mTORC1 signaling by activating receptor tyrosine kinases (RTKs) located at the plasma membrane. Various adaptor proteins convert these extracellular signals by stimulating the PI3K/AKT and Ras/ERK pathways. Many additional cues promote mTORC1 activation, including glucose and amino acids via small Rag GTPases, which help translocate mTORC1 to the surface of lysosomes. In turn, insufficient energy resources (energy stress) and hypoxia inactivate mTORC1 via the LKB1/AMPK pathway and REDD1, respectively. mTORC2 also responds to agonists that stimulate the production of phosphatidylinositol-3,4,5-triphosphate (PIP3) and promotes the activity of AGC kinase family members (PKC, AKT, and SGK) by phosphorylating residues located in their hydrophobic motifs. mTORC1 modulates mRNA translation by promoting the phosphorylation of downstream substrates, including the 4E-BPs and S6Ks, the latter having phosphorylation substrates of their own (e.g. eIF4B, rpS6, PDCD4, and SKAR). Red T-bars represent inhibitory signals, whereas black arrows indicate stimulatory signals. P denotes phosphorylation. Abbreviations and detailed explanations about this signaling network are provided in the text. Philippe P. Roux, and Ivan Topisirovic Mol. Cell. Biol. 2018; doi:10.1128/MCB.00070-18. Reprinted with permission. Copyright © 2018, American Society for Microbiology
 \label{fig:elongation}}
\end{wrapfigure}

\clearpage

\paragraph{regulation of mTOR activity}

Next to the activation of mTOR through signalling pathways activated by growths factor es mentioned above there are other cellular cues governing mTOR activity. These cellular cues are centered around the energy status of the cell sensed through the ratio of adenosine-mono-phosphate (AMP) to adenosine-tri-phosphate (ATP) that actives the AMP-kinase (AMPK) [@Sanders2007,@Kimball2006]. Increase in AMPK activity leads to an inhibition of mTOR and therefore down regulation of mRNA translation initiation[@Inoki2003]. Nutrient availability (i.e. glucose and amino acids), oxygen levels are interwoven with cellular energy production by affecting the AMP/ATP ratios in the cell [,@Heerlein2005]. The role of cellular metabolism, of which its deregulation is considered a hallmark of cancer, on mTOR activity suggests that mRNA translation plays an important role in this disease (see \ref{translationInCancer}.

\paragrpah{Global and selective regulation of translation via mTOR}



Furthermore, substrates of mTORC1 include ribosomal S6 kinases (S6Ks) 1 and 2[@Schepetilnikov2013], and La ribonucleoprotein domain family member 1 (LARP1)[@Tcherkezian2014]. mTORC2 is found to associate with ribosomes to promote co-translational phosphorylation and foldingofnascent Akt polypeptide[@Oh2010]. As  mentioned  mTORC1  is  activated  via  growth hormones including insulin  and  insulin  like  growth  factor  (IGF). Upon binding of insulin to  the  insulin  receptor, tyrosine  kinases  (RTKs)  and  phosphoinositide  3-kinase  (PI3K)  are activated.  Phosphatidylinositol  3,4,5-triphosphate  (PIP3)  is  then  generated  by  PI3K  from  Phosphatidylinositol  4,5-biphoasphate  (PIP2).  This  step  is  reversed  by  PTEN  which  hydrolyzes  PIP3  to  PIP2,  thereby  working  antagonistically  to PI3K.  PIP3  recruits  AKT  and  phosphoinositide-dependent  kinase1  (PDK1)  towards  the  plasma  membrane  where  AKT  is phosphorylated  by  PDK1.  Ras  homologue  enriched  in  brain  (Rheb)  is  a  GTPase  that  stimulates  mTOR  in  its  GTP  bound form.  The  tuberous  sclerosis  complex  (TSC)  consists  of  TSC1  (scaffold  protein)  and  TSC2  is  a  GTPase-activating  protein (GAP) which inhibits Rheb through hydrolysis of Rheb-GTP to  Rheb-GDP, thereby inhibiting  mTOR activity. AKT mediates phosphorylation  of  TSC2,  leading  to  a  decreased  GAP  activity  and  reduced  mTOR  inhibition.  Signaling  through  the  Ras GTPase  by  growth  factors  may also  activate  mTORC1  through  the  RAF/MEK/ERK  axis  whereby  extracellular  signal-regulated kinase (ERK) leads to direct phosphorylation of TSC2 and raptor or via the RSKs[@Roux2018,@Roux2012,@Laplante2012].

\paragraph{Global regulation of translation by mTOR}
Protein synthesis is the most energy expensive process within cells[@Buttgereit1995].Therefore,regulation of mRNA translation is tied to cellular energy levels. AMP-activated protein kinase (AMPK) is a kinase activated by increased AMP/ATPratioas well as ADP/ATP  ratios.  AMPK  inhibits  protein  synthesis  by  activation  of  TSC2,  thereby  reducing  mTOR  activity.  Furthermore, cellular  oxygen  levels  are  linked  to  ATP  production,  where  low  oxygen  levels  reduce  ATP  production  leading  to  AMPK activation[@Leibovitch2018]. mTOR modulates global mRNA translation mainly through modulation of 4E-BPs and S6Ks[@Fonseca2014,@Hay2004]. S6Ks (S6K1 and S6K2) phosphorylates multiple component of the translational machinery such as RPS6, which is implicated  in  ribosome  biogenesis[@Magnuson2012]. Furthermore,  S6Ks  also  phosphorylates eEF2  kinase which  is a  negative  regulator of  protein  synthesis[@Wang2001].Lastly,  S6Ks  phosphorylate  programmed  cell  death  4  (PDCD4)triggering  its  SCFbetaTrCP-dependent 
degradation [@Carayol2008].PDCD4 is a factor blocking the eIF4A-eIF4G interaction by binding to eIF4A. Binding of PDCD4 to eIF4A leads to  inhibition of  eIF4A  activity  and thus translation of mRNAs that  require  RNA  helicase  activity[@Yang2003]. More recent work has shown the interaction of mTORC1 with LA motif (LAM)-containing factor family La-related protein 1 (LARP1). LARP1 is thought to bind to the 5' mRNA cap of mRNAs with a terminal oloigopyrimidine motif (TOP) via its DM15 domain and represses translation thereof. Binding of LARP1 close the the 5' CAP of TOP mRNAs hinders binding of the EIF4F complex and therefore mRNA translation initiation. mTORC1 physically interacts and phosphorylates LARP1. When phophorylation occurs close to the DM15 domain of LARP1 the inhibitory effect on mRNA translation of TOP mRNAs is abolished. [@Jia2021]

\paragraph{selective regulation of translation by mTOR}



\subsection{The integrated stress response} \label {ISR}

eIF2 delivers  Met-tRNAi to the 40s ribosomal subunit [@Sonenberg2009]. During the integrated stress response (ISR) the alpha subunit of eIF2 is phosphorylated on Ser51 which leads to a global suppression of 5’ cap dependent mRNA translation. Upon eIF2alpha phosphorylation, eIF2alpha directly engages the guanine nucleotide exchange factor eIF2beta. eIF2beta converts the inactive eIF2-GDP to eIF2-GTP, therefore eIF2alpha phosphorylation limits eIF2 recycling of Met-tRNAi to the ribosome [@Sonenberg2009]. Simultaneously, eIF2alpha phosphorylation stimulates selective translation of mRNAs with upstream open reading frames (uORFs) (see section \ref{uORF}). There are multiple kinases, activated depending on the cellular stress, which phosphorylate eIF2alpha. These kinases include Protein kinase R-like endoplasmic reticulum kinase (PERK) which is activated by misfolded peptides in the endoplasmatic reticulum (ER), Heme regulated eIF2alpha kinase (HRI) which is activated during oxidative stress, protein kinase R (PKR) which is activated in response to certain viral infections and GCN2 which is activated when cells are deprived of amino acids[@Kapur2017,@Guan2017,@Taniuchi2016,@Andreev2015]. Therefore, several distinct stress origins converge on the same pathway regulating mRNA translation.

\subsection{Regulation of translation through 5’ and 3’ UTRs} \label{UTR}

\subsection{cis and trans elements in the 5' UTR}

Recruitment of the PIC to the 5’ UTR is followed by scanning until recognition of a start codon. During scanning a process called leaky scanning can occur where the first encountered AUG is not recognized due to sub-optimal sequences flanking the start codon. Leaky scanning is influenced by eukaryotic elongation factors (eEF) 1 and eEF5 where high levels of eEF1 promote leaky scanning and blocking of non-cognate initiation whereas eEF5 works antagonistically to eEF1. Nonetheless, translation is most favorably initiated when an AUG is encountered with the “Kozak” context. Near cognate triplets e.g. NUG can also initiate translation at a much lower frequency as compared to cognate triplets [@Hinnebusch2014]. Structures in the 5’ UTRs can influence translation initiation e.g. stem loops (SL) like the iron responsive element (IRE), which regulates translation of mRNAs involved in iron homeostasis depending on iron availability [@Paraskeva1999] and RNA G-quadruplexes which block scanning [@Wolfe2014].  Therefore, the degree of structure of a 5’ UTR can be an indicator for the degree of translational regulation of an mRNA. In eukaryotes the vast majority of mRNAs have 5’ UTRs with a median length ranging from 53 to 218 nucleotides, where humans have 5’ UTRs with the longest median length. Furthermore, mRNAs with long and structured 5’ UTRs often encode for proteins related to proliferation, survival, and metastasis [@Wolfe2014,Rubio2014].

Next to 5’ UTR structures affecting cap dependent mRNA translation there are also cap independent regulators of mRNA translation such as the viral or cellular internal ribosome entry sites (IRES). The scope of the cellular IRES is still controversial, however cellular IRES are thought recruit the 43s ribosomal subunit towards the 5’ UTR [@Komar2005]. Additionally, eIF3 was suggested to directly bind to stem loop structures of a subset of mRNAs and repress or activate their translation [@Hinnebusch2006,@Lee2015]. RNA modifications in the 5’ UTR could also potentially regulate mRNA translation such as the m6A modification, which can serve as an alternative cap and binds eIF3 to initiate translation or to assist ribosome scanning [@Meyer2015,Zhou2015,@delaParra2018].

Cis elements within the 5’ UTR are also a found to regulate mRNA translation e.g. mRNAs encoding for mitochondrial proteins with an extremely short (~12 nucleotides) 5' UTR which harbors the translation initiator of short 5’ UTR (TISU element). These mRNAs undergo scanning free translation initiation [@Haimov2015]. Another well-studied 5’ UTR element is the 5’ terminal oligo pyrimidine (TOP) element, which consists of a C at the 5’ terminus followed by a stretch of 4-15 pyrimidines [@Yamashita2008].  These TOP mRNAs are fully dependent on the C at the 5’ cap. Translation of TOP mRNAs is tightly linked to mTOR activity and is often considered as mTOR dependent translation, where mTOR almost fully controls their translation activity [@Hamilton2006].

More recent work indicates an effect of mTORC1 on LA motif (LAM)-containing factor family La-related protein 1 (LARP1). In that study the conserved RNA-binding protein of LARP1 interacts with raptor and is phosphorylated by mTORC1. However, the scope of LARP1 mediated effects remain controversial. It has been suggested LARP1 stabilizes or regulates translation of mRNAs with the terminal oligo pyrimidine (TOP) motif in a context dependent manner[@Tcherkezian2014,@Fonseca2015,@Deragon2015).


Nevertheless, TOP mRNAs can contain other regulatory elements in their 5’ UTR alongside the TOP motif, which can override the TOP element translational control in a context dependent manner [@Avni1997].

\subsection{3' UTR}
The importance of the poly-A tail has been observed in several studies where the poly-A tail promotes efficient translation. mRNAs with short poly-A tails generally have a lower translational efficiency, however loss of a poly-A tail does not lead to complete inhibition of protein synthesis [@Munroe1990]. Furthermore, there are many RNA- binding proteins binding to RNA elements in the 3’ UTR such as PABP, CEBP and LARP1 which confer translational control [@Tcherkezian2014,@Fonseca2015,@Matoulkova2012,@Mendez2001]. Cytoplasmic polyadenylation elements (CPEs) are U-rich sites in the 3’ UTR (UUUUUAU) on which RNA binding proteins can bind [@Mendez2001]. Cytoplasmic polyadenylation element binding proteins (CPEBs) are able to recruit either poly(A) polymerases e.g. terminal nucleotidyltransferase 2 (TENT2) or deadenylation enzymes like the CCR4/NOT complex [@Moore2018]. Therefore, the interaction of CPEBs with the recruited enzymes dictates whether a poly(A) tail is shortened or extended. Next to their predominant role in polyadenylation some members of the CPEB family are known to bind to general translation regulation factors, where CPEB4 binds eIF3 [@Hu2014] and CPEB1 regulates mRNA stability by binding to PABPC1 and PABPC1L  [@Seli2005,@Guzeloglu-Kayisli2008]. In Fragile X syndrome, an inheritable intelecutal disability, protein synthesis in the brain is elevated due to loss of FMRP. FMRP has been shown to be involved in ribosome stalling [@Darnell2011] and therefore impacts ribosome elongation. An interesting interplay between FMRP and CEPB1 has been observed, where loss of both CEBP1 and FMRP could restore normal levels of protein synthesis [@Udagawa2013]. Poly-A-binding protein (PABP) is a multifunctional protein contributing to mRNA processing, stability and translation and is thought to bind to the 3’ UTR. Regulation of translation by PABP is achieved through binding to various components of the translational machinery. These components include eIF4B, an initiation factor that aids RNA helicase unwinding function, and eIF4G , eukaryotic Release factor 3 (eRF3) which supports a role of PABP in ribosome recycling, and eIF3 [@Thakor2017,@Cheng2013,@Ivanov2016]. Lastly, the PABP-eIF4G interaction forms the closed loop complex that connects the ends of the mRNA. 
\subsection{RNA binding proteins}

\subsection{Regulation of mRNA translation by tRNAs} \label{tRNA}

is a balance between supply and demand of charged tRNAs of actively translated codons. This concept is also referred to as codon optimality. 

\subsection{Upstream open reading frames} \label{uORF}

An upstream open reading frame is a reading frame that originates in the 5' UTR of an mRNA upstream of which the AUG precedes that of the coding sequence. uORFs are out of frame with the main ORF and when translated lower the expression of the mORF[@Kozak1984]. Ribosome profiling studies indicate that 50% of mammalian mRNAs harbour uORFs including oncogenes and transcripts important in differentiation and cell cycle [@Calvo2009,@Ingolia2011,@Morris1995]. The surrounding context of the uORF is important for its inhibitory through more efficient initiation, where the classical Kozak context is most efficient[@Kozak1986,@Calvo2009]. A prime example where uORFs regulate gene expression is during the ISR. ATF4, a transcription factor for stress response genes, contains two uORFs of which one partially overlaps with the mORF. Under normal conditions mRNA translation is initiated at uORF1 and reinitiation at uORF2 occurs. The close proximity of uORF2 to the mORF causes ribosomes to skip the mORF start thereby inhibiting the translation of the coding sequence. During the ISR, ternary complex formation is reduced causing longer ribosome scanning times (see section \ref{ISR}. This leads to that ribosomes scan past uORF2 and initiate at the mORF[@Pakos-Zebrucka2016]. 

\section{mRNA translation in cancer} \label{translationInCancer}

\section{Translation efficiency}

Each ribosome synthesises a single protein during translation of an mRNA from start to end. Typically an mRNA is loaded with multiple ribosomes (i.e. polysomes). Therefore, the rate at which proteins can be synthesised is dependent on the availability of mRNAs as well as ribosome dynamics at the initiation, elongation and termination of mRNA translation.

\section{Regulatory modes of gene expression}
 EXPAND EVOLUTION - NATURE PAPER COMENSATORY EFFECTS ETC
In transcriptome-wide studies of translation efficiencies the interplay between total mRNA and translated mRNA levels are interrogated. Traditionally it was thought that changes in translation efficiencies lead to altered protein levels. A change in translation efficiency is observed for mRNAs whose polysome-association is altered whereas their total mRNA does not change to a similar magnitude as the polysome-association (I.e. change in translation). An example thereof is TOP mRNA translation under conditions where mTOR is stimulated [@Masvidal2017]. 
\begin{wrapfigure}{l}{0.6\textwidth}
  \includegraphics{./figures/geneModes_MRNA.pdf}
  \caption{Regulatory modes of gene expression - Schematic representation of regulatory modes of translation efficiency in a fold-change scatter plot. Indicated in red are changes in translation (i.e. changes in translated mRNA but not total mRNA), in green changes in mRNA abundance (i.e. congruent changes between total mRNA and translated mRNA) and in blue translational buffering (i.e. changes in total mRNA levels but not translated mRNA levels). TE changes as the TE-score would estimate them are indicated.\label{fig:mRNA}}
\end{wrapfigure}
In recent years, evidence emerged where translation efficiencies of mRNAs can be altered to compensate for changes in total mRNA levels. Within this newly identified mode of regulation of mRNA translation “translational buffering”, mRNA translation is altered such that changes in total mRNA levels do not influence their corresponding protein levels [@Oertlin2019,@McManus2014,@Lorent2019]. Translational buffering is observed to compensate for inter-tissue, inter-species and inter-individual difference [@Artieri2014,@Cenik2015,@Perl2017]. Furthermore, in bacteria translational buffering maintains protein complex stoichiometry as well as protein levels for conserved pathway across species [@Li2014,@Lalanne2018]. Recently translational buffering has been observed under conditions where estrogen receptor alpha (ERalpha) is depleted. ERalpha modulates activity of specific tRNA modification enzymes. These enzymes are needed for the U34 tRNA modification. Loss of ERalpha led to reduced U34 tRNA modification thereby hindering translation of mRNAs requiring such modified tRNAs.  For these mRNAs, even though their total mRNA levels were induced across conditions, their protein levels remained constant [@Lorent2019]. Given these multiple roles of mRNA translation to regulate the proteome it is critical to distinguish them as their underlying mechanisms can have different biological implications.



\section{Expertimental methods to measure mRNA translation} 

\subsection{Polysome profiling} is a technique to measure changes in translational efficiencies of mRNAs between two or more conditions. Polysome profiling allows for separation of polysomes from monosomes, ribosomal subunits and messenger ribonucleoprotein particles (mRNPs). During the assay, ribosomes are immobilized on the mRNAs using translation elongation inhibitors (e.g. cycloheximide). Cytoplasmic RNA extracts are then sedimented on a linear sucrose gradient (5-50%) using ultra centrifugation. 

\begin{wrapfigure}{r}{0.6\textwidth}
    \includegraphics[width=0.9\linewidth]{./figures/polysome_shifts.pdf}
  \caption{Polysome profliles -  (top left) Schematic representation of a polysome profile using linear sucrose gradient fractionation. Indicated in the polysome profiles are the 40S, 60S ribosomal subunits as well as the 80S monosome. H.P. indicates heavy polysome fractions.Between conditions distribution changes for mRNA abundance (top right), translation (bottom left) and translation within high polysome fractions (bottom right) are illustrated. \label{fig:polysome}}
\end{wrapfigure} 
The resulting gradient is fractionated and mRNAs with different number of bound ribosomes can be extracted and analyzed for changes in translational efficiency [@Gandin2014]. An illustration of a polysome profile with peaks for the 40S, 60S subunits and 80S ribosome can be seen in (**Fig \ref{fig:polysome} top left**). Subsequent peaks along the frations indicate the mRNAs with 1 or more bound ribosome. mRNAs are typically normally distributed along the fractions, i.e. a pool of the same mRNA will be associated with 1- n number of ribosomes. Changes in mRNA abundance will lead to an overall increase in the amount of isolated polysome-associated mRNA without a shift of the distribution along the fractions (**Fig \ref{fig:polysome} top right**). This means that the translation efficiency per mRNA remains unchanged. Changes in translational efficiency  can be observed by shifts of polysome association for mRNAs from the light (inefficiently translated) towards the heavy (efficiently translated) polysome fractions or vice versa (**Fig \ref{fig:polysome} bottom left**). Shift within the heavy polysome fractions (i.e. 3 bound ribosome to 7 bound ribosome) can also occur (**Fig \ref{fig:polysome} bottom right**). These shift remain undetected in cases where the distribtion of polysome-associated mRNAs does not sufficiently shift across the fractions and is a limitation of polysome profiling. Quantification of mRNA levels within each fraction can be assessed using Northern blotting or reverse transcription quantitative polymerase chain reaction (RT-qPCR).
For transcriptome wide studies, pooling of efficiently translated mRNAs (mRNAs with >3 bound ribosomes) followed by quantification using either DNA-microarrays or RNA sequencing is common. Pooling of mRNAs as well as collection of multiple fractions makes polysome profiling inconvenient when dealing with large samples sizes or experiments with low amounts of input RNA. Therefore, an optimized sucrose gradient was developed where most efficiently translated mRNAs are collected on a sucrose cushion and thereby can be isolated from one single fraction [@Liang2018]. This optimized gradient allows for application of polysome profiling in small tissue samples where RNA quantity is limiting and reduces labor intensity of the assay.
Polysome-associated mRNA levels are subject to changes in translation efficiency as well as factors contributing to cytosolic mRNA levels. Mechanisms such as transcription (i.e. in the case of mRNA abundance) or mRNA stability can affect cytosolic mRNA levels which impacts the pool of mRNAs that can be associated to polysomes. Therefore, to identify bona fide changes in translation efficiency it is important to collect cytoplasmic mRNA levels in parallel to polysome-associated mRNA to correct for such mechanisms (e.g. transcription or mRNA stability) during downstream analysis [@Gandin2014,@Oertlin2019]. 



\subsection{Ribosome profiling} 
is a technique that enables sequencing of ribosome protected mRNA fragments (RPFs). In the assay ribosomes are immobilized on the mRNAs using, similar to polysome profiling, translation elongations inhibitors (e.g. cyclohexamide) [@Ingolia2010,@Ingolia2016]. One limitation with the use of translation elongation inhibitors is the distortion of ribosome distributions especially at translation initiation sites. These introduced artefacts need to be accounted for in the downstream analysis when assessing ribosome position along the mRNA. Following the translation elongation inhibitor treatment, cells ought to be immediately flash frozen using liquid nitrogen. Alternatively, using only flash freezing has been seen as a robust approach in a wide range of diverse organisms [@Brar2015]. Generation of RPFs is achieved by RNAse treatment breaking the links of RNA between ribosomes leaving single ribosomes with a ~28 nucleotide long RNA fragment within each ribosome. RPFs are then isolated using ultra centrifugation through a sucrose cushion. Co-migration of RNA fragments such as structured non-coding RNAs or large ribonucleoprotein complexes within the sucrose gradient can be a cause of contamination and thereby can provide false readouts of translation. A polyacrylamide gel loaded with RPFs and a reference ladder is used to select RPFs of the right size. Typically, RPFs with lengths of 25 to 30 nucleotides are selected. The RPFs can then be pooled if sample specific barcodes are used. After size selection a pre-adenylated DNA is ligated to the RPFs. This RNA-DNA construct is then used as template for reverse transcription. Through gel-based purification, full-length products of the reverse transcription are selected and circularized. Following circularization, a double stranded DNA library is constructed and PCR amplified. This library is suitable for quantification using RNAseq. In parallel to RPF selection, randomly fragmented total mRNA of the same size is also retrieved. This is achieved by extraction of total mRNA from cell lysate followed by purification via recovery of polyadenylated messages or removal of ribosomal RNA. Fragmentation of total RNA is done using an alkaline fragmentation buffer [@Ingolia2010,@Brar2015].


\subsection{Comparing ribosome and polysome profiling}
Albeit both methods generate count data after quantification with RNAsequencing, there are some key aspects that differ between the techniques. Polysome profiling separates efficiently translated mRNAs from non- efficiently translated mRNAs along a sucrose thereby creating an mRNA based perspective for analyzing changes in translational efficiencies. In contrast, ribosome profiling determines translational efficiencies by counting the number of RPFs of both efficiently and non-efficiently translated mRNAs. Changes in translational efficiencies, e.g. shifts between the polysomal fractions, can be dramatic (I.e. near complete dissociation of ribosomes from an mRNA) or subtle (shifts from 2 to 4 ribosomes) [@Livingstone2015]. Ribosome profiling has been shown to be biased towards identification of dramatic shifts of associated ribosomes to mRNAs, whereas subtle shifts are masked which can lead to false biological conclusions. Polysome profiling is affected by this to a much lesser extent, thereby more robust in identifying such changes [@Masvidal2017]. RPFs in ribosome profiling provide exact nucleotide positions occupied by ribosomes thereby offering single nucleotide resolution. Polysome profiling cannot reveal ribosome locations along the mRNA. However, polysome profiling allows access to full-length mRNAs that includes the UTRs. The single nucleotide resolution of ribosome profiling is necessary in contexts studying local translation events such as ribosomal frame shifts [@Rato2011] or uORF translation [@Andreev2015]. Higher sensitivity in detecting changes in translational efficiencies on a global scale makes polysome profiling more suitable for transcriptome-wide studies [@Gandin2016]. Both methods have their strengths and weaknesses and therefore each method should be considered depending on the underlying biological question of each experiment.



\section{Algorithms for analysis of changes in translation efficiencies}

In polysome-profiling and ribosome profiling translated mRNAs (i.e. polysome-associated mRNAs and RPFs) are separated in parallel from their total mRNA counterpart. Estimating translation efficiencies requires that changes in translated mRNA are corrected for changes in total mRNA to accurately identify the regulatory modes of gene expression (i.e. translation, mRNA abundance and translational buffering)[@Oertlin2019]. Here we will discuss methods that analyse polysome-profiling and ribosome profiling data to estimate changes in translation efficiencies across 2 or more conditions and how these methods identify different regulatory modes of gene expression.

Initially analysis of transcriptome-wide translation studies used an approach called the translation efficiency (TE-score) that uses the following equation: 
\[\varDelta TE = \frac{\frac{P_{c2}}{T_{c2}}} {\frac{P_{c1}}{T_{c1}}}\\\]

This score calculates the ratio of the ratios between polysome-associated mRNA levels (P) divided by total mRNA levels (T) within each condition (i.e. C1 and C2). The TE- score approach has been shown to be prone to spurious correlations [@Larsson2010]. Spurious correlations arise due to that the ratio of polysome-associated mRNA and total mRNA can systematically correlate with total mRNA levels which is not corrected for in this equation and leads to an elevated type-1 error.
\clearpage
\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_TE.pdf}
  \caption{TE scores for regulatory modes of gene expression -  Schematic representation of regulatory modes of translation efficiency in a fold-change scatter plot. Indicated in red are changes in translation efficiency altering protein levels, in green changes in mRNA abundance and in blue changes in translation efficiency leading to translational buffering/offsetting. The shifts for the translation efficiency (TE) score are indicated. \label{fig:TE}}
\end{wrapfigure}
**Figure \ref{fig:TE}** gives an overview of the relationship between a change in  TE) and each regulatory mode of gene expression. Changes in mRNA abundance will lead to a \(\varDelta\)TE close to 0 in log space (i.e. no change) as total mRNA and translated mRNA change with a similar magnitude. However, in the case of both translation and translational buffering, terms in the TE-score equation change leading to a \(\varDelta\)TE (TE < 0 or TE > 0) and thereby identification of both changes in translation and translational buffering simultaneously. Therefore, the TE-score method fails to differentiate between changes in translation and translational buffering which can have drastic consequences for the biological interpretation of the results [@Oertlin2019].

The TE-score approach was challenged by the Analysis of Translation Activity (anota) algorithm which was developed for DNA-microarray data [@Larsson2011]. anota combines analysis of partial variance (APV)[@Schleifer1993] with a random variance model (RVM)[@Wright2003]. RVM estimates gene variance using shared informatio across all genes to increase power for detection of differential expression[@Wright2003]. anota uses a two-step process that firstly assesses the model assumptions for (i) absence of highly influential data points, (ii) common slopes of sample classes, (iii) homoscedasticity of residuals and (iv) normal distribution of per gene residuals. In the second step then performs analysis of changes in translational activity using the following model:

\[log(y_{gi}) = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 
  
here \(\beta_g^{RNA}\) described the relationship to total RNA  \(gth\) gene \(ith\) sample of model matrix \(X\); \(\beta_g^{cond}\) represent the log2 fold change for treatment classes and \(\varepsilon_{gi}\) denotes the residual error. 

\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_anota_Larsson.pdf}
  \caption{anota gene models - Schematic representation of the anota analysis models. Translation mRNA expression is set out against total mRNA expression for each biological replicate and treatment condition. Top left shows the model of a gene that is differentially translated (i.e. change in translated but not total mRNA). The difference in the slope intercepts are used to estimate changes in translation efficiencies between conditions i.e. dTE. Other gene models are shown; change in translation efficiency with varying total mRNA levels (top right); change in mRNA abundance (bottom left) and translational buffering (bottom right).
  \label{fig:anota}}
\end{wrapfigure}

Within anota a common slope for the treatment classes that describes the translated mRNA to total mRNA relationship is calculated. The difference between the slope intercepts is then interpreted as the \(\varDelta\) TE. A simplified view of this model can be seen in (**Figure \ref{fig:anota} top left**). Here expression for translated mRNA and total mRNA are modeled over two sample classes with each 4 replicates. Furthermore, changes in translation efficiencies can also be observed when translated mRNAs shift to a larger extent than the total mRNA levels (**Figure \ref{fig:anota} top right**). Identification of genes in this categorie can be a challenge, especially in highyl variable data set, as they resemble mRNA abundance genes (**Figure \ref{fig:anota} bottom left**). Nevertheless, Using the linear regression analysis anota accurately corrects changes in translated mRNA as can be seen in (**Figure \ref{fig:anota} bottom right**) where a change in total mRNA   but not translated mRNA levels is observed. For this gene the difference in slope intercepts is small and will not be identified as difference in translation as would be the case in the TE-score approach. anota was developed at a time where translational buffering was uncommonly seen in data sets. Naturally, the methods lacks a setting to analyse translational buffering. This was addressed in anota's successor, anota2seq, and will be discussed in **Study 1**.

Advances in experimental methods warrant for appropriate statistical methods to analyse data resulting from them. DNA- microarray was the dominant platform to assess genome-wide changes before the advent of RNA sequencing. In DNA- microarray RNA hybridizes probes on a chip and generate a signal of which the measured intensity is an indicator of expression, whereas in RNA sequencing reads from RNAs are counted. Intensity data from DNA microarray can be normalised and transformed (i.e. log transformation) to fulfill the requirements for application of linear models, whereas RNA sequencing harbours additional characteristics that need to be accounted for. Therefore, algorithms developed for analysis of DNA- microarray are not directly applicable to RNA sequencing data as is the case for the anota algorithm.

RNA sequencing data shows variance that is greater than the mean which is commonly referred to as overdispersion. Count data from RNA sequencing have been initially approached using Poisson distributions which assumes that the variance is equal to the mean[@Lu2005]. Now established RNA sequencing analysis frameworks such as edgeR and DESeq2  use negative binomial distributions in combination with generalized linear models (GLMs) [@Robinson2010, @Love2014]. The negative binomial distribution uses a dispersion parameter to account for overdispersion [@McCarthy2012]. While analysis principles of DESeq2 and edgeR are similar they differ in their normalisation method, dispersion estimation and information sharing across genes. In a simple differential expression analysis between two conditions with one RNA type the GLM model would be as in the following equation: 

\[log(y_{gi}) = \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 

here \(\beta_g^{cond}\ X_i^{cond}\) represent the condition (i.e. control and treatment) log2 fold change for the \(gth\) gene \(ith\) sample of the model matrix X and \(\varepsilon_{gi}\) denotes the residual error. When analysing changes in translation effiencies additional parameter for RNA type (i.e. total mRNA or translated mRNA) and the interaction between the RNA type and condition are added so that:

\[log(y_{gi}) = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \beta_g^{RNA:cond}\ X_i^{interaction} + \varepsilon_gi\] 

In this model the interaction term is interpreted as the change in translation effiencies [@Chothani2019]. Other methods (i.e. Ribodiff[@Zhong2017], Riborex[@Li2017] and deltaTE [@Chothani2019]) borrow this analysis principle of an GLM with an interaction term by often applying this exact model. A noteable difference is that Ribodiff allows dispersion estimation for translated mRNA and total mRNA separetly as variance differences between the RNA types can be expected due to varying experimental protocols [@Zhong2017, @Liang2018]. While the flexibility of GLMs allows for complex study designs involving 2 or more treatment conditions, Riborex and Ribodiff limit the study design to only two conditions. DeltaTE gives their users full flexibility of the DESeq2 GLM model. Xtail is a method developed for ribosome profiling that makes use of DESeq2 for RNAseq count normalisation [@Xiao2016]. Their assessment of differences in translation efficiencies relies on probability matrices for the ratio of translated mRNA over total mRNA within condition and a between condition ratio of these ratios. Babel was the first algorithm designed solely for analysis of differential translation and uses an error-in-varaibles regression analysis [@Olshen2013]. The error-in- variables regression allows accounting for variable total mRNA levels when assessing changes in translation.  Although these methods have distinct approaches to identify changes in translation efficiencies, their principle of analysis is similar to comparing a ratio of ratios. Therefore these methods suffer from similar issues as the TE-score which will be discussed in **Study 1**.



<!--chapter:end:01-intro.Rmd-->

---
output: pdf_document
---

\chapter{Aims of this thesis}


The aims of this thesis are to expand current methodologies for analysis of translation efficiency data and explore the regulation of gene expression in cancer.

In **Study I** we adapted an algorithm for ANalysis Of Translation Activity data (anota) so that it could be applied to next generation sequencing data. Furthermore, we implemented the analysis of translational buffering a recently described regulatory mode of gene expression. The resulting algorithm was named anota2seq.

We then applied the anota2seq algorithm to investigate changes in translation efficiencies in two cancer models:

In **Study II** we unravelled the effects of eIF4A, an RNA helicase, inhibition using a synthetic rocaglate CR-1-31-B (CR-31) in pancreatic ductal adenocarcinoma.

In **Study III** we explored the effects of insulin on gene expression in multiple cell lines.

<!--chapter:end:02-aims.Rmd-->

---
output: pdf_document
---


\chapter{Results and discussion}

<!--chapter:end:03-results_and_discussion.Rmd-->

---
output: pdf_document
---


\chapter{Conclusions}

<!--chapter:end:04-conclusion.Rmd-->

---
output: pdf_document
---

# Acknowledgments {-}

<!--chapter:end:05-acknowledgments.Rmd-->

---
output: pdf_document
---

# References {-}

<!--chapter:end:06-references.Rmd-->

