---
output:
  bookdown::pdf_book:
    includes:
      in_header: preamble.tex
    keep_tex: yes
    pandoc_args: [ "--csl", "_apa.csl" ] # apa citation style
    latex_engine: xelatex
    pdf_document:
    fig_caption: yes
    number_sections: yes
fontsize: 12pt
linestretch: 1.25
toc-depth: 4
secnumdepth: 4
site: "bookdown::bookdown_site"
documentclass: book
classoption: openany # This removes the blank page after ToC
geometry: "left=4cm, right=3cm, top=3cm, bottom=3cm"
header-includes:
- \usepackage{amsmath}
- \usepackage{placeins}
- \usepackage{titlesec}
- \usepackage{amsmath}
- \usepackage{wrapfig}
- \usepackage{caption}
- \captionsetup[figure]{font=scriptsize}
- \usepackage{float}
- \usepackage{subcaption}

subparagraph: yes
bibliography: library/thesis.bib

---

\cleardoublepage
\pagenumbering{gobble}
\pagestyle{fancy}
\fancyhf{}
\renewcommand{\headrulewidth}{0pt}
\fancyfoot[LE,RO]{\thepage}
\renewcommand{\floatpagefraction}{.9} <!-- This specifies that only pages with more than 90% of floats will become a figure only page -->
\setcounter{page}{11} <!-- needs to be adjusted individually depending on size of prebody -->

# Abbreviations {-}

```{r, echo=FALSE, warning=FALSE,message=FALSE,prompt=FALSE,tidy=TRUE}
library(tidyverse)
library(knitr)
library(kableExtra)
tibble(
  Abbreviation = c("RPF","TE","UTR","TOP","uORF","mORF","eIF","PDA","RNA","mRNA","mTOR","DNA","5'","3'","tRNA","PIC","met-tRNAi","TC","GTP","A-site","E-site","P-site","eEF","eRF","ATP","ABCE1","AKT","PI3K","MAPK","AMPK","AMP","4E-BP","PDCD4","S6K","LARP1","PERK","ER","PKR","HRI","dsRNA","GCN2","ODC1","VEGF","ISR","GDP","ELP3","ALKBH8","CTU1/2","URM","mcm^5s^2U","RBP","miRNA","PABP","HuR","HIF-1","mRNP","CHX","RT-qPCR","rRNA","ncRNA","ERalpha","APV","RVM","GLM","NB","CR-31","ROC","AUC","GC/MS","nM","ChIP","INSR","IGF1R","IGF1","sgRNA"),
  
  Term = c("Ribosome protected fragment","Translation efficiency","Untranslated region","Terminal oligopyrimidine","Upstream open reading frame","Main open reading frame","Eukaryotic initiation facotr","Pancreatic ductal adenocarcinoma","Ribonucleic acid", "Messenger RNA","Mammalian/mechatistic target of rapamycin","Deoxyribonucleic acid","5 prime","3 prime","Transfer RNA","Pre-initiation complex","Methionyl-initiatior transfer RNA","Ternary complex","Guanosine triphosphate","Acceptor-site","Exit-site","Peptidyl-site","Eukaryotic elongation factor","Eukaryotic release factor","adenosine triphosphate","ATP binding cassette protein","Protein kinase A","Phosphoinositide 3-kinase","Mitogen-activated protein
kinase","Adenosine-mono-phosphate
kinase","Adenosine-mono-phosphate","4E binding protein","Programmed cell death protein 4","S6 kinase","La ribonucleoprotein domain family member 1","Protein kinase R-like endoplasmatic reticulum kinase","Endoplasmatic reticulum","Protein Kinase R","Heme regulation eIF2alpha kinase","double-stranded RNA","General control nonderepressible 2","Ornithine decarboxylase","Vascular endothelial growth factor","Integrated stress response","Guanosine-di-phosphate","Acetyltransferase elongator","methyltrasnferase TRM9-like domain of alkylation repair homolog 8","cytosolic thiourdylase 1/2","urmylation","5-methoxycarbonyl-methyl-2-thiouridine","RNA binding protein","microRNA","Poly A binding protein","Human antigen R","Hypoxia inducible factor 1","Messenger ribonucleoprotein particle","Cyclohexamide","Teverse transcription quantitative polymerase chain
reaction","Ribosomal RNA","non-coding RNA","Estrogen receptor alpha","Analysis of partial varaince","Random variance model","Generalised linear model","Negative binomial","CR-1-31-B","Receiver operating characteristics","Area under the curve","Gas chromatography
mass spectrometry","Nano molar","Chromatin immunoprecipitation","insulin receptor","IGF1 receptor","insulin-like growth factor 1","single guide RNA")) %>%
  arrange(Term) %>% # i.e. alphabetical order by Term
  kable( booktabs = TRUE, longtable = TRUE) %>%
  kable_styling(latex_options = c("hold_position", "repeat_header")) # linesep command suppresses the standard 5 row chunking
```
\clearpage
\pagenumbering{arabic}
\setcounter{page}{1} <!-- needs to be adjusted individually depending on size of prebody -->

<!-- using inkscape filename.svg --without-gui --export-pdf=filename.pdf to export from Inkscape svg files to pdf -->

<!--chapter:end:index.Rmd-->

---
output: pdf_document
---

\chapter{Introduction}
\section{Gene expression}
\subsection{The central dogma of gene expression}
  \begin{wrapfigure}{r}{.6\textwidth}
  \includegraphics{./figures/geneExprPath_2.pdf}
  \caption{The gene expression pathway - DNA is transcribed into RNA containing a 5' cap (black oval) introns (teal boxes),exons (purple boxes) and a poly(A) tail. RNAs are processed into mRNAs that consisting of a 5' cap, exons and a poly(A) tail. mRNAs can be transported out of the cellular nucleus into the cytoplasm where they can be degraded, stored or translated into proteins depending on cellular demands. Synthesised proteins can be degraded by proteosomes. \label{fig:geneExprPath}}
\end{wrapfigure}
 The whole genetic code of an organisms is stored as deoxyribonucleic acid (DNA) molecules in a double stranded formation as highly condensed chromosomes. Transcription is the process whereby temporary copies of the DNA are generated, called transcripts or ribonucleic acid (RNA), and occurs in the nucleus of eukaryotic cells. RNAs undergo processing by which multiple different variants coming from the same region are produced. A subset of protein-encoding processed RNAs is the so called messenger RNA (mRNA). mRNAs are transported from the nucleus into the cytoplasm where they can be stored, degraded or tranaslated into proteins. Proteins themselves can also be degraded. (**see figure \ref{fig:geneExprPath}**). This flow of genetic information into expressed proteins is commonly referred to as the central dogma in molecular biology [@Crick1970].
\clearpage
\subsection{Contribution to gene expression}
 Proteins are the last product of the gene expression pathway and carry out the vast majority of all cellular functions. While it is apparent that modulation of protein levels will offer information on the changes in gene expression, it cannot completely answer the question as to why the levels change. In a disease context, protein levels alone might offer sufficient insight to explain phenotypic differences. However, information on how these differences arise mechanistically is obscured. Yet, these differences in underlying biological mechanisms can be used as targets in therapeutic strategies.
  
Recently developed developed system biology methods allow investigation of gene expression at multiple levels on a genome-wide scale. Initially, transcriptomics studies were applied to study gene expression with the assumption that mRNA expression is the main determinant for protein levels and therefore could be used as a proxy for them. However, this view was challenged by several landmark studies that observed a poor mRNA to protein correlation and indicated a larger role of post-transcriptional regulation in gene expression than previously assumed [@Lu2005; @Vogel2012; @deSousaAbreu2009; @Schwanhausser2011; @Silva2016].
  
 The debate regarding which step of the gene expression pathway contributes most to the composition of the proteome is ongoing, nevertheless an understanding has been reached that the context is a major determinant. At steady state, mRNA levels seem to explain protein abundance best, however in perturbed systems the contribution of transcript abundance appears to have less impact relative to post-transcriptional steps in regulation of gene expression [@Liu2016].  For example in a study that stimuleted bone marrow-derived dendritic cells with LPS, protein levels were dependent on cellular transcript levels [@Jovanovic2015]. In contrast a study investigating cells under endoplasmatic reticulum stress observed extensive modulation at the protein levels, whereas mRNA transcript abundance was only mildly affected [@Cheng2016].
  
 While the contribution of different steps of the gene expression is dependent on many different factors, e.g. cellular state or treatments, mRNA translation (synthesis of proteins) is an essential process in determining composition of the proteome. Furthermore, dysregulation of mRNA translation has been observed in multiple diseases, ranging from neurological disorders to cancer which warrants for a comprehensive understanding of this process [@Kapur2018; @Ruggero2013; @Lee2021; @Graff2009]. This thesis will focus on the role of mRNA translation in the context of cancer.
\newline
\section{mRNA translation}
\subsection{Structure of an mRNA}
 After transcription, RNA transcripts are processed into mRNAs. mRNAs contain a protein coding region which is flanked by untranslated regions (5' and 3' UTRs). UTRs contain post-transcriptional regulatory elements that effect may affecy localisation, stability and translation of the mRNA [@Loya2008; @Leppek2018; @Mignone2002] (*see \ref{fig:UTRFeat} and sections \ref{translation},\ref{regmRNA} for details*). The 5' has a 7-methyl-guanylate (m7G) cap that is important for translation initiation, while the 3' end has a poly-A tail protecting the mRNA against degradation [@Grifo1983;@Wilusz2001]. Multiple different mRNAs (isoforms or transcript variants) from the same genomic region exist. These variants may arise due to alternave transcription start site or a process called alternative splicing which alters the exon composition (e.g. coding region) of an mRNA. These variants can co-exist at the same time and have distinct properties and can perform distinct functions [@JolyAnne-Laure2018]. Furthermore, splicing resulting in alternative UTRs can introduce differences in translation for mRNAs encoding the same protein [@Jewer2020;@Floor2016].
\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/UTRFeatures.pdf}
  \caption{ An mRNA consists of a coding sequence, 5' and 3' untranslated regions flanking the coding sequence, a 5' cap and a poly-A tail. Located within the 5' and 3' untranslated region are post-transcriptional regulatory elements that can influence gene expression. Furthermore, the codon composition of the coding sequence influences translation elongation rates (see section \ref{tRNA}). uORF; upstream open reading frame; IRES, internatl ribosome entry site; CPE, cytoplasmic polyadenylation site; AAUAAA, polyadenylation signal.
 \label{fig:UTRFeat}}
\end{wrapfigure}
\clearpage
\subsection{Translation of an mRNA} \label{translation}
  For the vast majority of protein coding mRNAs, eukaryotic mRNA translation occurs in the cytoplasm, however a small subset of not nuclear-encoded mRNAs is translated in the mitochondria [@DSouza2018]. mRNA translation is a process that includes initiation, elongation, termination and ribosome recycling and is an essential process for protein homeostasis (**see Figure \ref{fig:doodlemRNASteps}**). 
  
  
  During the initiation phase of 5' cap dependent mRNA translation a ribosome is recruited to the mRNA following by scanning toward the 3' end for a start codon to begin synthesis of the polypeptide chain by incorporating amino acids [@Hinnebusch2014]. Furthermore, 5' cap independent translation initiation can also occur, e.g. via the internal ribosome entry site in the 5' UTRs (*see Fig \ref{fig:UTRFeat}). Amino acids are transferred to the ribosome by specialised RNAs called transfer RNAs (tRNA) that can recognise the genetic code in the mRNA. The availability of tRNAs as well as their modification can influence the rate of elongation. The order by which amino acids are incorporated is dictated by the order of the codons of the open reading frame (ORF). There are more possible codon combinations in comparison to amino acids. This allows that amino acids are encoded by multiple codons, for example lysine is encoded by AAA and AAG. Once the ribosome encounters a stop codon, translation will terminate and the polypeptide chain will be released. The ribosome then disassociates from the mRNA and can be recycled to engage translation of the same or another mRNA. The following sections describe these processes in more detail. 

\begin{wrapfigure}{o}{1\textwidth}
  \includegraphics{./figures/doodleTranslation.pdf}
  \caption{mRNA translation initiation, elongation, termination and ribosome recycling steps - The ribosome binds to the mRNA and initiates scanning for a start codon. The elongation phase incorporates amino acids into a polypeptide chain (i.e. the protein product). Once the end of the coding sequence is detected, by recognition of stop codons, the ribosome terminates translation and releases the polypeptide chain. The ribosome can then be recycled to participate in the translation of another mRNA or reinitiate. \label{fig:doodlemRNASteps}}
\end{wrapfigure}
\clearpage

\subsection{Initiation} \label{initiation}

In eukaryotes 5' cap dependent mRNA translation consists of multiple stages. First, a ternary complex (TC) consisting of GTP bound eukaryotic initiation factor (eIF) 2 and methionine-initiator tRNA (met-tRNAi) is formed. This is followed by formation of the 43S pre initiation complex (PIC) consisting of a 40S ribosome subunit, eIF1, eIF1A, eIF3, eIF5 and the TC [@Asano2000]. mRNAs then undergoe "activation" by which the 5' cap proximal structure is unwound by eIF4F with eIF4B in an adenosine-tri-phosphate (ATP) dependent manner. eIF4F is the 5' cap binding complex consisting of;  eIF4E, 5' cap binding protein; eIF4A, an RNA helicase and; eIF4G, a scaffold protein [@Grifo1983]. Poly a binding protein (PABP) binds in the 3' UTR and causes circularisation of the mRNA to improve its stability and aid in recruitment of translation initiation factors [@Ivanov2016]. The 43S complex then binds to this unwound region and starts scanning in the 5' to 3' direction. Recognition of the translation initiation codon (AUG) induced the 48S initiation complex formation. Here displacement of eIF1 occurs which allows eIF5 to hydrolyse eIF2-bound GTP. The 60S subunit then joins the 48S initiation complex which causes the release of eIF2-GDP and other initiation factors (eIF1, eIF3, eIF4B, eIF4F and eIF5) and is mediated by eIF5B. After subunit joining the ribosome starts the elongation process (**Figure \ref{initiation}**) [@Asano2000; @Hinnebusch2006; @Jackson2010].

\subsection{Elongation}

 The 80S ribosome contains three sites important for decoding an mRNA: the aminoacyl (A), peptidyl (P) and exit (E) sites. During elongation in eukaryotes, aminoacytelated tRNAs are delivered to the A-site in a TC with eukaryotic elongation factor 1A (eEF1A) and guanosine triphosphate (GTP). If the tRNA is cognate to the condon in the A-site of the ribosome, eEF1A is hydrolysed causing its release and accommodating the tRNA in the A-site. This is followed by a peptidyl transferase reaction forming the peptide bond by the transfer of the nascent polypeptide from peptidyl tRNA in the P-site to the amino group of the A-site aminoacyl-tRNA (aa-tRNA). Then a translocation steps occurs through eEF2-GTP hydrolysis which translaocates deacetylated tRNA into the E-site and peptidyl tRNA to the P-site. The deacytelated tRNA is then released from the ribosome [@Dever2012]. This process is repeated until a stop codon (UAA, UGA or UAG) enters the A site of the ribosome .

\begin{figure}[ht]
\includegraphics{./figures/initiation.jpg} 
  \caption{Pathway of eukaryotic translation initiation via ribosomal scanning.
  \label{fig:initiation}}
\end{figure}

\clearpage

\subsection{Termination and recycling} 
\begin{wrapfigure}{r}{.4\textwidth}
  \includegraphics{./figures/polysome.pdf}
  \caption{Electromicrongraph of ribosomes extracted from different positions along a sucrose gradient used for polysome fractionation (A-C). For details on polysome fractionation see section \ref{exptMethod}. Reprinted with permission. DR. T. STAEHELIN et. al. Nature.1963 Aug 31;199:865-70.doi: 10.1038/199865a0. Copyright © 1963, Nature Publishing Group.
  Schematic of the TAPES method - A ribosome (A) can initiate with rate \(\alpha\) on an mRNA with a coding sequence with codons i= 1 ... L. The elongation rate at a specific codon is defined by \(\omega_i\) and \(\beta\) determines the termination rate once a stop codon is encountered. TASEP has been constantly modified, e.g. to allow for correction of initiation or elongation when the following codon is already occupied. Reprinted with permission. Juraj Szavits-Nossan and Martin R. Evans. 10.1103/PhysRevE.101.062404. ©2020 American Physical Society. 
 \label{fig:polysomes}}
\end{wrapfigure}
 mRNA translation termination is  facilitated  by  two  eukaryotic release  factors (eRF), eRF1  and  eRF3-GTP [@Stansfield1995;@Alkalaeva2006]. The  eRF2:eRF3-GTP complex binds to the A-site of the ribosome upon recognition of a stop codon. This causes an hydrolysis event resulting in a conformational change and release of the polypeptide chain. eRF1 and the ATP binding cassette protein (ABCE1) together promote  the  splitting  of  the  60S  and  40S  subunits after which they can be recycled [@Pisarev2010;@Hellen2018]. 

\subsection{Translation efficiency}
 Each ribosome synthesises a single protein during translation of an mRNA assuming it is not prematurely terminated. It has been known since the '60s that translation of an mRNA occurs via multiple bound ribosomes (polysomes) simultaneaously (**see figure \ref{fig:polysomes}A-C**) [@Warner1962;@Staehelin1963]. Therefore the translation efficiency of an mRNA depends on the number of ribosomes it is associated with. While all steps of translation can affect the translation efficiency of an mRNA, it is thought to most commonly be regulated at the initiation step [@Richter2015;@Dever2012;@Jackson2010]. 

 Experimental methods, e.g. polysome profiling, are used to study translation efficiencies. Furthermore, there are efforts to model translation kinetics such as totally asymmetric simple exclusion process (TASEP) to obtain a better understanding of ribosome movements along the mRNA [@MacDonald1968;@Maniloff1969] (**See figure \ref{fig:polysomes}D**). A model that assessed ribosome traffic under steady state and high initiation rates indicates traffic is not only dependent on the densities of ribosomes but also codon specific elongation rates and their distribution along the mRNA [@Szavits-Nossan2020a]. The next section will go further into detail how initiation rates and elongation can be regulated.
\newline
\section{Regulation of mRNA translation} \label{regmRNA}

 mRNA translation is the most energy consuming process in the cell. In a study using concanavalin A stimulated rat thymocytes, it was estimated that translation accounts for  ~20% of the cellular energy consumption [@Buttgereit1995]. The high energy consumption of mRNA translation  and its central role in the gene expression pathway requires it to be tightly regulated, especially in cancer where dysregulated metabolism is common [@Hanahan2011].

 Regulation of translation can be exerted at a global level (i.e. regulation of a large set of mRNA simultaneously) by e.g. perturbations of major signalling pathways impinging on mRNA translation. Furthermore, a strong feature of translational control is that it can affect distinct mRNA  populations selectively. This form of translational regulation acts on characteristics of mRNAs, e.g. through cis elements in the UTRs, RNA binding proteins (RBP) (**see figure \ref{fig:UTRFeat}**) or regulation of initiation factors  [@Leppek2018]. Below we will discuss multiple mechanisms that regulate mRNA translation globally or selectively.

\subsection{mTOR} \label{mTOR}
 mTOR is  a  conserved  Ser/Thr  kinase  and  is  found  in  two  structurally  and  functionally  distinct  complexes,  mTORC1  and mTORC2 [@Saxton2017;@Pearce2007]. In a growth promoting environment mTOR switches cell metabolism to increase protein synthesis, lipids and nucleotides, while suppressing catabolic pathways, e.g. autophagy. mTORC2 promotes survival via signalling through protein kinase A (AKT), anabolic metabolism, and cytoskeleton regulation [@Sarbassov2005,@Zoncu2011].

 mTOR activity is modulated via hormone and growth factor signalling (i.e. insulin and insulin-like growth factor; IGF1). This signalling is predominantly mediated through the phosphoinositide 3-kinase (PI3K) / AKT pathway. However, the Ras/mitogen-activated protein kinase (MAPK) pathway 

 
 
 Both these pathways are involved in oncogenic signalling and under investigation as therapeutic targets in cancer [@Hilger2002;@Yang2019]. In several cancers (e.g. breast, lung, prostate and colon) the gene encoding the p110\(\alpha\) subunit of PI3K is frequently mutated or amplified [@Levine2005;@Samuels2004;@Lee2005]. 
 \begin{wrapfigure}{r}{0.6\textwidth}
  \includegraphics{./figures/mTORsignal.jpg}
  \caption{Schematic representation of mTOR signaling to the translational machinery. Philippe P. Roux, and Ivan Topisirovic Mol. Cell. Biol. 2018; doi:10.1128/MCB.00070-18. Reprinted with permission. Copyright © 2018, American Society for Microbiology
 \label{fig:mtorsignal}}
\end{wrapfigure} The E545K mutation  leads to a reduced inhibition of p85 at the catalytic p110\(\alpha\) subunit of PI3K [@Jiang2018]. In **study 3** we investigate oncogenic signalling via the PI3K pathway activated by insulin and the role of mTOR in mediating the resulting effects on gene expression in the MCF7 breast cancer cell line that harbours the E545K mutation [@Schneck2013]. Furthermore, hyperactivity of PI3K/AKT and Ras/MAPK signalling pathways have been reported in multiple cancers and been linked anti- cancer therapy resistance [@Populo2012;@Tan2013;@Salaroglio2019]. mTOR, as a downstream actor of these pathways integrating their signals, has therefore become a focus of anti cancer therapy by either targeting mTORC1 or using dual inhibitors for PI3K and mTOR [@Bhat2015].

 mTOR also fulfills a central role in metabolic signalling where it integrates amino acid availability, glucose and cellular oxygen levels to form an appropriate response. For example, increased amino acid availability induces relocalisation of mTOR into proximity of Rag GTPases leading to its activation [@Sancak2008]. Furthermore, Glucose deprivation and oxygen availability regulate mTOR activity via adenosine-mono-phosphate kinase (AMPK) signalling. AMPK is activated when a shift in the cellular AMP to adenosine-tri-phophate (ATP) ratio is sensed [@Sanders2007;@Kimball2006]. While hypoxia (i.e. deprivation of oxygen) inhibits protein synthesis in normal cells, in breast cancer, protein synthesis was not inhibited during hypoxia which is attributed to uncontrolled mTOR signalling [@Connolly2006]. 

 Given the central of mTOR governing proliferation, growth and metabolism, which are often deregulated in cancer, it is vital to comprehensively understand mTORC1 signalling herein [@Hanahan2011].

\subsubsection{Global regulation of translation via mTOR}
 mTOR regulates mRNA translation mainly by regulating availability of eIF4E through phosphorylation of 4E binding proteins (4E-BPs)[@Gingras1999]. eIF4E is required for mRNA 5' cap binding and formation of the eIF4F complex. Therefore, inhibition of mTOR leads to a down regulation of cap dependent mRNA translation.
Other downstream targets of mTOR such as S6k has been shown to regulate components of the translation machinery, e.g. by activation of programmed cell death protein 4(PDCD4). PDCD4 competes for binding to eIF4G with eIF4A [@Dorrello2006;@Goke2002].  

\subsubsection{Selective or "mTOR sensitive" regulation of translation} 
 Selective or "mTOR sensitive" regulation of translation acts on transcripts with a terminal oligo pyrimidine motif (TOP mRNAs). This TOP motif consists of a C followed by a stretch of 4-15 pyrimidines directly after the 5' cap. TOP mRNAs show near complete dissociation from ribosomes under conditions when mTOR is inhibited and are  enriched for genes encoding for parts of the translation machinery [@Meyuhas2000; @Thoreen2012; @Yamashita2008]. Recent works indicate the importance of La ribonucleoprotein domain family member 1 (LARP1) in regulation of TOP mRNAs with contradictory findings [@Hopkins2016;@Jia2021;@Fonseca2015;@Maraia2017]. A panel of researches were asked to evaluate these findings which led to the establishment of a model for translational regulation via LARP1 [@Berman2021]. Herein, LARP1 is thought to bind to the 5' mRNA cap of TOP mRNAs via its DM15 domain and represses translation by obstructing eIF4E binding in instances where mTOR activity is abolished. In environments where mTOR is active phosphorylation of the DM15 occurs by mTOR causing it to release the 5' cap, wile the la domain of LARP1 remains bound to the mRNA thereby stabilizing it to facilitate translation [@Berman2021]. Other instances of selective translation are for mRNAs that lack the TOP motif, but show sensitivity to mTOR activity due to its role in eIF4F complex formation. 

\subsection{selective regulation through members of the eIF4F complex}
 As mentioned before, the eIF4F complex consists of eIF4E, eIF4A and eIF4G and is required for translation of all cap dependent mRNAs. However, characteristics of 5' UTRs introduce variation to how well mRNAs can be translated. Graff et al. proposed a model of strong and weak mRNAs. Herein, strong mRNAs are widely expressed and represent the majority of cellular mRNAs and are characterised by short unstructured 5' UTRs, e.g. \(\beta\)-actin. On the other hand, weak mRNAs show long and structured 5' UTRs and encode for potent growth and survival factors, e.g. c-Myc, ornithine decarboxylase (ODC1) and vascular endothelial growth factor (VEGF). Translation of "strong" mRNAs would remain effective in conditions where eIF4F complex availability would be limited. However, "weak" mRNAs show sensitivity eIF4F availability which is dependent on eIF4E expression [@Graff2008]. Elevated eIF4E expression is common in cancer and drives malignancy [@DeBenedetti2004]. 

 Studies investigated the effects of eIF4A inhibition on mRNA translation and the underlying characteristics of eIF4A dependent mRNAs. Therein, mRNAs that were senstitive to silvesterol treatment showed characteristics of "weak" mRNAs, i.e. long and structured 5' UTRs [@Wolfe2014;@Rubio2014;@Waldron2018]. In addition one third of eIF4A dependent mRNAs were encoded by genes with multiple 5' UTR variants, while for eIF4A independent mRNAs this was < 1% [@Rubio2014]. Translation of mRNAs with different 5' UTR variants has recently been implicated to drive cancer cell plasticity towards more "stem cell like" phenotypes during hypoxia [@Jewer2020].

 Recently, a more nuanced picture for translation of mRNAs sensitive to inhibition of components of the eIF4F complex has been shown [@Gandin2016]. Therein, a comparison of treatments with an mTOR inhibitor (i.e. limiting eIF4F complex assembly) and silvesterol were evaluated leading to identification of mTOR-eIF4E and mTOR-eIF4A senstive mRNAs. Indeed, as shown before mTOR-eIF4A senstive mRNAs were characterised by long and structured 5' UTRs and endoced for pro surival proteins, however their translation was also dependent on eIF4E availability. mTOR-eIF4E sensitive mRNAs were characterised by short 5' UTRs and did not show dependence on eIF4A. Furthermore, these mRNAs encode for proteins related to metabolism.
 
\subsection{The integrated stress response} 
 The integrated stress response is a pathway which is activated through kinases responding to various stress signals. These kinases include Protein kinase R-like endoplasmic reticulum kinase (PERK) which is activated by misfolded peptides in the endoplasmatic reticulum (ER), Heme regulated eIF2alpha kinase (HRI) which is activated during oxidative stress, protein kinase R (PKR) which is activated in response to certain viral infections by binding to double-stranded RNA (dsRNA) and general control nonderepressible 2 (GCN2) which is activated when cells are deprived of amino acids[@Kapur2017;@Guan2017;@Taniuchi2016;@Andreev2015;@Lemaire2005]. During the integrated stress response the alpha subunit of eIF2 is phosphorylated. Phosphorylated eIF2\(\alpha\) directly engages the guanine nucleotide exchange factor eIF2B and prevents conversion of inactive eIF2-GDP to  active eIF2-GTP needed for met-tRNAi incorporation in the TC, therefore inhibiting translation by reducing TC availability [@Sonenberg2009] (**see also figure \ref{fig:initiation}**).

\subsubsection{Global and selective regulation of translation via the ISR}

 Global and selective regulation of translation via the ISR is, similar to mTOR signalling, achieved at a global and selective level. Phosphorylation of eIF2 alpha limits ternary complex availability, therefore met-tRNAi recruitment to the ribosome is limited which results in a reduction of translation initiation. While global translation is reduced upon ISR, translation of a selective subset of mRNA with upstream open reading frames (uORFs) is increased. A uORF is a reading frame that originates in the 5' UTR of an mRNA upstream of the coding sequence ORF. uORFs can be out of frame with the main ORF (mORF) and, when translated, lower the expression of the mORF [@Kozak1984]. Activating transcription factor 4 (ATF4), a transcription factor for stress response genes, contains multiple uORFs of which one partially overlaps with the mORF. Under normal conditions ATF4 mRNA translation is initiated at uORF1 and reinitiation at uORF2 occurs. The overlap of uORF2 with the mORF causes ribosomes to synthesise protein from uORF2 thereby inhibiting the translation of the coding sequence. Limitation of TC availability during ISR causes longer ribosome scanning times leading to that ribosomes scan past uORF2 and initiate at the mORF (i.e. delayed initiation) [@Pakos-Zebrucka2016].

 Ribosome profiling studies indicate that 50% of mammalian mRNAs harbour uORFs . mRNAs containing uORFs include oncogenes and transcripts important in differentiation and cell cycle [@Calvo2009;@Ingolia2011]. Apart from delayed initiation, uORF translation can also be due to "leaky" scanning or reinitiation downstream of a uORF stop codon. The surrounding sequence of the uORF is important for initiation of translation. An AUG in the classical Kozak context (i.e. RNN**AUG**G) is most efficient for translation initiation due to better recognition by the met-tRNAi [@Kozak1986;@Calvo2009]. 
 
 However, uORFs that do not contain AUG have been described as well. For example in Fli-1, two conserved uORFs have been described where one contains an AUG and the other GUG [@Sarrazin2000]. Translation of either uORF results in different proteins isoforms endoded in the same mRNA. Furthermore, other elements (e.g. structures) in proximity of the uORF can also influence its translation [@Ruan1994]. 
\clearpage
\subsection{Regulation of mRNA translation by tRNAs} \label{tRNA}
\begin{wrapfigure}{o}{0.6\textwidth}
  \includegraphics{./figures/tRNA.pdf}
  \caption{ (A) Schematic representation of the tRNA cloverleaf structure with indicated anti-codon and amino acid acceptor sites.  (B) Schematic representation of the proliferation and differentiation mRNAs dependent on distinct tRNA subsets. (C) Schematic representation of the U34 wobble position and the catalytic enzymes involved in this modification that is implied in tumor progression.
 \label{fig:tRNA}}
\end{wrapfigure}
 As touched upon earlier, tRNAs are an essential part of the translation machinery that carry the amino acids to the ribosome. tRNAs consist of a 76-90 long nucleotide sequence set into a "cloverleaf" structure forming several loops [@Sharp1985] (**see figure \ref{tRNA}**). The acceptor stem binds the amino acid carried by the tRNA, while the anti-codon loop binds to the mRNA within the ribosome via classical watson-crick pairing [@Watson1953]. Multiple codons can encode for the same amino acid (synonymous codons), however the availability of the tRNAs for different codons may vary which can influence elongation rates. 

 This supply ( i.e. tRNA availability) and demand (i.e. codon in expressed mRNA transcripts) relationship has been found to vary across different cellular states, e.g. proliferation and differentiation. Gingold et. al. observed two differen tRNA subsets; one induced under proliferation that is otherwise repressed and a subset with similar regulation under differentiation of which the supply matches the codon demand of the transcriptome [@Gingold2014]. This model has been disputed and it was proposed that the observed differences would be attributed to GC content in the mRNA [@Rudolph2016]. Nevertheless, abbarent tRNA expression and codon usage have been reported in cancer [@Zhang2018]. Furthermore, a comprehensive study including small RNAseq (i.e. for identification of tRNAs) and protein samples across 17 tissues obtained from the The Cancer Genome Atlas (TCGA) reported a tRNA signature stratified by Ki67 (a proliferation marker) staining with implications for patient survival [@Hernandez-Alias2020]. Therefore, while a consensus on proliferation specific tRNA subsets might not have been reached, emerging evidence implicates a role thereof in cancer [@Hernandez-Alias2020;@Gingold2014;@Zhang2018]. More specifically, increased expression of \(tRNA_{CCG}^{Arg}\) and \(tRNA_{UUC}^{Glu}\) has been observed in breast cancer cell lines and proposed to drive metastasis [@Goodarzi2016]. 

 Others report of a role of tRNAs in cancer attributed to tRNA modifications, specifically at the U34 anti- codon (or wobble) position which is highly conserved [@Rapino2017;@ElYacoubi2012]. Wobbling was proposed by Francis Crick and refers to the ability of non-watson-crick base pairing of tRNA anti codons [@Crick1966]. This enables a smaller set of tRNAs (41-55 in eukaryotes) to encode for the 64 possible codon combinations [@Goodenbour2006]. In mammals, the U34 modification catalytic cascade involves the acetyltransferase Elongator (ELP3), the methyltransferase TRM9-like domain of Alkylation repair homolog 8 (ALKBH8), and the urmylation (URM) pathway, that includes the cytosolic thiouridylase homolog 1 and 2 (CTU1/CTU2) [@Kalhor2003;@Karlsborn2014]. These enzymes ultimately modify the U34 modification into 5-methoxycarbonyl-methyl-2-thiouridine (\(mcm^5s^2U\)) which ensures cognate codon recognition. This modification is thought to occur in a small subset of tRNAs, namely  \(tRNA^{UUU}\), \(tRNA^{UUC}\), \(tRNA^{UUG}\), \(tRNA^{UCC}\), and \(tRNA^{UCU}\). 

 Loss of the ability to modify U34 has been shown to reduce translation elongation rates with varying effects on protein expression [@Nedialkova2015;@Deng2015;@Zinshteyn2013]. While in some cases U34 dependent signalling led to ribosome stalling resulting in protein aggregates and increased stress [@Nedialkova2015;@Zinshteyn2013], others reported a subtle dowregulation of proteins encoded by mRNAs requiring U34-modified tRNAs [@Deng2015]. U34 modification dependent tRNAs have been shown to play a role in cancer. For example, ELP3 is important in tumor initiation in the intestine and promotes breast cancer invasion as well as progression to metastasis [@Ladang2015;@Delaunay2016]. Furthermore, loss of U34 modification in a prostate cancer model lead to an adaptive transcriptional response for mRNAs whose translation was dependent on the U34 modification [@Lorent2019].

\subsection{RNA binding proteins and trans-acting factors}
 The UTRs of an mRNA contain sequence elements to which small RNA and RNA binding proteins (RBPs) bind and exert translational regulation. For instance, microRNAs (miRNA), a small class of non coding RNA. The precise role and mechanisms in regulation of translation of miRNAs is still under active investigation [@Oliveto2017]. However, they can directly bind to other RNAs and silence them accomplished through translational repression or destabilisation [Jonas2015]. Regulation of gene expression by miRNAs has been observed in cancer where miRNAs have been implicated to promote tumorgenesis or act as tumor suppressors [@Sampson2007;@Muniyappa2009;@Tian2010;@Nagpal2015].

 RBPs are a class of proteins involved in many regulatory steps of gene expression and account for ~7.5% of the protein coding genes. The poly-A-binding-protein (PABP) is a RBP involved in mRNA translation initiation. PABP is thought to form a closed loop complex of the 3' end to the 5' by interacting with eIF4G. This closed loop is proposed to promote translation and prevent mRNA decay [@Afonina2014;@Amrani2008] (**see also figure \ref{fig:initiation}**). Another important RBP implicated in regulation of translation is Human antigen R (HuR). HuR preferentially binds to AU-rich sequences in the 3' UTR and acts as a stabilizing agent and is involved in RNA-processing [@Levine1993;@Baou2011;@Fan1998;@Peng1998]. Furthermore, a study in adenocarcinoma implicates that HuR inhibits translation of proapoptotic factors [@Winkler2014]. Other studies in breast, colon and lung cancer observed correlation between HuR and malignancy. Among its targets are hypoxia inducible factor 1 (HIF-1), VEGF and c-Myc which are known drivers for tumor progression [@Denkert2004;@LopezdeSilanes2003;@LopezdeSilanes2005].
\clearpage
\section{Expertimental methods to measure mRNA translation} \label{exptMethod}
\begin{wrapfigure}{r}{0.6\textwidth}
    \includegraphics{./figures/polyRibo.pdf}
  \caption{Polysome profiling and ribosome profiling workflows. In polysome profiling a fraction from whole cytoplasmic RNA is loaded onto a sucrose gradient on which they get separated by sedimentation using ultra centrifugation. Fractions corresponding to efficiently translated mRNAs are collected and can be quantified with for example RNA sequencing (left). During ribosome profiling a fraction from the whole cytoplasmic RNA is exposed to a digestion agent which disturbs the RNA. The ribosomes will protect fragments thereby creating ribosome protected fragments. These fragments are then isolated and can be sequenced.  \label{fig:polyRibo}}
\end{wrapfigure}
 Methods that measure mRNA translation try to capture the number of ribosomes an mRNA is associated with on the principle that this is directly correlated with their translation efficiency. This is under the assumption that elongation rates across pools of mRNAs are similar which is supported by ribosome kinetic studies [@Ingolia2011]. Two methods are predominantly used  for measuring mRNA translation, or changes in translation efficiencies across conditions, namely polysome profiling and ribosome profiling (**see figure \ref{fig:polyRibo}**). The methods will be explained below in further detail.


\subsection{Polysome profiling} 
 Polysome profiling is a technique to measure changes in translational efficiencies of mRNAs between two or more conditions. Polysome profiling allows for separation of polysomes from monosomes, ribosomal subunits and messenger ribonucleoprotein particles (mRNPs). During the assay, ribosomes are immobilized on the mRNAs using translation elongation inhibitors (e.g. cycloheximide (CHX)). A portion of cytoplasmic RNA extracts then sediment on a linear sucrose gradient (5-50%) using ultra centrifugation. The resulting gradient is fractionated and mRNAs with different number of bound ribosomes can be extracted and analyzed for changes in translational efficiency [@Gandin2014]. Typically fractions belonging to mRNAs with more than 3 bound ribosomes are pooled. A 3-ribosome cut off has been chosen as it is thought to capture most biologically relevant changes in translation efficiency [@Gandin2016a].
\begin{wrapfigure}{r}{0.6\textwidth}
    \includegraphics[width=0.9\linewidth]{./figures/polysome_shifts.pdf}
  \caption{Polysome profliles -  (top left) Schematic representation of a polysome profile using linear sucrose gradient fractionation. Indicated in the polysome profiles are the 40S, 60S ribosomal subunits as well as the 80S monosome. H.P. indicates heavy polysome fractions.Between conditions (i.e. black an pink lines) distribution changes for mRNA abundance (grey and green; top right), translation (grey and red; bottom left) and translation within high polysome fractions (grey and red; bottom right) are illustrated. \label{fig:polysome}}
\end{wrapfigure}
An illustration of a polysome profile with peaks for the 40S, 60S subunits and 80S ribosome can be seen in (**Fig \ref{fig:polysome} top left**). Subsequent peaks along the frations indicate the mRNAs with 2 or more bound ribosomes. mRNAs are typically normally distributed along the fractions, i.e. in a pool of the same mRNA many will be associated with different numbers of ribosomes [@Gandin2016a]. Changes in mRNA abundance will lead to an overall increase in the amount of isolated polysome-associated mRNA without a shift of the distribution along the fractions (**Fig \ref{fig:polysome} top right**). This means that the translation efficiency per mRNA remains unchanged. Changes in translational efficiency  can be observed by shifts of polysome association for mRNAs from the light (inefficiently translated) towards the heavy (efficiently translated) polysome fractions or vice versa (**Fig \ref{fig:polysome} bottom left**). Shift within the heavy polysome fractions (i.e. with a mean distribution around 4 bound ribosomes to 7 bound ribosomes) can also occur (**Fig \ref{fig:polysome} bottom right**). Quantification of mRNA levels within each fraction can be assessed using Northern blotting or reverse transcription quantitative polymerase chain reaction (RT-qPCR). For transcriptome wide studies quantification is done using either DNA-microarrays or RNA sequencing. 

 Pooling of mRNAs as well as collection of multiple fractions makes polysome profiling inconvenient when dealing with large samples sizes or experiments with low amounts of input RNA. Therefore, an optimized sucrose gradient was developed where mRNAs bound to >3 ribosomes are collected on a sucrose cushion and thereby can be isolated from one single fraction [@Liang2018]. This optimized gradient allows for application of polysome profiling in small tissue samples where RNA quantity is limiting and reduces labor intensity of the assay. 

 Polysome-associated mRNA levels are subject to changes in translation efficiency as well as factors contributing to cytosolic mRNA levels that impact to pool of mRNAs that can associate with ribosomes e.g. transcription and mRNA stability.  Therefore, to identify true changes in translation efficiency it is important to collect cytoplasmic mRNA and polysome-associated mRNA from the same sample in parallel to correct for such mechanisms during downstream analysis. 

\subsection{Ribosome profiling} \label{riboseq}
 Ribosome profiling is a technique that enables sequencing of ribosome protected mRNA fragments (RPFs) [@Steitz1969]. In the assay ribosomes are immobilized on the mRNAs using, similar to polysome profiling, translation elongations inhibitors (e.g. CHX) [@Ingolia2009;@Ingolia2016]. One limitation with the use of translation elongation inhibitors is the distortion of ribosome distributions especially at translation initiation sites [@Gerashchenko2014;@Hussmann2015]. For example, treatment with cyclohexamide can affect identification of uORFs in ribosome profiling experiments [@Spealman2018]. These introduced artefacts need to be accounted for in the downstream analysis when assessing ribosome position along the mRNA. 

 RPFs are obtained by RNAse treatment that degrades the links of RNA between ribosomes leaving single ribosomes with a ~28 nucleotide long RNA fragment within each ribosome. However, as ribosomal RNAs (rRNA) is also degraded during this process they represent a big fraction herein. The RPFs are then isolated using ultra centrifugation through a sucrose cushion. During this step other RNA fragments such as rRNAs,  non-coding RNAs (ncRNAs) or large ribonucleoprotein complexes can co-migrate and contaminate the sample. Typically RPFs with a size ranging from 25-30 nucleotides are selected for quantification. However, among these sizes could be RNA protected by RBPs that have no ribosome associated to it. Furthermore, ribosomes undergoing conformational changes have been shown to protect fragments corresponding to a length of 21nt only when translation elongation is blocked by e.g. CHX [@Lareau2014]. Therefore, size selection can distort the estimation of translation efficiency from ribosome profiling data [@Andreev2017]. From the resulting RPFs libraries can be constructed and quantified using RNA sequencing. During library construction additional biases due to enzyme sequence preferences can be introduced that can lead wrong estimations of codon positions within the ribosome [@Artieri2014a].

 Initially, fragmented total mRNA using alkaline hydrolysis of the same size were retrieved in paralell to RPFs. This was achieved by extraction of total mRNA from cell lysate followed by purification via recovery of polyadenylated messages or removal of ribosomal RNA [@Ingolia2009;@Brar2015]. The random fragmentation of total mRNA has been shown to underlie experimental bias. Therefore, now unfragmented total mRNA is sequenced in parallel.

\subsection{Comparing ribosome and polysome profiling}
 Albeit both methods generate count data after quantification with RNA sequencing, there are some key aspects that differ between the techniques. Polysome profiling separates efficiently translated mRNAs from non- efficiently translated mRNAs along a sucrose gradient thereby creating an mRNA based perspective for analysing changes in translational efficiencies. In contrast, ribosome profiling determines translational efficiencies by counting the number of RPFs of both efficiently and non-efficiently translated mRNAs. This can have implications for identifaction of transcript variants. For example  If a ribosome would not protect a fragment spanning a variant the information would be lost during ribosome profiling [@Floor2016]. Whereas in polysome profiling the quantification is based on the whole mRNA. This gives polysome profiling the advantage in cases of transcript variants with different 5' UTR lengths.

 Changes in translational efficiencies, e.g. shifts in ribosome association, can be dramatic (i.e. near complete dissociation of ribosomes from an mRNA) or subtle (shifts from e.g. 2 to 4 ribosomes) and could be due to different properties of the mRNAs [@Gandin2016]. When these two changes in translation efficiencies are present in parallel, ribosome profiling is biased towards identification of dramatic shifts and masks the subtle ones, leading to false biological conclusions [@Gandin2016a]. This has been attrituted to the indirect estimation of translation efficiencies from counting RPFs for ribosome profiling, which is highly dependent on the abundance of mRNAs. In polysome profiling  this effect is much less pronounced as changes in translation efficiency are directly estimated from the mRNAs associated with heavy polysomes [@Masvidal2017]. Higher sensitivity in detecting changes in translational efficiencies on a global scale makes polysome profiling more suitable for transcriptome-wide studies [@Gandin2016a;@Masvidal2017].

 An advantage of ribosome profiling is that it provides exact nucleotide positions occupied by ribosomes. This offers information at a single nucleotide level where the ribosome sits. Polysome profiling cannot reveal ribosome locations along the mRNA. The single nucleotide resolution of ribosome profiling is necessary in contexts studying local translation events such as ribosomal frame shifts [@Rato2011] or uORF translation [@Andreev2015]. 

 Both methods have their strengths and weaknesses and therefore each method should be considered depending on the underlying biological question.
\newline
\section{Modes for regulation of gene expression through mRNA translation} \label{modes}

From transcriptome-wide assessments of translation using ribosome or polysome profiling expression levels are obtained for both cytoplasmic and polysome-associated mRNAs (or RPFs). For simplicity, from now on, these RNA types will be referred to as total mRNA (i.e. cytoplasmic mRNA) and translated mRNA (i.e. polysome-associated mRNA or RPFs). The estimation of expression levels for both translated mRNA and total mRNA allows for interrogation at two steps of the gene expression pathway and their interaction. The interplay of total mRNA with translated mRNA can give valuable insights about the underlying mechanisms that govern gene expression in the studied system.
\begin{wrapfigure}{l}{0.6\textwidth}
  \includegraphics{./figures/geneModes_MRNA.pdf}
  \caption{Regulatory modes of gene expression - Schematic representation of regulatory modes of translation efficiency in a fold-change scatter plot. Indicated in red are changes in translation (i.e. changes in translated mRNA but not total mRNA), in green changes in mRNA abundance (i.e. congruent changes between total mRNA and translated mRNA) and in blue translational buffering (i.e. changes in total mRNA levels but not translated mRNA levels). TE changes as the TE-score would estimate them are indicated.\label{fig:modes}}
\end{wrapfigure}
 When comparing perturbed systems to their corresponding control state we typically observe three "modes" in which translated mRNA and total mRNA distinctly interact that impact gene expression (**See figure \ref{modes}**). We refer to these modes as "translation", "mRNA abundance" and "translational buffering". 

\subsection{Translation} 
 A change in translation occurs when translated mRNA levels either increase or decrease, however corresponding total mRNA levels remain constant or change to a lesser extent resulting in a change of their translation efficiency. A prominent example of this mode can be observed for TOP mRNAs. These mRNAs, under conditions when mTOR is inhibited, show a near complete dissassociation from ribosomes [@Gandin2016]. mRNAs under the translation mode are expected to reshape the proteome (**See figure \ref{modes}**). 

\subsection{mRNA Abundance} 
 Another mode that impacts the proteome is a change in mRNA abundance, here the translated mRNA level changes to a similar magnitude as the total mRNA level. For these mRNAs the translation efficiency is unaltered, as the change in total mRNA levels explains the change in translated mRNA levels. Nevertheless, as there are more (or less) mRNAs being translated an effect the proteome is expected. The underlying biological implication for this mode mRNAs is often related to transcription or mRNA stability (**See figure \ref{modes}**).

\subsection{Translational buffering} \label{modeBuffering}

The third mode for regulation of gene expression is termed translational buffering. Under this mode a change in total mRNA levels is observed, whereas polysome-associated mRNA levels remain constant between conditions. Translational buffering also reflects a change in translation efficiency as the proportion of mRNAs to associated with ribosomes is altered. In contrast to changes in translation and mRNA abundance, translational buffering has been shown to maintain protein homeostasis rather than reshape the proteome [@Lorent2019;@Lalanne2018;@McManus2014].

Currently the literature supports multiple forms of translational buffering. Translational buffering can compensate for differences in mRNA levels due to e.g. differences in gene dosages, so that protein levels remain similar [@Dassi2015;@Lalanne2018]. Furthermore, rather than compensating it can "offset" translation to temporarily alter the mRNA:protein ratio in response to a stimulus [@Lorent2019]. 

Translational buffering in its compensation form has been observed at steady state levels. A study comparing co evolution of transcription and translation across seven different organs and mammals showed overall greater divergence of transcription as compared to translation [@Wang2020]. Similar compensation at the level of translation has been observed between individuals, tissues and prokaryotes [@Cenik2015;@Perl2017;@Lalanne2018;@Dassi2015;@Artieri2014].

Compensation via translational buffering can also enforce equilibration of pathway or protein complex stoichometry [@Lalanne2018;@Li2014]. An example of this was observed in  evolutionary distant bacetria, i.e. B. subtilis and E. coli. In B. subtilis translation related factors rpsP and rplS are located in different operons, whereas in E. coli they lie within an operon together with rimM and trmD. While B. subtilis can fine tune transcription at the different operons, in E. coli these will be transcribed together. However, rimM and trmD are only needed in low protein abundance, whereas rpsP and rplS are required in high abundance. E. coli compensates the transcriptional input at the translational level and thus equilibriates for requirements in pathway stoichometry [@Lalanne2018].
 
As mentioned above a different form of translational buffering can be observed  in perturbed systems that offset changes in transcription at the level of translation temporarily. For example, translational offsetting was observed in prostate cancer cells where a transcriptional program was induced under estrogen receptor \(\alpha\) (ER\(\alpha\)) depletion that was showed no increase in polysome-associated mRNA. mRNAs whose transcription was translationally offset required the tRNA u34 modification. ER\(\alpha\) has been shown to regulate the expression of the catalytic enzymes required for the u34 modification [@Lorent2019]. Thus, depletion of ER\(\alpha\) led to that tRNAs could not be properly modified at the U34 position and therefore the translation efficiency of mRNAs requiring the modification was reduced despite increased total mRNA levels. 
\newline
\section{Algorithms for analysis of changes in translation efficiencies}\label{algorithm}

 Given multiple roles of mRNA translation to regulate the proteome via translation, mRNA abundance and buffering it is critical to distinguish them as their underlying mechanisms can have different biological implications. In this section we will discuss methods that analyse polysome-profiling and ribosome profiling data to estimate changes in translation efficiencies across 2 or more conditions and how these methods identify different modes of gene expression.

 Initially analysis of transcriptome-wide translation studies used an approach called the translation efficiency score (TE-score) that uses the following equation: 
\[\varDelta TE = \frac{\frac{P_{c2}}{T_{c2}}} {\frac{P_{c1}}{T_{c1}}}\\\]

This score is calculated the ratio of the ratios between polysome-associated mRNA levels (P) divided by total mRNA levels (T) within each condition (i.e. C1 and C2). The TE- score approach has been shown to be prone to spurious correlations [@Larsson2010]. Spurious correlations arise due to that the ratio of polysome-associated mRNA and total mRNA can systematically correlate with total mRNA levels which is not corrected for in this equation and leads to an elevated type-1 error.

 **Figure \ref{fig:TE}** gives an overview of the relationship between a change in  TE-score and each gene expression mode (**see also figure \ref{fig:modes}**). Changes in mRNA abundance will lead to a \(\varDelta\)TE close to 0 in log space (i.e. no change in translation efficiency) as total mRNA and translated mRNA change with a similar magnitude. However, in the case of both translation and translational buffering, the nominator and denominator in the TE-score equation change leading to a \(\varDelta\)TE (TE < 0 or TE > 0) and thereby identification of both changes in translation and translational buffering simultaneously. Therefore, the TE-score method fails to differentiate between changes in translation and translational buffering which can have drastic consequences for the biological interpretation of the results [@Oertlin2019] (**see also section \ref{modes}**).
\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_TE.pdf}
  \caption{TE scores for regulatory modes of gene expression -  Schematic representation of regulatory modes of translation efficiency in a fold-change scatter plot. Indicated in red are changes in translation efficiency altering protein levels, in green changes in mRNA abundance and in blue changes in translation efficiency leading to translational buffering/offsetting. The shifts for the translation efficiency (TE) score are indicated. \label{fig:TE}}
\end{wrapfigure}
 The TE-score approach was questioned when proposing the Analysis of Translation Activity (anota) algorithm which was developed for DNA-microarray data [@Larsson2010]. anota combines analysis of partial variance (APV) [@Schleifer1993] with a random variance model (RVM) [@Wright2003]. RVM estimates gene variance using shared information across all genes to increase power for detection of differential expression [@Wright2003]. anota uses a two-step process that firstly assesses the model assumptions for (i) absence of highly influential data points, (ii) samples classes sharing a common slope, (iii) homoscedasticity of residuals and (iv) normal distribution of per gene residuals. In the second step anota performs analysis of changes in translational activity using the following model:

\[y_{gi} = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 
  
here \(y_{gi}\) is the polysome associated mRNA expression, \(\beta_g^{RNA}\) describes the relationship to total RNA for the \(gth\) gene of the \(ith\) sample column of model matrix \(X\); \(\beta_g^{cond}\) represent the difference in intercept between treatment classes and \(\varepsilon_{gi}\) denotes the residual error. 

\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_anota_Larsson.pdf}
  \caption{anota gene models - Schematic representation of the anota analysis models. Translation mRNA expression is set out against total mRNA expression for each biological replicate and treatment condition. Top left shows the model of a gene that is deferentially translated (i.e. change in translated but not total mRNA). The difference in the slope intercepts are used to estimate changes in translation efficiencies between conditions i.e. dTE. Other gene models are shown; change in translation efficiency with varying total mRNA levels (top right); change in mRNA abundance (bottom left) and translational buffering (bottom right).
  \label{fig:anota}}
\end{wrapfigure}

 Within anota a common slope for the treatment classes that describes the translated mRNA to total mRNA relationship is calculated. The difference between the slope intercepts is then interpreted as the change in translation efficiency. A simplified view of this model can be seen in (**Figure \ref{fig:anota} top left**). Here expression for translated mRNA and total mRNA are modeled over two sample classes with each 4 replicates. Furthermore, changes in translation efficiencies can also be observed when translated mRNAs shift to a larger extent than the total mRNA levels (**Figure \ref{fig:anota} top right**). Identification of genes in this category can be a challenge, especially in highly variable data sets, as they resemble mRNA abundance genes (**Figure \ref{fig:anota} bottom left**). Nevertheless, using the linear regression analysis anota accurately corrects changes in translated mRNA as can be seen in (**Figure \ref{fig:anota} bottom right**) where a change in total mRNA   but not translated mRNA levels is observed. For this gene the difference in slope intercepts between sample classes is small and will not be identified as differentially translated as would be the case in the TE-score approach. anota was developed at a time where translational buffering was not considered in data sets. Naturally, the methods lacks a setting to analyse translational buffering. This was addressed in anota's successor, anota2seq, and will be discussed in **Study 1**.

 Advances in experimental methods warrant for appropriate statistical methods to analyse data resulting from them. DNA-microarray was the dominant platform to assess transcriptome-wide changes before the advent of RNA sequencing. DNA-microarrays measure intensity after hybridisation events is an indicator of expression, whereas in RNA sequencing reads of constructed libraries are counted. Intensity data from DNA-microarray can be normalised and transformed (i.e. log transformation) to fulfill the requirements for application of linear models, whereas RNA sequencing harbours additional characteristics that need to be accounted for. Therefore, algorithms developed for analysis of DNA- microarray are not directly applicable to RNA sequencing data as is the case for the anota algorithm.

 RNA sequencing data shows variance that is greater than the mean which is commonly referred to as overdispersion. Count data from RNA sequencing have been initially approached using Poisson distributions which assumes that the variance is equal to the mean [@Lu2005]. Now established RNA sequencing analysis frameworks such as edgeR and DESeq2  use negative binomial distributions in combination with generalised linear models (GLMs) [@Robinson2010; @Love2014]. The negative binomial distribution uses a dispersion parameter to account for differences in the mean-variance relationship across the expression range [@McCarthy2012]. While analysis principles of DESeq2 and edgeR are similar they differ in their normalisation method, dispersion estimation and information sharing across genes. In a simple differential expression analysis between two conditions with one RNA type the GLM model would be as in the following equation: 

\[log(y_{gi}) = \beta_g^{cond}\ X_i^{cond} + \varepsilon_{gi}\] 

here \(\beta_g^{cond}\ X_i^{cond}\) represent the condition (i.e. control and treatment) log2 fold change for the \(gth\) gene \(ith\) sample column of the model matrix X and \(\varepsilon_{gi}\) denotes the residual error. When analysing changes in translation effiencies additional parameter for RNA type (i.e. total mRNA or translated mRNA) and the interaction between the RNA type and condition are added so that:

\[log(y_{gi}) = \beta_g^{RNA}\ X_i^{RNA}+ \beta_g^{cond}\ X_i^{cond} + \beta_g^{RNA:cond}\ X_i^{interaction} + \varepsilon_gi\] 

 In this model the interaction term is interpreted as the change in translation effiencies [@Chothani2019]. Other methods (i.e. Ribodiff [@Zhong2017], Riborex [@Li2017] and deltaTE [@Chothani2019]) borrow this analysis principle of an GLM with an interaction term by applying this exact model. A notable difference is that Ribodiff allows dispersion estimation for translated mRNA and total mRNA separately as variance differences between the RNA types can be expected due to varying experimental protocols [@Zhong2017; @Liang2018]. While the flexibility of GLMs allows for complex study designs involving 2 or more treatment conditions, Riborex and Ribodiff limit the study design to only two conditions. DeltaTE gives their users full flexibility of the DESeq2 GLM model. Xtail is a method developed for ribosome profiling that makes use of DESeq2 for RNAseq count normalisation [@Xiao2016]. Their assessment of differences in translation efficiencies relies on probability matrices for the ratio of translated mRNA over total mRNA within condition and a between condition ratio of these ratios. Babel was the first algorithm designed solely for analysis of differential translation and uses an error-in-variables regression analysis [@Olshen2013]. The error-in- variables regression allows accounting for variable total mRNA levels when assessing changes in translation.  Although these methods have distinct approaches to identify changes in translation efficiencies, their principle of analysis is similar to comparing a ratio of ratios (see TE-score equation above). Therefore these methods suffer from similar issues as the TE-score which will be discussed in **Study 1**.



<!--chapter:end:01-intro.Rmd-->

---
output: pdf_document
---

\chapter{Aims of this thesis}
The aims of this thesis are to expand current methodologies for analysis of translation efficiency data and explore the regulation of gene expression in cancer.

In **Study I** we adapted the ANalysis Of Translation Activity data (anota) algorithm so that it could be applied to next generation sequencing data. Furthermore, we implemented the analysis of translational buffering a recently described regulatory mode of gene expression. The resulting algorithm was named anota2seq.

We then applied the anota2seq algorithm to investigate changes in translation efficiencies in two cancer models:

In **Study II** we unraveled the effects of eIF4A, an RNA helicase, inhibition using a synthetic rocaglate CR-1-31-B (CR-31) in pancreatic ductal adenocarcinoma.

In **Study III** we explored the effects of insulin on gene expression in multiple cell lines.

<!--chapter:end:02-aims.Rmd-->

---
output:
  pdf_document: default
---


\chapter{Results and discussion}

\section{Study 1 - Generally applicable transcriptome-wide analysis of translation using anota2seq}

 Initially changes in translation efficiencies were estimated using the TE-score approach as outlined in section \ref{algorithm}. However, this method was being shown to be prone to spurious correlations leading to elevated false positive identification that can result in false biological conclusions [@Larsson2010]. Spurious correlations, when using the TE-score, can be attributed the inadequate correction for changes in total mRNA levels when estimating translation efficiencies [@Larsson2010;@Larsson2011]. The Analysis of Translation Activity (anota) algorithm facilitates analysis of translational efficiencies that are corrected for changes in total mRNA levels [@Larsson2011]. 

 anota was developed for analysis of transcriptome-wide analysis for data quantified by DNA- microarrays [@Larsson2010]. However, advances in experimental methodologies lead to the development in RNA sequencing. RNA sequencing and DNA microarray data have distinct characteristics that need to be accounted for before analysis (**see section \ref{algorithm}**). Therefore, while the statistical framework of anota had been shown as an adequate approach for analysis of translational efficiencies for data from DNA microarrays it was not directly applicable to RNA sequencing data. The characteristic of RNA sequencing data, that makes applying anota directly not possible, is the mean variance relationship. This encompasses that the counts for lower expressed genes show higher variability than counts for higher expressed genes even after log transformation. Efforts have been made to make RNA sequencing data more "DNA- microarray like" so that algorithms developed for intensity based microarray data can be applied to count based RNA sequencing data [@Law2014;@Love2014]. Anota2seq, the algorithm developed in this study, allows for transformation and normalisation of RNA sequencing data so that the anota statistical frame work can be applied for analysis of count data. 

 Another feature of anota2seq is that it allows for statistical analysis of translational buffering. The need for the analysis of translational buffering, or the uncoupling of transcription from translation, has been noted before anota2seq's development by comparing 20 translatomes and transcriptomes with different underlying  stimuli in mammalian cells [@Tebaldi2012]. The same authors proposed a framework, called tRanslatome, that combines several methodologies for analysis of differential transcription and translation efficiencies, including anota, for a comprehensive analysis of transcription and translation as well as their underlying mechanisms [@Tebaldi2014].

\begin{wrapfigure}{o}{0.5\textwidth}
  \includegraphics{./figures/geneModes_anota2seq.pdf}
  \caption{anota2seq gene model for analysis of translational buffering /offsetting - Total mRNA expression is set out against translated mRNA expression for each biological replicate and treatment condition. The model shows total mRNA changes that are independent of translated mRNA changes which is classified as translational buffering. It is important to distinguish between the gene modes as their regulation could be due to different underlying biological mechanisms (see section \ref{modes}).
  \label{fig:anota2seq}}
\end{wrapfigure}

 Nevertheless, commonly observed in polysome and ribosome profiling data sets are three gene expression modes, translation, translational buffering and mRNA abundance. While anota can be used to identify genes among the translation and mRNA abundance mode, analysis for translational buffering was not implemented therein (*See Figure \ref{fig:anota}*). Therefore, one would need to rely on the integration of several methods to efficiently analyse transcriptome-wide studies of translation efficiences. Anota2seq addresses this issue by changing the analysis model as described in section \ref{algorithm} to analyse changes in total mRNA levels corrected for changes in translated mRNA levels (i.e. translational buffering, **see figure \ref{fig:anota2seq}**). 

 Appliance of anota2seq has successfully identified translational buffering to which biological mechanisms could be linked, e.g. as mentioned earlier translationally bufferring under ER\(\alpha\) depletion in prostate cancer (**see section \ref{modeBuffering}**) [@Lorent2019]. Furthermore, in **study 2** translational buffering can be observed as a compensating mechanisms in "healthy" cells upon treatment with an eIF4A inhibitor and in **study 3** we identify  mTOR dependent translational buffering for mRNAs with certain 3' UTR characteristics.

 The aim of this study was to compare anota2seq's performance to other established algorithms (i.e. DESeq2, RiboDiff, babel, TE-score and Xtail) for analysis of translation efficiencies, specifically their ability to distinguish the three prominent modes of gene expression. To achieve this we used simulated data. While it is arguable to what extent conclusion drawn from simulated data can be extended towards empirical data it allows for a controlled environment where true positive changes are known in advance. Furthermore, the mean-variance relationship in the simulated data is based on a real polysome profiling data set to increase confidence that drawn conclusions are also applicable to empirical data [@Guan2017]. Furthermore, during testing of our simulation we used an additional data set to estimate parameters from to generate data sets. Using these simulated data to compare the performance of the above mentioned algorithms showed almost identical results. 

 The simulated data consisted of four replicates for translated mRNA and total mRNA with a "control" and a "treatment" condition. Furthermore, the data sets contained a combination of the following gene sets:

*"Unchanged"*: For this simulation category we sampled reads from the same NB distribution for both the control and treatment conditions in both the translated and total mRNA. This category represents genes that would be unaffected by e.g. a stimulus of between cellular states.

*"mRNA abundance"*: For this category the control condition for both the translated mRNA and total mRNA were sampled from the same NB distribution. The NB distribution for **both translated mRNA and total mRNA** of the treatment condition was altered so that values would be drawn corresponding to a fold change (negative or positive) ranging between 1.5 to 3.0. The directionality of the fold changes (i.e. up or down regulation) was the same for translated mRNA and total mRNA.

*"translation"*: For this category the control condition for both the translated mRNA and total mRNA were samled from the same NB distribution. The NB distribution for **translated mRNA only** of the treatment condition was altered so that values would be drawn corresponding to a fold change (negative or positive) ranging between 1.5 to 3.0.

*"buffering"*: For this category the control condition for both the translated mRNA and total mRNA were sampled from the same NB distribution. The NB distribution for **total mRNA only** of the treatment condition was altered so that values would be drawn corresponding to a fold change (negative or positive) ranging between 1.5 to 3.0.

 As a first step we tested whether the methods could properly control for type-1 errors (i.e. false positive identification). For this we simulated a data set with genes belonging only to the "unchanged" category. This revealed that babel, but to an even greater extent Xtail, were unable to control their type-1 error as these methods assigned low p-values and FDRs when no real changes were present. DESeq2 was marginally affected by this also. This indicated a limited applicability of Xtail and babel for statistical analysis of translatomes.

 From the comparative analysis  of the analysis for changes in translation efficiencies affecting protein levels we concluded that anota2seq outperforms all other methods. This was assessed by comparing the area under the curve from Receiver operating characteristics (ROC) and precision recall curves. The ROC curves showed a, albeit slightly, better performance for detecting changes in translation. However, the precision recall was much higher for anota2seq which can be accredited to that the analysis principle of the other methods is based on identifying changes regardless of whether the change is in the translated mRNA or total mRNA (*as explained in section \ref{algorithm}*). Nevertheless, when comparing the performance using simulated data in the absence of genes belonging to the "buffering" category anota2seq still showed superior performance. 

 Next to the a prior knowledge of the introduced changes in the simulation data, it also allowed us to modify parameters to investigate the robustness of the methods to increased variance, overall sequencing depth and differing sequencing depth between samples. Here, all methods showed robustness against variance and sequencing depth differences between samples as long as a minimum of 5 million counts per sample was reached. 

 A shortcoming in the simulation study is that we did not assess the effects of systematic batch effects. Batch effects can be introduced e.g. during experimental design and there are many methods that try to correct for these [@Johnson2007;@Leek2014;@Zhang2020]. Other ways to correct for batch effects is their inclusion in the analysis model, which can be supplied to the analysis model in DESeq2, edgeR as well as anota2seq. Indeed analysis of a dataset with prominent batch effects showed that batch effects can dampen the efficiency of the anota2seq algorithm to identify changes but can be effectively corrected for in the algorithm.

 In this study we developed an analysis algorithm for efficient transcriptome-wide analysis of translation efficiencies applicable to DNA-microarrays and RNA seq. Furthermore, anota2seq has been successfully applied to broaden the knowledge around mRNA translation in various different contexts [@Lorent2019;@Chan2019;@Hipolito2019;@Chaparro2020]. 
\newline
\section{Study 2 - eIF4A supports an oncogenic translation program in pancreatic ductal adenocarcinoma}

 Pancreatic cancer is considered a lethal malignancy and has limited treatment options. A study on predictions for european cancer mortality rates for 2021 concluded that health efforts should focus on pancreatic cancer. While other cancers (e.g. ovary, breast and stomach) showed a decline in mortality rates, no major overall decline was observed for pancreatic cancer in the period of 1970-2021 [@Carioli2021].

 Pancreatic ductal adenocarcinoma (PDA) accounts for over 90% of exocrine pancreatic cancer, whereas non ductal pancreatic cancers e.g. acinar cell carcinomas are uncommon [@Feldmann2007;@Jun2016]. It is estimated the 60-70% of the PDAs arise in the head of the pancreas [@Luchini2016]. So far treatment options are mostly limited to surgical removal, which often is impossible due to the anatomical location of the pancreas head. 

 With the increasing understanding of tumor heterogeneity, i.e. tumors with similar tissue origin are diverse at a molecular level,  anti cancer therapy improved [@Biankin2011]. In breast cancer stratification by histological, molecular and gene expression features identified several breast cancer subtypes for which different treatment options exist, e.g. \(ER^+\) breast cancer subtypes respond to endocrine therapy, whereas \(ER^-\) do not [@Andre2006;@Parker2009]. While breast cancer treatment strategies benefit from a rather well established understanding of the molecular subtypes, in pancreatic cancer transcriptomic based subtyping is still ongoing [@Bailey2016;@Puleo2018;@Collisson2011;@Moffitt2015;@Collisson2019]. Therefore, it is warranted to extend current knowledge around pancreatic cancer to advance therapeutic strategies in this lethal disease.

 While intricacies of molecular subtypes are still being investigated, research has shown that  oncogenic mutations in KRAS as well as inactivation of tumor suppressor TP53 are commonly shared among PDAs [@Jones2008]. Furthermore, PDAs have been shown to be dependent on increased protein synthesis mediated by KRAS [@Chio2016]. This indicates an important role of mRNA translation in PDA.  

 The aim of *study 2* was to investigate the therapeutic effects of targeting eIF4A in a three dimensional PDA organoid cell culture with mutations in the \(Kras^{LSL-G12D}\), \(Trp53^{LSL-R172H}\) and Pdx1-cre alleles that has been shown to recapitulate PDA tumor progression [@Boj2015]. The inhibition of eIF4A was done using a synthetic rocaglate, CR-31. Rocaglates have been shown to inhibit eIF4A helicase activity and displayed anti tumor activity [@Cencic2009]. 

 We first wanted to establish the therapeutic validity of targeting eIF4A in PDA. *In vitro* experiments comparing treated PDA organoids (KP) to their normal (N) counter parts revealed heightened sensitivity of KP to CR-31 treatment. OP-puromycin incorporation showed reduced protein synthesis in KP, whereas N were affected to a lesser extent. Furthermore, similar effects were found *in vivo* for PDA tumours. Here CR-31 reduced protein synthesis (assessed by SUnSET assay), tumor growth (assessed by ultra sound imaging) and increased survival. The effect on protein synthesis was not due to inhibition of oncogenic signalling pathways which was evaluated western blot assessing the phosphorylation of e.g. AKT, mTOR and 4E-BP1. From these findings we concluded that there is therapeutic validity in targeting eIF4A in PDA that acts downstream of oncogenic signalling pathways.

 Using polysome profiling we then sought to decipher the mechanisms explaining the increased sensitivity to CR-31 in KP organoids.  First we investigated the differences in gene expression between untreated KP and N organoids. Analysis of changes in translation efficiencies using anota2seq revealed massive modulation at both transcription and translation indicative of underlying differences in e.g. genomic stability and enhanced oncogenic signalling impinging on protein synthesis reported in PDA [@Boj2015]. Consistent with the *in vitro* OP-puromycin incorporation and *in vivo* SUNsET experiments, CR-31 had only little effect on translation in N organoids, whereas in KP organoids ample changes in translation were detected.

 We then compared the mRNAs affected by CR-31 in KP to the acquired translational profile of untreated KP and N organoids. In order to achieve this we visualised the mRNAs affected by CR-31 in KP onto the data of the untreated KP vs N organoids comparison. This revealed that the acquired translational program of KP organoids is reversed when treated with CR-31. 

 Performing a similar analysis, we visualised mRNAs sensitive to CR-31 in KP onto the data of the CR-31 treated N organoids. mRNAs affected by CR-31 in KP were translationally offset in CR-31 treated N organoids. Translational buffering, as an adaptive response to treatment, has been shown to maintain protein homeostasis by inducing a transcriptional response increasing the pool of mRNAs that can be translated [@Lorent2019]. The ability for N organoids in increase transcription for mRNAs affected by CR-31, whereas KP cannot, could partially explain as to why protein synthesis is not reduced to a similar extent in N as in KP.

 We then assessed 5' UTR characteristics of the translationally regulated mRNAs upon CR-31 treatment in KP organoids. It was reported that eIF4A-senstive mRNAs showed overall and more structured 5' UTRs (e.g. containing G-quadruplexes) [@Rubio2014;@Wolfe2014;@Gandin2016a]. Furthermore, a mechanisms by which rocaglates would clamp eIF4A to mRNAs with [A,G] repeats in their 5 UTR was described [@Iwasaki2016]. However, mRNAs sensitive to CR-31 treatment herein showed overall shorter 5' UTRs that were more structured when corrected for their length  without enrichment for 4G-quadruplexes or  [A,G] repeats. From the polysome profiling and UTR analysis we concluded that eIF4A supports an oncogenic translation program in PDA cells for mRNAs with shorter but structured 5' UTRs.

 Shorter 5' UTRs have been shown to be sensitive to eIF4E expression and encode for metabolic functions (see section \ref{regmRNA}). When we compared an eIF4E overexpression signature in the KP vs N and CR-31 treated KP we observed that in KP organoids translationally regulated mRNAs under eIF4E overexpression were also translationally activated. This observation is consistent with reports of 4E-BP1 loss in pancreatic cancer [@Martineau2014]. eIF4A inhibition in tumors resistant to mTOR inhibiton by loss of 4E-BP1 has been shown to circumvent this resistance [@Muller2019]. Indeed, CR-31 treatment in KP reversed the translational profile for eIF4E sensitive mRNAs. Therefore, while rocaglates have been shown to directly target eIF4A [@Cencic2009], eIF4F complex formation could be disrupted by reduced eIF4A availability to an extent that eIF4E dependent translation is affected in PDA organoids herein. 

 When further inspecting the regulated gene sets in treated and untreated KP compared to their normal counterparts we could see an enrichment in metabolic pathways, e.g. Oxidative phosphorylation. This pathway was upregulated at the polysome associated RNA levels in untreated KP compared to N, whereas in KP CR-31 treatment reversed the translational profile of this pathway. Indeed, when we measured mitochondrial respiration (by oxygen consumption rates) we found that CR-31 treatment in KP disrupted this process, whereas N organoids where affected to a lesser extent. 

 A way to counter loss of energy production through oxidative phosphorylation is to increase activity of other anaerobic metabolic pathways, i.e. glycolysis. However, in CR-31 treated KP we could not detect a upregulation of glycolysis measured by \(U-C^{13}\) glucose labeling and extra cellular acidification rates nor did CR-31 treatment affect  expression of glycolytic enzymes (e.g. HK1,HK2, LDHA, SLCA1, SLCA3). Furthermore, glucose deprivation did not further sensitise to CR-31 treatment. This was rather unexpected, given that increased glycolysis is characteristic for cancers as described by Otto walburg [@Warburg1927]. However, the polysome prolfing data revealed translational downregulation and subsequent reduction of protein expression for the glucose transporter Slc2a6. Indeed, perturbation of Slc2a6 using \(sgRNA^{Slc2a6}\) in N and KP organoids revealed a decrease in glucose uptake. From this we concluded that glycolytic compensation of KP is diminished by translational regulation of the glucose transporter Slc2a6 upon CR-31 treatment.

 Among the translationally activated genes in the CR-31 treated KP organoids where mRNAs involved in the glutamine metabolism (i.e. Slc1a5 and Gls1). Furthermore, glutamine levels were elevated in patient derived PDA cell lines treated with CR-31. Glutamine can be converted into \(\alpha\)-ketoglutarate and funneled into the krebs cycle and therefore can serve to increase energy production [@Xiao2016a]. Indeed, using gas chromatography mass spectrometry (GC/MS) to quantify metabolites after culturing PDA cells in \(C_5^{13}\)- glutamine, we identified a shift towards reductive carboxylation of \(\alpha\)-ketoglutarate obtained from \(C_5^{13}\)- glutamine to produce acetyl-CoA. Notably, the glutamine metabolism was not elevated in N organoids.

 A combined treatment of CR-31 with glutaminase inhibitors (BPTES or CB839) could sensitise to CR-31 treatment patient-derived PDA cells to CR-31 treatment. Therefore, our study suggests an eIF4A dependent translational program in PDA that can act as a theurapeutic target in PDA. Furthermore, a recently published ribosome profiling study of a CR-31 treated human pancreatic cancer cell line (PANC1) observed the same therapeutic effect of CR-31 treatment *in vivo* on survival and tumor volume [@Singh2021]. This underlines the significance of our study in identifying eIF4A as therapeutic target in PDA. 

 Nevertheless, the same study indicated differences on the underlying regulated mRNA subsets [@Singh2021]. They report, in line with the literature, that eIF4A depdent mRNAs show long and structured 5' UTRs containing \(GGC_4\) sequence motifs they propose to form G-quadruplexes [@Singh2021;@Wolfe2014].  While the 5' UTRs of the mRNAs identified herein where overall shorter, they were overall more structured. Also, the mRNAs identifed in Singh et. al. were involved in KRAS signalling, which was unaltered in our study.  Furthermore, Singh et. al.  chose to use ribosome profiling in their study that provides nucleotide resolution of ribosome occupancy [@Singh2021]. The nucleotide resolution of ribosome profiling could give insights whether ribosomes stall at G-quadruplex sites. Yet, the authors did not exploit this feature of their applied experimental methodology. Furthermore, studies indicate that the presence of G-quadruplex sites do not mean that they will form in the cell [@Weldon2016]. Indeed, Waldron et. al. showed that \(GGC_4\) structures fail to form G-quadruplexes in reporter mRNAs. However, classical hairpin secondary structures would form from these sequences and thus classical structures dictate eIF4A dependency [@Waldron2018].

 This raises some questions about the differences between experimental setups and their potential influence on biological outcomes and interpretation thereof. For instance, Singh et. al. performed ribosome profiling on a PANC1 cell line culture treated with 25nM CR-31, wheres herein we performed polysome profiling on a 3D-organoid culture treated with 10nM CR-31. The differences between ribosome and polysome profiling have been discussed extensively (see section \ref{exptMethod}). Furthermore, by measuring IC50 concentrations for CR-31 in a panel of pancreatic cancer cell lines, Singh et. al. show a ~6-fold difference in susceptability to CR-31 between the cell lines of which PANC1 cells were most sensitive to CR-31. Dosage dependent viability experiments of patient derived PDA cells in our study revealed that at 10nM CR-31 treatment cell viability was reduced by ~30%, whereas treatment with 25nM reduced viability by > 50%. Furthermore, in patient derived PDA a combination treatment of CR-31 and CB839 showed no synergistic effects at 25nM CR-31, whereas at 12.50nM synergy was observed. Therefore, combining the findings of these two studies indicate that CR-31 treatment in PDA indeed has a therapeutic effect, however the underlying mechanisms that are observed in the transcriptome wide analysis of translation efficiencies could be dependent on the experimental method to assess mRNA translation, the model system and drug concentrations.
\newline
\section{Study 3 - mTOR-dependent translational buffering overrides transcriptome alterations leading to
maintained proteome composition}

 Breast cancer is an umbrella term for a heterogeneous disease with numerous molecular subtypes with different clinical behaviour. Currently, breast cancer is classified into five major groups; luminal A, luminal B, HER-2, basal and normal like. The classification is based on histopatholgy and receptor status. Histopathological are determined by the degree of tumor differentiation (tubule formation), nuclear pleomorphism and proliferation (mitotic count). These features are then scored into a histological grade (I-III) [@Eliyatkin2015]. As mentioned earlier, receptor status of the progesterone (PR), estrogen (ER) and HER-2 are evaluated and have implications for neo adjuvant and adjuvant treatment strategies. Advances in technologies lead to classification of breast cancer subtypes using gene expression profiles,e.g. the PAM50 classification, that allow for unbiased classification of breast cancer [@Parker2009]. A study that correlated the gene expression profiles of the PAM50 classification found a significanlty higher mRNA:protein correlation for PAm50 genes, indicating their prognostic value [@Johansson2019]. Nevertheless, the authors observed intermixed luminal B and HER2 clustering based on protein expression in line with the literature that HER2 subtypes receive conflicting mRNA based classifications [@Johansson2019;@Prat2014]. Thus, the understanding of breast cancer subtypes has increased tremendously leading to improved and targeted treatment options, however there is need for further research to understand the full breast cancer spectrum on a molecular level.

 Another important factor to consider in breast cancer treatment are life style and other health related issues that could impact cancer progression or response to treatment, e.g. obesity. Studies in the 1970 observed unfavorable prognosis for breast cancer in obese women [@Abe1976]. More recent studies observed endocrine therapy to be less effective in obese patients and increased cancer incidence in individuals with higher weight or lack of physical activity [@Lee2019;@Eheman2012]. Obesity can pose an increased risk to develop metabolic disorders such as metabolic syndrome or type 2 diabetes that can lead to hyperinsulinemia, i.e. elevated physiological insulin levels [@Saltiel2001]. 

 The role of insulin in the body is to regulate glucose and lipid metabolism as well as protein synthesis. Protein synthesis and metaboloism are often dysregulated in cancer [@Saltiel2001;@Hanahan2011]. Insulin can bind to both IGF1R and INSR that activate the PI3K/AKT/mTOR and RAS/ERK signalling pathways. A role of insulin in cancer progression has initially been observed in long term tissues cultures where it increased metabolism as well as growth [@Osborne1976]. Another growth factor that is similar to insulin is IGF1 which acts to the same receptors as insulin. IGF1 plays a role in cancer progression and its levels are elevated upon hyperinsulinemia [@Christopoulos2015;@Bailyes1997;@Molinaro2019;@Gallagher2010]. 

 The importance of both IGF1 and insulin signalling in cancer has been well established now and led to development of therapeutic strategies by e.g. targeting both IGF1R and INSR or the PI3K signalling pathway [@Kuijjer2013;@Mayer2017]. Yet, to this day the full extent of insulin signalling on gene expression in cancer has not been investigated. This study aims to bridge this gap in knowledge by elucidating the effects of insulin signalling on gene expression using a multi-omics (including transcriptomics, translatomics and proteomics) approach to capture multiple steps of the gene expression pathway simultaneously. Furthermore, we asses insulin signalling in cancer cells as well as non-transformed epithelial cells.

 We first investigated the effects of insulin on gene expression in a luminal A breast cancer cell line, i.e. MCF7 cells. Polysome-profiling revealed a strong transcripional and translational response upon insulin stimulation. Among the translationally activated mRNAs were mRNAs with short 5' UTRs that harboured TOP motifs which is n accordance that insulin signalling leads to activation of mTORC1 and phosphorylation of its downstream targets. When visualising the mRNAs in the data where MCF7 cells were stimulated with insulin in the presence of torin1, an mTOR active site inhibitor, we observed that the changes in mRNA translation were almost fully reversed. This led to the conclusion that the effects of insulin on gene expression are to a great extent dependent on mTOR activity. 

 What was surprising to us was the observation that a subset of mRNAs exhibited translational buffering upon insulin stimulation while mTOR is inhibited. For these mRNAs, the total mRNA levels were increased, whereas the polysome-association was unaltered between conditions effectively uncoupling transcription from translation. As has been seen by others, we confirm that translationally buffered mRNAs maintain constant protein expression across conditions using HiRIEF LC/MS [@Lorent2019; @Branca2014]. These findings indicate that mTOR can act as a gatekeeper for transcriptional programs to fine tune translation in response to extra cellular or intra cellular signals.

 MCF7 cells are of epithelial origin and therefore not "classical" insulin sensitive such as adipose tissue or muscle cells. Their strong response to insulin prompted us to investigate whether this could be due to cellular plasticity in cancer. To assess this we chose to compare the effects found in MCF7 to a non-malignant epithelial cell type, i.e. HMEC cells. We found that insulin alone was not sufficient to stimulate the PI3k/AKT/mTOR pathway in HMEC to a similar extent as in MCF7. However, a combination treatment of insulin and IGF1 in HMEC cells elicited a similar reponse as insulin treatment in MCF7 alone.  Furthermore, it is known that during hyperinsulinemia IGF1 levels are increased and both IGF1 and insulin signalling has been associated with cancer [@Gallagher2010]. We therefore opted to compare the combined insulin + IGF1 treatment in HMEC to that of MCF7 assuming their signalling cascades are nearly identical as proposed in the literature [@Pollak2008;@Boucher2010].

 Polysome profiling of the insulin + IGF1 stimulated HMEC cells revealed a strong translational response, but was lacking the transcriptional activation found in MCF7 cells. Translationally activated mRNAs in HMEC showed 5' UTR features similar to those in MCF7 cells. Consequently, their translational activation was dependent on mTOR signalling evident from their translational suppression under conditions when mTOR was inhibited during insulin + IGF stimulation. Furthermore, comparing the mRNA signatures of HMEC and MCF7 in the opposite cell line we observed that changes at the level of translation were almost fully in accord across cell types.

 In contrast to MCF7 cells, HMEC did not elicit a strong transcriptional response as shown by the small number of changes in mRNA abundance. However, when comparing a recently described transcription signature induced after INSR translocation to the nucleus, we could see that in both HMEC and MCF7 these mRNA show changes in total mRNA levels but of differing magnitude [@Hancock2019]. A possible explanation for the different response in transcription could be due to differences in chromosome instability between the cell lines that expose different parts of the DNA to trans acting factors. While not assessed herein, a transcription factor analysis paired with chromatin immunoprecipitation (ChIP) sequencing could provide insight into this [@Solomon1988;@Park2009]. Assuming a consequence of having a weaker transcriptional response, HMEC cells did not elicit translational buffering upon insulin + IGF1 stimulation when mTOR was inhibited. Thus the effects of insulin and IGF1 signalling on mRNA translation are foremost mTOR dependent, however total mRNA responses differ between malignant and non-malignant epithelial cells.

 The translational buffering in MCF7 prompted us to investigate this phenomenon more. To assess differences dependent on mRNA characteristics we defined two subsets. The "reversed" and the "uncoupled" (that is translationally buffered when mTOR is inhibited) subsets that only differed in their total mRNA response when mTOR was inhibited during insulin stimulation. To rule out that the observed effects on total mRNA are technical artifacts we validated total mRNA levels for two genes from each subset. The differences in regulation of gene expression were not dependent on codon usage which has been described before in a different context [@Lorent2019]. However, overall uncoupled mRNAs had shorter 3' UTRs with a higher GC content and were depleted for HuR binding sites. 

 The depletion of HuR binding sites in the 3' UTRs of the uncoupled subset prompted us to investigate mRNA stability differences. Using a time series experiment under actinomycin D treatment to block transcription quantified using nanoString, we found significant longer mRNA half lifes for the uncoupled subset as compared to the reversed subset. From these data we hypothesised that there are different underlying mechanisms that regulate gene expression under conditions where mRNA translation is dampened between these subsets. Under this hypothesis the reversed subset is likely regulated through mRNA stability, whereas the more stable uncoupled subset requires to be regulated by translation as their total mRNA level remains high for longer periods of time.

 The involvement of HuR in this cannot be fully supported with our current data as the analysis only supports a correlation between the occurrence of HuR binding sites and the 3' UTRs of the reversed subset. The effect on stability could be due to other trans acting factors such as miRNAs [@ValinezhadOrang2014]. A way to increase confidence is to investigate HuRs involvement experimentally, e.g. we could use a single guide (sgRNA) to silence HuR and measure total mRNA levels of the reversed and uncoupled mRNAs previously validated by qPCR. If HuR is involved, we would expect that the reversed mRNAs retain higher total mRNA levels under the condition where mTOR is inhibited during insulin stimulation in MCF7 cells as compared to control. Furthermore, while the differences in total mRNA levels between the insulin and the insulin and torin1 treated conditions in the RNAseq data imply a treatment effect on the mRNA stability we did not observe this in the time chase experiment quantified by nanoString. This raises the question whether the transcription block induced by actinomycin D could influence the regulation of mRNA stability between treatments. We could address this by including actinomycin D in the sgHuR experiment and see if effects thereof differ or an experiment independent of sgHuR. Presence of such an effect could indicate a cross talk between transcription and regulation of mRNA stability. 

 Since the translational buffering identified herein was only observed in insulin treated cancer cells, we wondered whether this is only specific to this system. Cellular plasticity in cancer allows cancer cells to obtain stem cell like features [@Jewer2020; @Wahl2017; @Quail2012]. Therefore, we reasoned that a system were we study gene expression of stem cells could give some insight whether cancer cells obtained stem cell features that normal epithelial cells do not have. Furthermore, we wanted to investigate whether other means of mTOR inactivation, e.g. hypoxia, would lead to similar effects on gene expression. To assess these aspects we cultured H9 stem cells in medium with insulin present in normoxia and hypoxia. This experimental setup differs to that of MCF7 and HMEC cells as these were serum starved (i.e. no insulin in medium) prior to induction with insulin.

 Studying this system in H9 we could observe changes for all three modes for regulation of gene expression. Most notably, we observed a large fraction of translationally buffered mRNAs with similar 3' UTR characteristics to that of the uncoupled subset in MCF7 cells. Using publicly available data on mRNA stability we saw that the translationally buffered mRNA were overall more stable as compared to their background. Furthermore, visualising the reversed and uncoupled subset identified in MCF7 in the H9 data we observe differences in their regulation of total mRNA levels, indicating that these subset underlie different modes of regulation even across these two models. Furthermore, these data argue for that translational buffering observed in insulin treated MCF7 cells during mTOR inhibition is not limited to that system but can also occur under more physiological conditions. 

 Here we present an unprecedented and comprehensive investigation of the effects on insulin on gene expression in cancer cells and non- transformed epithelial cells across multiple steps of the gene expression pathway. Our results indicate that cancer cells have acquired an increased sensitivity to insulin signalling as compared to non-transformed epithelial cells that is largely dependent on mTOR in both cell types. Furthermore, we observed that cancer cells have the ability to translationally buffer mRNAs which is a feature they share with stem cells.















<!--chapter:end:03-results_and_discussion.Rmd-->

---
output: pdf_document
---


\chapter{Conclusions}
 
Cancer is a vastly heterogeneous disease that is characterised by uncontrollable growth and proliferation as well as dysregulated metabolism that can evade therapy through acquiring resistance. mRNA translation is a common denominator of these processes and it is therefore paramount to understand the precise mechanisms by which mRNA translation is regulated to better formulate therapeutic strategies against cancer. 
 
This thesis provides an advance in methodology to analyse transcriptome-wide changes translation efficiencies that can be applied to study cancer models. Using this methodology in the context of pancreatic cancer we could propose a possible new therapeutic strategy for this lethal disease where treatment options are limited. Lastly, we explore the effects of insulin on gene expression in an unprecedented study that highlighted an adapted insulin responsiveness of cancer cells. Furthermore, we illuminate the ability of cancer cells to translationally buffer genes which is a feature they share with stem cells and could therefore be an acquired mechanism.

Taken together this thesis provides insight on gene expression in two different cancer models that could potentially lead to clinical applications. While these studies are a step forward, more research is needed to grasp the full extent of the mechanisms involved in cancer.


<!--chapter:end:04-conclusion.Rmd-->

---
output: pdf_document
---

# Acknowledgments {-}

Welcome to the, according to some, most important part of this thesis. I greatly appreciate you taking the time to read my thesis and now have arrived to this section.
Without a doubt research is not something that can be done alone, and this thesis would not have happened without collaboration between researchers. I have not met everyone with whom I share a space in the author lists of the research papers presented herein. However, I am extremely grateful for everyone’s contribution that enabled these studies. A special thanks goes to Christine Chio for involving us in the pancreatic cancer studies. I really enjoyed discussing and exploring the data with you. Furthermore, I would also like to thank everyone I worked with on projects that are not included in this thesis that contributed greatly to my development.

I would also like to take this opportunity to thank my opponent, Alessandro Quattrone and the examination board members, Lukas Käll, Arne Östman and Rickard Sandberg that have sacrificed their time and energy to examine my PhD studies.

Ola as my primary supervisor, you are probably the person that I have cost most time and energy. I found it valuable that you could always find the time if there was the need to talk about my projects where you nudged me in the right direction when I was stuck. I am thankful for your support and guidance throughout my studies.

Ivan, thank you for your co-supervision of my studies. Although you were on a different continent, you were always open for discussions about science to which you brought a hint of philosophy.  Your view on the projects was extremely valuable as it comes from a vastly different perspective than ours over here which enriches science immensely. Also, thank you for the opportunities to involve me in paper reviews as they were extremely fun and helpful.
 
I would also like to thank previous and current member of the Ola Larsson group; Laia, Vincent, Julie, Baila, Carl Murie, Shuo, Margaritha, Dong mei, Inci, Shan, Hui, Krzysztof, Kiana, Kathleen and Johannes. 
Laia, thank you for the peanut butter jelly times and for showing me around the cell lab it was extremely fun to work with you. Julie, I learned a great deal while working together with you. Vincent, you always had a positive attitude and open for discussing bioinformatics thank you for that. Margaritha, I will absolutely burn it.  Hui, thank you for discussions about science and showing me places that I must visit in China. It was nice sharing an office with you. Shan, it was refreshing to also talk about other things than science while at work, thank you for that. Inci, your engagement in science is inspiring I wish you all the best during you studies. Kathleen, I encourage you to keep putting things into the square hole. Krzyzstof, I was always looking forward to your monthly visits it was a blast working together with you. Kiana, it was great discussing science with you, you will do great in your studies. Johannes, suffering together through the process of finishing up our shared project makes it (yes, still not done) more bearable. Thank you for all your input and help it would not get done without you.

I also want to thank the members of the Rolny group; Sabrina, Yangxun and Charlotte I enjoyed our interactions. 

Lars Eijssen, I really appreciated your supervision during my bachelor project in BiGCaT. Working with you encouraged me to continue the bioinformatics path in science, which partly led me here.
 
While I enjoyed support and guidance from my supervisors, co-workers and collaborators in my professional career there were also numerous people that had a large influence on my sanity outside of my working environment.
 
Anne-Laure, while I only ever once made use of your mentorship you pretty well nailed it, thank you for that. I hope that we can grow a lot of vegetables in the allotment garden together soon.

Katharina and Alex, I really enjoy our interactions and nerf gun fights. Also, Katharina you will always have a special place in my heart for introducing the tree bark cheese to me.

John, thank you for your advice throughout the years and providing an environment full of humour (the right kind) when it was greatly needed. And of course, your networking skills should be noted.

Carl, Matthias, Irena, Kilian, Julia, Rickard, Christian, Naemi and Paula, I cannot be any happier to have such great climbing companions. I really like our group dynamic where we push each other’s limits in climbing. Training with you is always invigorating and helped me a lot to recharge!
I also want to thank Tom, Linda, Matthew, Nicole, Solmaz and Phillip you are great friends, and I am grateful to know you! It was extremely nice to get to travel with all of you and spend time together just relaxing.

Susanne, Antoine & Louise (AnSuLu), thank you for showing us around Switzerland and Nashville when we visited you there. I really hope we can soon meet up again.

Mam & Pap, ich danke euch für die undenliche Unterstützung die ich von euch bekomme in allem was ich tue. Tobias, danke für die tollen Spieleabende die wir zusammen verbracht haben, es ist toll so unseren Kontakt zu erhalten trotz der Ferne. Wie in alten Zeiten.  Sascha & Anna, ich hoffe bald eurer Paradis in der Eifel wieder besuchen zu können und freue mich Lotta kennen zu lernen. Kathrin, ich hoffe eines meiner Bücher findet einen Platz in eurem neuen Haus, ich bin ganz gespannt wie es wird!

To my partner Christina, nothing of this would have happened without getting to know you. Your support in everything along the way was, and always will be, invaluable. On a more personal note, I am grateful that I can call you my partner in life as in difficult times we bond together to face every obstacle, and in easier times we encourage each others complete and utter craziness.  I could not wish for anymore. 










<!--chapter:end:05-acknowledgments.Rmd-->

---
output: pdf_document
---

# References {-}

<!--chapter:end:06-references.Rmd-->

